<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Molecular Cloning | Protocol 1 Print Version</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1251"><!--================== Designed and Developed by WriteDesign =================--><!--======================= Powered by RomeBlack, Inc. =======================--><!--=== Copyright 2000, 2001 Cold Spring Harbor Labs. All Rights Reserved. ===-->
<META content="" name=keywords>
<META 
content="This protocol describes how to generate a plasmid construct (pBAIT) that expresses a target protein fused to the bacterial LexA protein. pBAIT is cotransformed into yeast with a lexAop-lacZ reporter plasmid carrying the bacterial lacZ gene under the control of the lexA operator. The recipient yeast strain contains a chromosomally integrated leu2 reporter gene, also under the control of the lexA operator. pBAIT is analyzed in transformants to establish whether the bait protein is expressed as a stable nuclear protein of the correct size that does not independently activate transcription of either of the lexA operator-reporter genes to a significant extent. This protocol was provided by Erica A. Golemis and Ilya Serebriiskii (Fox Chase Cancer Center, Philadelphia, Pennsylvania)." 
name=description>
<META content="MSHTML 5.50.4134.600" name=GENERATOR></HEAD>
<BODY bgColor=#ffffff background=../izo/fon.gif>
<table width=600 border="0" cellspacing="0" cellpadding="0"><tr><td width=600><IMG height=49 hspace=4 src="../izo/logo_csh.gif" 
width=169 align=left vspace=4><IMG height=49 hspace=4 
src="../izo/print_header.gif" width=293 align=left vspace=4><BR 
clear=all><BR><FONT face="arial,verdana, helvetica" size=4><STRONG>Chapter 18, 
Protocol 1</STRONG></FONT><BR><BR clear=all><FONT face=arial,verdana,helvetica 
size=3><STRONG>Two-hybrid Systems<BR>Stage 1: Characterization of a Bait-LexA 
Fusion Protein</STRONG></FONT><BR><BR><FONT face="arial,verdana, helvetica" 
size=2>This protocol describes how to generate a plasmid construct (pBAIT) that 
expresses a target protein fused to the bacterial LexA protein. pBAIT is 
cotransformed into yeast with a <I>lexAop-lacZ</I> reporter plasmid carrying the 
bacterial <I>lacZ</I> gene under the control of the <I>lexA</I> operator. The 
recipient yeast strain contains a chromosomally integrated <I>leu2</I> reporter 
gene, also under the control of the <I>lexA</I> operator. pBAIT is analyzed in 
transformants to establish whether the bait protein is expressed as a stable 
nuclear protein of the correct size that does not independently activate 
transcription of either of the <I>lexA</I> operator-reporter genes to a 
significant extent. This protocol was provided by Erica A. Golemis and Ilya 
Serebriiskii (Fox Chase Cancer Center, Philadelphia, Pennsylvania).</FONT> 
<BR><BR>
<TABLE border=1>
  <TBODY>
  <TR>
    <TD width=700><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" 
      width=11 border=0><FONT face=arial,verdana,helvetica 
      size=2><STRONG>CAUTION</STRONG></FONT><BR><IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0><FONT 
      face="arial,verdana, helvetica" size=2> <STRONG>RECIPE</STRONG></FONT> 
  </TD></TR></TBODY></TABLE><BR><BR><FONT face=arial,verdana,helvetica 
size=3><STRONG>MATERIALS</STRONG></FONT><BR><BR>
<TABLE cellPadding=0 cellborder="0">
  <TBODY>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Buffers and Solutions</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>2x SDS 
      gel-loading buffer</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Nucleic Acids and Oligonucleotides</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Target DNA 
      encoding the protein of interest (bait)</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Antibodies</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Monoclonal 
      antibody to LexA (CLONTECH) or Polyclonal antibody to LexA (Invitrogen) or 
      Specific antibody to the fusion domain of the target protein (if 
      available)</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Media</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>CM selective 
      medium</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Use the 
      table below to prepare the necessary selective media.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Yeast 
      selective X-gal medium</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>YPD 
      medium</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><IMG 
      src="../izo/chapt18_1.gif"><BR><I>To prepare liquid or agar medium, mix 
      the ingredients in a final volume of 1 liter of H<SUB>2</SUB>O. Autoclave 
      for 20 minutes. Cool the agar medium to 55°C before pouring plates: 
      Leucine stock = 4 mg/ml; Tryptophan stock = 4 mg/ml; Histidine stock = 4 
      mg/ml.</I><BR><I><SUP>a</SUP>For media containing X-gal, prepare the 
      appropriate base medium in 900 ml of H<SUB>2</SUB>O, and use it in the 
      recipe for yeast selective X-gal medium.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Additional Reagents</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 1 of this 
      protocol requires the reagents for subcloning listed in <A href="../01/17.htm">Chapter 
      1, Protocol 17</A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 2 of this 
      protocol requires the reagents for transformation of yeast.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 17 of 
      this protocol requires the reagents for immunoblotting.</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Vectors and Yeast Strains</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>S. 
      cerevisiae</I> strains for selection and propagation of vectors</FONT> 
  </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Vectors 
      carrying LexA and activation domain fusion sequences and LacZ reporter 
      plasmids (please see <A target=_blank 
      href="http://www.fccc.edu/research/labs/golemis/com_sources1.html">http://www.fccc.edu/research/labs/golemis/com_sources1.html</A>)</FONT> 
    </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 
  border=0></TD></TR></TBODY></TABLE><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" 
size=3><STRONG>METHOD</STRONG></FONT> <BR><FONT face="arial,verdana, helvetica" 
color=#000000 size=2>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Clone the 
  target DNA encoding the protein to be used as bait into the polylinker of a 
  LexA fusion vector (e.g., pMW101 or pMW103) to synthesize an in-frame fusion 
  to LexA. Ensure that a translational stop sequence is present at the carboxyl 
  terminus of the desired bait sequence. The resulting plasmid is referred to as 
  pBait. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Set up a series 
  of transformations of the EGY48 <I>lexAop-LEU2</I> selection strain of yeast 
  using the following combinations of LexA fusion and <I>lexAop-lacZ</I> 
  reporter plasmids: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
      size=2><B>a.</B></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>pBait + pMW112 
        (test for activation)</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
      size=2><B>b.</B></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>pSH17-4 + 
        pMW112 (positive control for activation)</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
      size=2><B>c.</B></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>pRFHM1 + pMW112 
        (negative control for activation)</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
      size=2><B>d.</B></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>pBait + pJK101 
        (test for repression/DNA binding)</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
      size=2><B>e.</B></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>pRFHM1 + pJK101 
        (positive control for repression)</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
      size=2><B>f.</B></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>pJK101 alone 
        (negative control for repression)</FONT></TD></TR></TBODY></TABLE></FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Plate each 
  transformation mixture on selective dropout plates: CM(Glu)-Ura-His (for 
  plasmid combinations <B>a-e</B>) or CM(Glu)-Ura (for plasmid combination 
  <B>f</B>), as appropriate. Incubate the plates for 2-3 days at 30°C to select 
  for transformed yeast colonies that contain the plasmids. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Make a master 
  plate of transformants, from which specific colonies can be assayed for the 
  phenotype of activation of <I>lacZ</I> and <I>LEU2</I> reporters as described 
  in Steps 5-9.<BR><I>Steps 5-9 are used to test the bait-LexA fusion protein 
  for transcriptional activity and to demonstrate that the fusion of the bait 
  does not affect LexA DNA-binding activity. Several independent colonies are 
  assayed for each combination of plasmids transformed in Step 2.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>From each 
  transformation <B>a-f</B> (from Step 2), use sterile, standard flat-edged 
  toothpicks to pick approx. 8 colonies. Touch a clean toothpick to the colony 
  to pick up cells, and restreak them as a 1-cm-long streak in a grid on a fresh 
  CM(Glu)-Ura-His or CM(Glu)-Ura plate. As many as 60-80 streaks can generally 
  be grown on a single plate. Incubate the plates overnight at 30°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>On the second 
  day, restreak from the two master plates to each of the 
  following:<BR><I>Transformations <B>a-f:</B></I> Streak onto CM(Glu, 
  X-gal)-Ura and onto CM(Gal, X-gal)-Ura<BR><I>Transformations <B>a-c:</B></I> 
  Streak onto CM(Glu)-Ura-His-Leu and onto CM(Gal)-Ura-His-Leu<BR><I>Only uracil 
  is dropped out (histidine is present) from the X-gal plates, to allow 
  side-by-side comparison of the JK101 plasmid-only transformation 
  </I><B>(f)</B><I> with </I><B>(d,e)</B><I>. Lack of selection for the 
  LexA-fused plasmid does not notably affect transcriptional activation over the 
  period of this assay.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Incubate the 
  plates for up to 4 days at 30°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Assay for 
  repression and activation activities: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>a.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>For repression, 
        observe the X-gal phenotype at approx. 12-24 hours after 
        streaking.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>b.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>For activation, 
        observe the X-gal phenotype between 18 and 72 hours after 
        streaking.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>c.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Observe the 
        Leu2 phenotype between 48 and 96 hours. The expected results for a 
        well-behaved bait are summarized below.</FONT></TD></TR></TBODY></TABLE>
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>•</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Optimally, at 
        12-24 hours after streaking to CM(Gal, X-gal)-Ura, the <B>d</B> + 
        <B>e</B> transformants should be discernibly lighter in color than 
        <B>f</B>.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>•</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>At 48 hours 
        after streaking to CM(Glu, X-gal)-Ura, <B>d</B> transformants should be 
        bright blue, <B>c</B> should be white, and <B>a</B> should be white or 
        very light blue.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>•</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>At 48 hours 
        after streaking, <B>b</B> transformants should be as well grown on 
        CM(Glu)-Ura-His-Leu or CM(Gal)-Ura-His-Leu as on a CM(Glu)-Ura-His 
        master plate, whereas <B>a</B> and <B>c</B> should show no 
      growth.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>•</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Ideally, 
        <B>a</B> transformants will still display no apparent growth at 96 hours 
        after streaking.</FONT></TD></TR></TBODY></TABLE></FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Select the 
  appropriate candidate colonies, based on the results of the repression and 
  activation assays. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>On the master 
  plate, mark the colonies that are to be assayed for protein expression. Use 
  the colony that has been shown to express bait appropriately as the founder to 
  grow a culture for transformation of a library (in <A href="02.htm">Chapter 
  18, Protocol 2</A>). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Analyze at 
  least two primary transformants for each novel bait construct. Include two 
  transformants as positive controls for protein expression (e.g., pRFHMI). 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>a.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Use a sterile 
        toothpick to pick colonies from the CM(Glu)-Ura-His master plate into 
        CM(Glu)-Ura-His liquid medium.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>b.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Grow the 
        cultures overnight on a roller drum or other shaker at 
30°C.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>c.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>In the morning, 
        dilute the saturated cultures into fresh tubes containing 3-5 ml of 
        CM(Glu)-Ura-His, with a starting density of OD<SUB>600</SUB> of approx. 
        0.15. Incubate the cultures for 4-6 hours at 30°C until the optical 
        density has doubled approx. twice (OD<SUB>600</SUB> approx. 
        0.45-0.7).</FONT></TD></TR></TBODY></TABLE></FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Transfer 1.5 ml 
  of each culture to a microfuge tube, and centrifuge the cells at maximum speed 
  for 3-5 minutes in a microfuge. The volume of the visible pellet should be 2-5 
  µl. Carefully decant or aspirate the supernatant. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Add 50 µl of 2x 
  SDS gel-loading buffer to the tube, and vortex the tube rapidly to resuspend 
  the pellet. Immediately place the tube either on dry ice or in a dry 
  ice/ethanol bath. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Transfer the 
  samples from the dry ice or -70°C directly to 100°C and boil them for 5 
  minutes. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Chill the 
  samples on ice and centrifuge them at maximum speed for 5-30 seconds in a 
  microfuge to pellet large cell debris. Load 20-50 µl into each lane of a 
  SDS-polyacrylamide gel. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Run the gel and 
  analyze the products to determine whether bait protein of the expected size is 
  expressed at reasonable levels. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>To anticipate 
  and forestall potential problems, analyze the lysates of yeast containing 
  LexA-fused baits by immunoblotting. </FONT></LI></OL></FONT><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>RECIPES</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/circle.gif" width=12 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Glycerol</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 10% (v/v) 
      solution: Dilute 1 volume of molecular-biology-grade glycerol in 9 volumes 
      of sterile pure H<SUB>2</SUB>O. Sterilize the solution by passing it 
      through a prerinsed 0.22-µm filter. Store in 200-ml aliquots at 
    4°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>SDS</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Also called sodium lauryl 
      sulfate. To prepare a 20% (w/v) solution, dissolve 200 g of 
      electrophoresis-grade SDS in 900 ml of H<SUB>2</SUB>O. Heat to 68°C and 
      stir with a magnetic stirrer to assist dissolution. If necessary, adjust 
      the pH to 7.2 by adding a few drops of concentrated HCl. Adjust the volume 
      to 1 liter with H<SUB>2</SUB>O. Store at room temperature. Sterilization 
      is not necessary. Do not autoclave.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>SDS Gel-loading Buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM Tris-Cl (pH 
      6.8)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>4% (w/v) SDS 
      (electrophoresis grade)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.2% (w/v) bromophenol 
      blue</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>20% (v/v) 
    glycerol</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>200 mM dithiothreitol or 
      <IMG src="../izo/beta.gif" border=0>-mercaptoethanol</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>For a 2x buffer.</P>
      <P>1x and 2x SDS gel-loading buffer lacking thiol reagents can be stored 
      at room temperature. Add the thiol reagents from 1 M (dithiothreitol) or 
      14 M (<IMG src="../izo/beta.gif" border=0>-mercaptoethanol) stocks just 
      before the buffer is used.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Cl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve 121.1 g of Tris base in 800 ml of H<SUB>2</SUB>O. Adjust the 
      pH to the desired value by adding concentrated HCl.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2><I>pH</I></FONT> </TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2><I>HCl</I></FONT> 
        </TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.4 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>70 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.6 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>8.0 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>42 ml 
        </FONT></TD></TR></TBODY></TABLE>
      <P>(1 M) Allow the solution to cool to room temperature before making 
      final adjustments to the pH. Adjust the volume of the solution to 1 liter 
      with H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
      autoclaving.</P>
      <P>If the 1 M solution has a yellow color, discard it and obtain Tris of 
      better quality. The pH of Tris solutions is temperature-dependent and 
      decreases approx. 0.03 pH units for each 1°C increase in temperature. For 
      example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5°C, 
      25°C, and 37°C, respectively.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>YPD</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>yeast extract, 10 
    g</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>peptone, 20 
g</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>glucose, 20 
g</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>H<SUB>2</SUB>O, to 1000 
      ml</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>YPD (YEPD medium) is a complex medium for routine growth of yeast.</P>
      <P>To prepare plates, add 20 g of Bacto Agar (2%) before 
      autoclaving.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Yeast selective X-gal medium</STRONG></FONT> 
    </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>i. Prepare the appropriate CM base medium in 900 ml of H<SUB>2</SUB>O. 
      Autoclave the base medium and cool it to 55°C.</P>
      <P>ii. In a separate bottle, autoclave 7 g of sodium phosphate (dibasic) 
      and 3 g of sodium phosphate (monobasic) in 100 ml of distilled 
      H<SUB>2</SUB>O.</P>
      <P>iii. Mix the two autoclaved solutions together, and add 0.8 ml of 100 
      mg/ml X-gal (in <I>N,N-</I>dimethylformamide). Pour the 
    plates.</P></FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;     helvetica" size=3><STRONG>CAUTIONS</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" width=11 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>SDS</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">SDS (Sodium dodecyl sulfate) is toxic, an irritant, and poses a 
      risk of severe damage to the eyes. It may be harmful by inhalation, 
      ingestion, or skin absorption. Wear appropriate gloves and safety goggles. 
      Do not breathe the dust. </FONT></TD></TR></TBODY></TABLE><BR><BR><BR>
<OL></OL><BR><BR><BR><BR><BR><FONT face=Arial,Verdana,Helvetica 
size=1><STRONG>Copyright © 2000 by Cold Spring Harbor Laboratory Press. All 
rights reserved.</STRONG><BR>No part of these pages, either text or image may be 
used for any purpose other than personal use. Therefore, reproduction 
modification, storage in a retrieval system or retransmission, in any form or by 
any means, electronic, mechanical, or otherwise, for reasons other than personal 
use, is strictly prohibited without prior written 
permission.</FONT><BR><BR>
</td></tr></table>
</BODY></HTML>
