<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Molecular Cloning | Protocol 1 Print Version</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1251"><!--================== Designed and Developed by WriteDesign =================--><!--======================= Powered by RomeBlack, Inc. =======================--><!--=== Copyright 2000, 2001 Cold Spring Harbor Labs. All Rights Reserved. ===-->
<META content="" name=keywords>
<META 
content="Shotgun sequencing of a large segment of DNA involves random fragmentation of the target region into smaller segments that are subsequently cloned into a bacteriophage M13 vector. The goal is to create a library of overlapping clones that provide at least fivefold coverage over the entire length of the target fragment.In this protocol, the target segment is sheared by sonication or nebulization. Various sonication methods are described in Table 12-1 on page 12.11 of the print version of the manual. The termini of the resulting DNA fragments are repaired, phosphorylated, and fractionated according to size by gel electrophoresis. Molecules 0.8 to 1.5 kb in size are then recovered and cloned into the linearized, dephosphorylated DNA of a bacteriophage M13 vector." 
name=description>
<META content="MSHTML 5.50.4134.600" name=GENERATOR></HEAD>
<BODY bgColor=#ffffff background=../izo/fon.gif>
<table width=600 border="0" cellspacing="0" cellpadding="0"><tr><td width=600><IMG height=49 hspace=4 src="../izo/logo_csh.gif" 
width=169 align=left vspace=4><IMG height=49 hspace=4 
src="../izo/print_header.gif" width=293 align=left vspace=4><BR 
clear=all><BR><FONT face="arial,verdana, helvetica" size=4><STRONG>Chapter 12, 
Protocol 1</STRONG></FONT><BR><BR clear=all><FONT face=arial,verdana,helvetica 
size=3><STRONG>Generation of a Library of Randomly Overlapping DNA 
Inserts</STRONG></FONT><BR><BR><FONT face="arial,verdana, helvetica" 
size=2>Shotgun sequencing of a large segment of DNA involves random 
fragmentation of the target region into smaller segments that are subsequently 
cloned into a bacteriophage M13 vector. The goal is to create a library of 
overlapping clones that provide at least fivefold coverage over the entire 
length of the target fragment.<BR>In this protocol, the target segment is 
sheared by sonication or nebulization. Various sonication methods are described 
in Table 12-1 on page 12.11 of the print version of the manual. The termini of 
the resulting DNA fragments are repaired, phosphorylated, and fractionated 
according to size by gel electrophoresis. Molecules 0.8 to 1.5 kb in size are 
then recovered and cloned into the linearized, dephosphorylated DNA of a 
bacteriophage M13 vector.</FONT> <BR><BR>
<TABLE border=1>
  <TBODY>
  <TR>
    <TD width=700><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" 
      width=11 border=0><FONT face=arial,verdana,helvetica 
      size=2><STRONG>CAUTION</STRONG></FONT><BR><IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0><FONT 
      face="arial,verdana, helvetica" size=2> <STRONG>RECIPE</STRONG></FONT> 
  </TD></TR></TBODY></TABLE><BR><BR><FONT face=arial,verdana,helvetica 
size=3><STRONG>MATERIALS</STRONG></FONT><BR><BR>
<TABLE cellPadding=0 cellborder="0">
  <TBODY>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Buffers and Solutions</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Ammonium 
      acetate (10 M)</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0><IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>ATP</FONT> 
      <IMG height=11 alt=caution hspace=4 src="../izo/triangle.gif" width=12 
      border=0><IMG height=12 alt="" hspace=4 src="../izo/circle.gif" width=12 
      border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Optional, 
      please see Step 1.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Deionized 
      distilled H<SUB>2</SUB>O</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Ethanol</FONT> 
    </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Glycerol 
      (sterile, 100%)</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>IPTG</FONT> 
      <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" width=12 
      border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>MgCl<SUB>2</SUB></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>NaCl</FONT> 
      <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" width=12 
      border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Phenol 
      (saturated with Tris at pH 7.6)</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0><IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Phenol:chloroform (1:1, v/v)</FONT> <IMG height=11 alt=caution 
      hspace=4 src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>PEG 8000 (30% 
      w/v)</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0><IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Optional, 
      please see note to Step 1.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>PEG 8000 (20% 
      w/v) in 2.5 M NaCl</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0><IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>TE (pH 
      7.6)</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>10x Tris-Mg 
      buffer</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>TTE</FONT> 
      <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" width=12 
      border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>X-gal</FONT> 
      <IMG height=11 alt=caution hspace=4 src="../izo/triangle.gif" width=12 
      border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Enzymes and Buffers</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Bacteriophage 
      T4 DNA ligase</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>10x 
      Bacteriophage T4 DNA ligase buffer</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Bacteriophage 
      T4 DNA polymerase</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Bacteriophage 
      T4 polynucleotide kinase</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>10x 
      Bacteriophage T4 polynucleotide kinase buffer</FONT> <IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Klenow 
      fragment of <I>E. coli</I> DNA polymerase I</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Nucleic Acids and Oligonucleotides</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Bacteriophage 
      <IMG src="../izo/lambda.gif" border=0> or <IMG 
      src="../izo/phi.gif">X174 DNA, cleaved to completion with <I>Alu</I>I (1 
      µg in a volume of 20 µl of TE (pH 7.6)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>After 
      digestion, remove the restriction enzyme from the DNA by extraction with 
      phenol:chloroform. Recover the DNA by precipitation with ethanol and 
      dissolve in TE (pH 7.6). The digested DNA will be used to check the 
      efficiency of ligation of the dephosphorylated, linearized vector as 
      described in Step 13. The digested DNA is not required if commercial 
      preparations of vector are used.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>DNA size 
      markers</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Appropriate 
      size markers are, for example, fragments generated by digestion of pUC18 
      or pUC19 DNA with </I>Sau<I>3AI, or the 123-bp ladder sold by Life 
      Technologies, or <IMG src="../izo/phi.gif">X174 DNA digested with 
      </I>Hae<I>III (New England Biolabs).</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>dNTP solution 
      containing all four dNTPs, each at 2.0 mM</FONT> <IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Target 
      DNA</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Target DNA 
      is usually prepared by digesting a recombinant constructed in a 
      high-capacity vector (e.g., cosmid, BAC, PAC, or P1) with a restriction 
      enzyme that does not cleave within the cloned sequences. Wherever 
      possible, use a restriction enzyme that generates cohesive termini. This 
      simplifies ligation of the purified target DNA into concatemers (Step 1). 
      After release from the vector, purify the target DNA by electrophoresis 
      through a gel cast with low-melting-temperature agarose (please see <A href="../05/06.htm">Chapter 
      5, Protocol 6</A> or <A href="../05/07.htm">Chapter 
      5, Protocol 7</A>). Dissolve the purified DNA in TE (pH 7.6) at a 
      concentration of 1 mg/ml. Check the integrity and recovery of the purified 
      DNA by analyzing an aliquot (50 ng) by agarose gel 
      electrophoresis.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Media</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Rich M13 
      medium containing 5 mM MgCl<SUB>2</SUB></FONT> <IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Top 
      agarose</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>YT agar 
      plates</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Vectors and Bacterial Strains</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Bacteriophage 
      M13 vector DNA, equipped for blue-white screening, linearized, 
      blunt-ended, and dephosphorylated.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Vector DNAs 
      of this type may be purchased ready-made from several commercial 
      vendors.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>E. coli</I> 
      competent cells of an appropriate strain (e.g., XL1F&acute;-Blue, DH5<IMG 
      src="../izo/alpha.gif" border=0>F&acute;)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>To obtain a 
      library that covers the target DNA to the greatest depth, it is important 
      that the efficiency of transfection of </I>E. coli<I> be as high as 
      possible (&gt;10<SUP>9</SUP> transfectants/µg of closed circular vector 
      DNA). Preparations of competent </I>E. coli<I> that can be transformed 
      with high efficiency may be purchased ready-made from several commercial 
      vendors or may be prepared as described in <A href="../01/23.htm">Chapter 
      1, Protocols 23</A>, <A href="../01/24.htm">Chapter 
      1, Protocol 24</A>, <A href="../01/25.htm">Chapter 
      1, Protocol 25</A>, and <A href="../01/26.htm">Chapter 
      1, Protocol 26</A>. If an electroporation apparatus is available, we 
      recommend using preparations of electrocompetent </I>E. coli<I>, because 
      of their higher efficiencies of transformation.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 
  border=0></TD></TR></TBODY></TABLE><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" 
size=3><STRONG>METHOD</STRONG></FONT> <BR><FONT face="arial,verdana, helvetica" 
color=#000000 size=2>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Self-ligation 
  is required to ensure that sequences at the ends of the target DNA are 
  adequately represented in the population of fragments used to construct the 
  library.<BR>Transfer 5-10 µg of the purified target DNA (see Materials) to a 
  fresh microfuge tube and add: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x 
        bacteriophage T4 DNA ligase buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2.5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 mM 
      ATP</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2.5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>30% w/v PEG 
        8000 (<I>optional</I>)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5.0 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>bacteriophage 
        T4 DNA ligase</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>0.5-2.0 Weiss 
        units</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 25 
      µl</FONT></TD></TR></TBODY></TABLE>Incubate the mixture for 4 hours at 16°C, 
  and then inactivate the ligase by heating the mixture for 15 minutes at 
  68°C.<BR><I>If the target DNA is blunt-ended, include PEG in the reaction to 
  increase the efficiency of ligation.</I><BR><I>Use 0.5 Weiss unit for cohesive 
  termini, and 2.0 Weiss units for blunt termini.</I><BR><I>Omit ATP if included 
  in the ligase buffer.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Add 175 µl of 
  TE (pH 7.6), and purify the ligated DNA by extraction with phenol:chloroform. 
  Recover the ligated DNA by precipitation with 3 volumes of ethanol in the 
  presence of 2.0 M ammonium acetate. After centrifugation at maximum speed for 
  5 minutes in a microfuge, wash the pellet with 0.5 ml of 70% ethanol at room 
  temperature and centrifuge again. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Remove as much 
  of the supernatant as possible and allow the last traces of ethanol to 
  evaporate at room temperature. Dissolve the DNA in 25 µl of TE (pH 7.6) in a 
  microfuge tube. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Fragment the 
  target DNA into segments 0.8-1.5 kb in length by sonication or 
  nebulization.<BR><BR><B>To fragment the DNA by sonication</B> 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>a.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Place ice water 
        in the cup horn of the sonicator. Set the sonicator power switch to on, 
        the timer to hold, and the power setting to 10. Apply two 40-second 
        pulses, and allow the sonicator to warm up.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>b.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Place the tube 
        containing the DNA in the ice water such that the bottom of the tube is 
        1-2 mm above the hole in the center of the cup horn probe (please see 
        Figure 12-3 on page 12.16 in the print version of the manual), and 
        sonicate the DNA.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>c.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Centrifuge the 
        tube briefly to collect the sonicated DNA sample at the bottom of the 
        tube and place it on ice.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>d.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Analyze 1 µl of 
        the sonicated DNA sample with the appropriate molecular-weight markers 
        by electrophoresis through a 0.7% agarose gel. Keep the remainder of the 
        DNA on ice while the gel is running.</FONT></TD></TR></TBODY></TABLE><B>To 
  fragment the DNA by nebulization</B> 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>a.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Prepare the 
        following DNA solution and place it in the nebulizer 
    cup.</FONT></TD></TR></TBODY></TABLE>
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>DNA sample 
        (from Step 3)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5-10 
      µg</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x Tris-Mg 
        buffer (pH 8.0)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>200 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>sterile 100% 
        glycerol</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    ml</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>sterile 
        H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to a final 
        volume of 2 ml</FONT></TD></TR></TBODY></TABLE>
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>b.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Place the DNA 
        sample in an ice-water bath and nebulize using the empirically 
        determined optimal conditions (please see Figure 12-4 on page 12.17 in 
        the print version of the manual).</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>c.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Place the 
        nebulizer in a suitable centrifuge rotor and cushion it with pieces of 
        styrofoam. Centrifuge at 2000<I>g</I> (1000 rpm in a centrifuge equipped 
        with a microplate adaptor) briefly at 4°C to collect the DNA sample at 
        the bottom of the nebulizer cup.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>d.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Divide the DNA 
        sample into four equal aliquots in 1.5-ml microfuge tubes, carry out 
        standard ethanol precipitation of the DNA, and dry the DNA pellets under 
        vacuum.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>e.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Dissolve each 
        DNA pellet in 35 µl of TE (pH 7.6), and analyze 1 µl of the sheared DNA 
        with appropriate molecular-weight markers by electrophoresis through a 
        0.7% agarose gel. Keep the remainder of the DNA at 4°C while the gel is 
        running.</FONT></TD></TR></TBODY></TABLE></FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>To the 
  fragmented DNA (approx. 25 µl) add: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x 
        bacteriophage T4 DNA polymerase buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>4.0 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2.0 mM solution 
        of four dNTPs</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>4.0 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>bacteriophage 
        T4 DNA polymerase</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 
      units</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 40 
      µl</FONT></TD></TR></TBODY></TABLE>Incubate the reaction for 15 minutes at 
  room temperature, and then add approx. 5 units of the Klenow fragment. 
  Continue incubation for a further 15 minutes at room temperature.<BR><I>The 
  termini produced by nebulization and sonication are highly heterogeneous, 
  consisting of blunt-ended and frayed ends, with and without phosphate 
  residues. Because only a fraction of these molecules can be repaired by DNA 
  polymerases, the efficiency with which hydrostatically sheared DNA can be 
  cloned in bacteriophage M13 vectors is generally low. However, 5-10 µg of 
  sonicated, repaired, and size-selected target DNA generally yields several 
  thousand recombinant clones.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Purify the DNA 
  by extraction with phenol:chloroform. Transfer the aqueous (upper) phase to a 
  fresh tube, and adjust the solution to 0.1 M NaCl. Recover the DNA by 
  precipitation with 2 volumes of ethanol. Wash the pellet with 70% ethanol. 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Redissolve the 
  precipitated DNA in 25 µl of TE (pH 7.6). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Combine the 
  following in a microfuge tube: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>fragmented 
        DNA</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>23 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x 
        polynucleotide kinase buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>3 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 mM 
      ATP</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>3 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>bacteriophage 
        T4 polynucleotide kinase</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    unit</FONT></TD></TR></TBODY></TABLE><I>Bacteriophage T4 polynucleotide kinase 
  catalyzes the phosphorylation of the 5&acute; termini of the blunt-ended DNA 
  fragments. This step is not mandatory, but, in most cases, leads to more 
  efficient ligation of the fragments to the vector.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Incubate the 
  reaction for 30 minutes at 37°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Purify the 
  fragments of DNA of the desired size (0.8-1.5 kb) by electrophoresis through a 
  low-melting-temperature agarose gel (0.8%) or 5% neutral polyacrylamide gel 
  (see Chapter 5). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Recover the 
  target DNA from the gel by one of the methods described in Chapter 5. Dissolve 
  the purified DNA in 25 µl of TE (pH 7.6). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Check the 
  integrity and recovery of the purified DNA by analyzing an aliquot (1.0 µl) by 
  electrophoresis through a 1% agarose gel. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Set up a series 
  of test ligations containing 50 ng (approx. 0.01 pmole) of linearized and 
  dephosphorylated vector DNA and increasing concentrations of fragmented, 
  blunt-ended, phosphorylated target DNA (please see the table below). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Introduce 
  aliquots of the test ligations into competent <I>E. coli</I> of the 
  appropriate strain by electroporation or transformation (please see Chapter 1, 
  Protocols 23-26). Plate the bacteria on media containing IPTG and X-gal. 
  Incubate the plates overnight at 37°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>The following 
  day, count the number of blue and colorless plaques. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Set up a 
  large-scale ligation reaction using the minimum amount of fragmented, 
  blunt-ended target that will yield sufficient recombinant clones to complete 
  the sequencing project and transform <I>E. coli</I> with the ligated DNA. 
  Incubate the plates overnight at 37°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>The following 
  day, collect the plates and store the resulting transformants under the 
  appropriate conditions until required. Prepare template DNAs from a series of 
  individual colorless plaques as described in <A href="../03/04.htm">Chapter 
  3, Protocol 4</A>.<BR><IMG src="../izo/chapt12_1.gif"><BR><SUP>a</SUP><I>A = 
  50 ng of linearized, dephosphorylated vector (in a volume of 2 µl); B = 20 ng 
  of bacteriophage <IMG src="../izo/lambda.gif" border=0> DNA or <IMG 
  src="../izo/phi.gif">X174 RF DNA cleaved to completion with </I>Alu<I>I (in 
  a volume of 2 µl); C = fragmented, blunt-ended, phosphorylated target DNA. 
  Tubes 4, 5, 6, and 7 should contain 10, 20, 50, and 100 ng of size-selected, 
  fragmented target DNA, respectively (100 ng of size-selected fragments = 
  approx. 0.1 pmole).</I><BR><SUP>b</SUP><I>Some commercial ligase buffers 
  contain ATP. When using such buffers, omit the ATP.</I><BR><SUP>c</SUP><I>The 
  efficiency of ligation can be increased by adding PEG 8000 (30% w/v) to a 
  concentration of 5% in the final ligation mixture (see <A href="../01/19.htm">Chapter 
  1, Protocol 19</A>). It is important to warm the stock solution of PEG 8000 to 
  room temperature before adding it as the last component of the ligation 
  reactions. DNA can precipitate at cold temperatures from solutions containing 
  PEG 8000.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>For each set of 
  96 clones, inoculate 100 ml of LB medium or 2x YT medium in a 500-ml flask 
  with a single colony of a suitable F&acute; strain of <I>E. coli</I> (e.g. XL1-Blue, 
  XL1-Blue MRF&acute;, or DH5<IMG src="../izo/alpha.gif" border=0>F&acute;). Incubate the 
  culture for 6-8 hours at 37°C with agitation at 300 rpm or until it enters the 
  early log phase of growth. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Add 
  MgSO<SUB>4</SUB> to the culture to a final concentration of 5 mM. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Use a 
  multichannel pipettor to transfer 0.8-ml aliquots of the cells to individual 
  tubes in a 96-tube box. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Wearing gloves, 
  use sterile toothpicks to transfer individual well-separated, colorless 
  bacteriophage M13 plaques into each tube of the 96-tube arrays. Stab the 
  toothpick into the middle of a plaque and then drop the toothpick into the 
  culture tube. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>To avoid 
  confusion, leave the toothpicks in the tubes until all 96 tubes in the box 
  have been inoculated. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>When the last 
  plaque has been picked, remove the toothpicks, and seal the box. Label the box 
  and place it in an orbital shaker set at 250-300 rpm and 37°C. Repeat Steps 20 
  and 21 as needed. Incubate the infected cultures for 8-12 hours. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Remove the 
  boxes from the incubator and pellet the bacterial cells by centrifugation at 
  2400<I>g</I> (3000-3250 rpm in a centrifuge equipped with a microplate 
  adaptor) for 20 minutes.<T2.5> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Use a 
  multichannel pipettor to transfer 120 µl of 20% PEG 8000 in 2.5 M NaCl to 
  individual tubes in a fresh 96-tube box. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Carefully 
  remove the tubes from the centrifuge; use the multichannel pipettor to 
  transfer 0.6 ml of each supernatant to a tube containing the PEG/NaCl 
  solution.<BR><B>IMPORTANT</B><I> Do not disturb the pellet of bacterial cells 
  during this step. Inclusion of bacteria will drastically reduce the quality of 
  DNA sequence obtained.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Place a 96-tube 
  cap over the tubes containing the bacteriophage suspensions and PEG/NaCl 
  solution. Make sure that a liquid-tight seal has formed, and then mix the 
  solutions by inverting the box several times. Incubate the box for 30 minutes 
  at room temperature, followed by a 30-60-minute incubation on ice. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Collect the 
  precipitated bacteriophage by centrifugation of the boxes at 2400<I>g</I> 
  (3000-3250 rpm in a centrifuge equipped with a microplate adaptor) for 30 
  minutes. Remove a row of tubes and drain the supernatant by inversion over a 
  sink. A small white pellet should be visible on the bottom of each tube. 
  Return the row of tubes to the box. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>When all of the 
  tubes have been emptied, invert the boxes on a paper towel and allow the last 
  traces of supernatants to drain for a few minutes. Keeping the box in an 
  upside-down position, replace the wet paper towel with a fresh dry towel. 
  Transfer the inverted box-towel combinations to the centrifuge. Remove the 
  last traces of PEG/NaCl solution from the bacteriophage M13 pellets by 
  centrifugation at 300 rpm for 3-5 minutes. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Remove the 
  boxes from the centrifuge, check that the pellets have remained in place, and 
  add 20 µl of TTE to each tube. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Seal the tubes 
  with 3M silver foil tape and shake the boxes vigorously on a multitube 
  vortexer for 15-30 minutes. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Centrifuge the 
  boxes briefly to bring the solution to the bottom of the tubes. Pry the base 
  from each of the 96-tube boxes and place the tubes in an 80°C water bath for 
  10 minutes. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Remove the 
  tubes from the water bath and allow them to cool to room temperature. Replace 
  the bottoms of the boxes and briefly centrifuge each unit to bring the 
  solutions to the bottom of the tubes. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Use a 
  multichannel pipettor to transfer 70 µl of sterile H<SUB>2</SUB>O to the 
  individual wells of 96-well microtiter plates. Transfer the bacteriophage 
  lysates from the tubes in Step 33 to the microtiter plate wells. Mix the two 
  solutions by pipetting up and down. Seal the plates with a strip of 3M silver 
  foil tape or with the plate lid if sequencing is to be carried out within the 
  next 24-48 hours. Label each plate and store it at -20°C.<BR><I>The yield of 
  single-stranded M13 DNA should be 2.5-5 µg per starting culture.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Examine 
  aliquots (5 µl) of DNA selected at random from a few wells by electrophoresis 
  through a 1% agarose gel.<BR><I>Use between 2 µl and 7.5 µl of each DNA 
  preparation in cycle sequencing reactions (see <A href="06.htm">Chapter 
  12, Protocol 6</A>).</I> </FONT></LI></OL></FONT><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>RECIPES</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/circle.gif" width=12 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>ATP</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve an appropriate amount of solid ATP in 25 mM Tris-Cl (pH 
      8.0).</P>
      <P>Store the 10 mM ATP solution in small aliquots at 
  -20°C.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Ammonium Acetate</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 10 M 
      solution in 1 liter, dissolve 770 g of ammonium acetate in 800 ml of 
      H<SUB>2</SUB>O. Adjust volume to 1 liter with H<SUB>2</SUB>O. Sterilize by 
      filtration. Alternatively, to prepare a 100-ml solution, dissolve 77 g of 
      ammonium acetate in 70 ml of H<SUB>2</SUB>O at room temperature. Adjust 
      the volume to 100 ml with H<SUB>2</SUB>O. Sterilize the solution by 
      passing it through a 0.22-µm filter. Store the solution in tightly sealed 
      bottles at 4°C or at room temperature. Ammonium acetate decomposes in hot 
      H<SUB>2</SUB>O and solutions containing it should not be 
    autoclaved.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Bacteriophage T4 DNA Ligase 
      Buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>200 mM Tris-Cl (pH 
      7.6)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>50 mM 
      MgCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>50 mM 
      dithiothreitol</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.5 mg/ml bovine serum 
      albumin (Fraction V; Sigma) (<I>optional</I>)</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Divide the 10x buffer in 
      small aliquots and store at -20°C. Add ATP when setting up the reaction. 
      Add ATP to the reaction to an appropriate concentation (e.g., 1 
    mM).</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Bacteriophage T4 Polynucleotide Kinase 
      Buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>700 mM Tris-Cl (pH 
      7.6)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM 
      MgCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>50 mM 
      dithiothreitol</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Divide the 10x stock into 
      small aliquots and store frozen at -20°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>EDTA</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare EDTA at 0.5 M 
      (pH 8.0): Add 186.1 g of disodium EDTA•2H<SUB>2</SUB>O to 800 ml of 
      H<SUB>2</SUB>O. Stir vigorously on a magnetic stirrer. Adjust the pH to 
      8.0 with NaOH (approx. 20 g of NaOH pellets). Dispense into aliquots and 
      sterilize by autoclaving. The disodium salt of EDTA will not go into 
      solution until the pH of the solution is adjusted to approx. 8.0 by the 
      addition of NaOH.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Glycerol</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 10% (v/v) 
      solution: Dilute 1 volume of molecular-biology-grade glycerol in 9 volumes 
      of sterile pure H<SUB>2</SUB>O. Sterilize the solution by passing it 
      through a prerinsed 0.22-µm filter. Store in 200-ml aliquots at 
    4°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>IPTG</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>IPTG is 
      isopropylthio-<IMG src="../izo/beta.gif" border=0>-D-galactoside. Make a 
      20% (w/v, 0.8 M) solution of IPTG by dissolving 2 g of IPTG in 8 ml of 
      distilled H<SUB>2</SUB>O. Adjust the volume of the solution to 10 ml with 
      H<SUB>2</SUB>O and sterilize by passing it through a 0.22-µm disposable 
      filter. Dispense the solution into 1-ml aliquots and store them at 
      -20°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>LB</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>deionized H<SUB>2</SUB>O, 
      to 950 ml</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>tryptone, 10 
  g</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>yeast extract, 5 
    g</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>NaCl, 10 g</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>For solid medium, please 
      see Media Containing Agar or Agarose.</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare LB 
      (Luria-Bertani medium), shake until the solutes have dissolved. Adjust the 
      pH to 7.0 with 5 N NaOH (approx. 0.2 ml). Adjust the volume of the 
      solution to 1 liter with deionized H<SUB>2</SUB>O. Sterilize by 
      autoclaving for 20 minutes at 15 psi (1.05 kg/cm <SUP>2</SUP>) on liquid 
      cycle.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Media Containing Agar or 
      Agarose</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Prepare liquid media according to the recipes given. Just before 
      autoclaving, add one of the following:</P>
      <P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face="arial, helvetica,verdana" size=2>Bacto Agar (for 
            plates)</FONT></TD>
          <TD><FONT face="arial, helvetica,verdana" size=2>15 
        g/liter</FONT></TD></TR>
        <TR>
          <TD><FONT face="arial, helvetica,verdana" size=2>Bacto Agar (for top 
            agar)</FONT></TD>
          <TD><FONT face="arial, helvetica,verdana" size=2>7 
        g/liter</FONT></TD></TR>
        <TR>
          <TD><FONT face="arial, helvetica,verdana" size=2>agarose (for 
            plates)</FONT></TD>
          <TD><FONT face="arial, helvetica,verdana" size=2>15 
        g/liter</FONT></TD></TR>
        <TR>
          <TD><FONT face="arial, helvetica,verdana" size=2>agarose (for top 
            agarose)</FONT></TD>
          <TD><FONT face="arial, helvetica,verdana" size=2>7 
        g/liter</FONT></TD></TR></TBODY></TABLE></P>
      <P>Sterilize by autoclaving for 20 minutes at 15 psi (1.05 kg/cm 
      <SUP>2</SUP>) on liquid cycle. When the medium is removed from the 
      autoclave, swirl it gently to distribute the melted agar or agarose evenly 
      throughout the solution. <I>Be careful!</I> The fluid may be superheated 
      and may boil over when swirled. Allow the medium to cool to 50-60°C before 
      adding thermolabile substances (e.g., antibiotics). To avoid producing air 
      bubbles, mix the medium by swirling. Plates can then be poured directly 
      from the flask; allow approx. 30-35 ml of medium per 90-mm plate. To 
      remove bubbles from medium in the plate, flame the surface of the medium 
      with a Bunsen burner before the agar or agarose hardens. Set up a color 
      code (e.g., two red stripes for LB-ampicillin plates; one black stripe for 
      LB plates, etc.) and mark the edges of the plates with the appropriate 
      colored markers.</P>
      <P>When the medium has hardened completely, invert the plates and store 
      them at 4°C until needed. The plates should be removed from storage 1-2 
      hours before they are used. If the plates are fresh, they will "sweat" 
      when incubated at 37°C. When this condensation drops on the agar/agarose 
      surface, it allows bacterial colonies or bacteriophage plaques to spread 
      and increases the chances of cross-contamination. This problem can be 
      avoided by wiping off the condensation from the lids of the plates and 
      then incubating the plates for several hours at 37°C in an inverted 
      position before they are used. Alternatively, remove the liquid by shaking 
      the lid with a single, quick motion. To minimize the possibility of 
      contamination, hold the open plate in an inverted position while removing 
      the liquid from the lid.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>NaCl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 5 M 
      solution: Dissolve 292 g of NaCl in 800 ml of H<SUB>2</SUB>O. Adjust the 
      volume to 1 liter with H<SUB>2</SUB>O. Dispense into aliquots and 
      sterilize by autoclaving. Store the NaCl solution at room 
      temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>PEG 8000</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Working concentrations of 
      PEG (polyethylene glycol) range from 13% to 40% (w/v). Prepare the 
      appropriate concentration by dissolving PEG 8000 in sterile 
      H<SUB>2</SUB>O, warming if necessary. Sterilize the solution by passing it 
      through a 0.22-µm filter. Store the solution at room 
  temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Phenol</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Most batches of 
      commercial liquified phenol are clear and colorless and can be used in 
      molecular cloning without redistillation. Occasionally, batches of 
      liquified phenol are pink or yellow, and these should be rejected and 
      returned to the manufacturer. Crystalline phenol is not recommended 
      because it must be redistilled at 160°C to remove oxidation products, such 
      as quinones, that cause the breakdown of phosphodiester bonds or cause 
      cross-linking of RNA and DNA.Before use, phenol must be equilibrated to a 
      pH of &gt;7.8 because the DNA partitions into the organic phase at acid 
      pH. Wear gloves, full face protection, and a lab coat when carrying out 
      this procedure. <BR>1. Store liquified phenol at -20°C. As needed, remove 
      the phenol from the freezer, allow it to warm to room temperature, and 
      then melt it at 68°C. Add hydroxyquinoline to a final concentration of 
      0.1%. This compound is an antioxidant, a partial inhibitor of RNase, and a 
      weak chelator of metal ions. In addition, its yellow color provides a 
      convenient way to identify the organic phase. <BR>2. To the melted phenol, 
      add an equal volume of buffer (usually 0.5 M Tris-Cl [pH 8.0] at room 
      temperature). Stir the mixture on a magnetic stirrer for 15 minutes. Turn 
      off the stirrer, and when the two phases have separated, aspirate as much 
      as possible of the upper (aqueous) phase using a glass pipette attached to 
      a vacuum line equipped with appropriate traps. <BR>3. Add an equal volume 
      of 0. 1 M Tris-Cl (pH 8.0) to the phenol. Stir the mixture on a magnetic 
      stirrer for 15 minutes. Turn off the stirrer and remove the upper aqueous 
      phase as described in Step 2. Repeat the extractions until the pH of the 
      phenolic phase is &gt;7.8 (as measured with pH paper). <BR>4. After the 
      phenol is equilibrated and the final aqueous phase has been removed, add 
      0.1 volume of 0.1 M Tris-Cl (pH 8.0) containing 0.2% <IMG 
      src="../izo/beta.gif" border=0>-mercaptoethanol. The phenol solution may 
      be stored in this form under 100 mM Tris-Cl (pH 8.0) in a light-tight 
      bottle at 4°C for periods of up to 1 month.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Rich M13</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>LB</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>YT</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>For solid medium, please 
      see Media Containing Agar or Agarose.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>TE</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM Tris-Cl (desired 
      pH)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>10 mM EDTA (pH 
    8.0)</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>(10x Tris EDTA) Sterilize 
      solutions by autoclaving for 20 minutes at 15 psi (1.05 kg/cm 
      <SUP>2</SUP>) on liquid cycle. Store the buffer at room 
    temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>TTE</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.5% Triton X-100 in TE 
      (pH 8.0)</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Top agarose</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Media Containing Agar or 
      Agarose.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Cl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve 121.1 g of Tris base in 800 ml of H<SUB>2</SUB>O. Adjust the 
      pH to the desired value by adding concentrated HCl.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2><I>pH</I></FONT> </TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2><I>HCl</I></FONT> 
        </TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.4 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>70 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.6 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>8.0 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>42 ml 
        </FONT></TD></TR></TBODY></TABLE>
      <P>(1 M) Allow the solution to cool to room temperature before making 
      final adjustments to the pH. Adjust the volume of the solution to 1 liter 
      with H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
      autoclaving.</P>
      <P>If the 1 M solution has a yellow color, discard it and obtain Tris of 
      better quality. The pH of Tris solutions is temperature-dependent and 
      decreases approx. 0.03 pH units for each 1°C increase in temperature. For 
      example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5°C, 
      25°C, and 37°C, respectively.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Mg buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.5 M Tris-Cl (pH 
      8.0)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>150 mM 
      MgCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>For a 10x buffer. 
    </FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>YT</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>deionized H<SUB>2</SUB>O, 
      to 900 ml</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>tryptone, 16 
  g</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>yeast extract, 10 
    g</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>NaCl, 5 g</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>For solid medium, please 
      see Media Containing Agar or Agarose.</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare 2x YT medium, 
      shake until the solutes have dissolved. Adjust the pH to 7.0 with 5 N 
      NaOH. Adjust the volume of the solution to 1 liter with deionized 
      H<SUB>2</SUB>O. Sterilize by autoclaving for 20 minutes at 15 psi (1.05 
      kg/cm <SUP>2</SUP>) on liquid cycle.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>dNTP Solution</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve each dNTP (deoxyribonucleoside triphosphates) in 
      H<SUB>2</SUB>O at an approximate concentration of 100 mM. Use 0.05 M Tris 
      base and a micropipette to adjust the pH of each of the solutions to 7.0 
      (use pH paper to check the pH). Dilute an aliquot of the neutralized dNTP 
      appropriately, and read the optical density at the wavelengths given in 
      the table below. Calculate the actual concentration of each dNTP. Dilute 
      the solutions with H<SUB>2</SUB>O to a final concentration of 50 mM dNTP. 
      Store each separately at -70°C in small aliquots. For polymerase chain 
      reactions (PCRs), adjust the dNTP solution to pH 8.0 with 2 N NaOH. 
      Commercially available solutions of PCR-grade dNTPs require no 
      adjustment.</P>
      <P>
      <TABLE width=400>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>Base</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana 
            size=2>wavelength<BR>(nm)</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Extinction 
            Coefficient<BR>(E)&nbsp;(M<SUP>-1</SUP>cm<SUP>-1</SUP>)</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>A</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>259</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.54 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>G</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>253</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.37 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>C</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>271</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.10 x 
            10<SUP>3</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>T</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>267</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.60 x 
            10<SUP>3</SUP></FONT></TD></TR></TBODY></TABLE></P>
      <P>For a cuvette with a path length of 1 cm, absorbance = EM. 100 mM stock 
      solutions of each dNTP are commercially available 
    (Pharmacia).</P></FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;     helvetica" size=3><STRONG>CAUTIONS</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" width=11 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Ammonium acetate</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Ammonium acetate H<SUB>3</SUB>CCOONH<SUB>4</SUB> may be harmful by 
      inhalation, ingestion, or skin absorption. Wear appropriate gloves and 
      safety glasses. Use in a chemical fume hood. 
</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>PEG</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">PEG (Polyethylene glycol) may be harmful by inhalation, ingestion, 
      or skin absorption. Avoid inhalation of powder. Wear appropriate gloves 
      and safety glasses. </FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Phenol</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Phenol is extremely toxic, highly corrosive, and can cause severe 
      burns. It may be harmful by inhalation, ingestion, or skin absorption. 
      Wear appropriate gloves, goggles, and protective clothing. Always use in a 
      chemical fume hood. Rinse any areas of skin that come in contact with 
      phenol with a large volume of water and wash with soap and water; do not 
      use ethanol! </FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Phenol:chloroform</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Phenol:chloroform, see Phenol; Chloroform 
</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Radioactive substances</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Radioactive substances: When planning an experiment that involves 
      the use of radioactivity, consider the physico-chemical properties of the 
      isotope (half-life, emission type, and energy), the chemical form of the 
      radioactivity, its radioactive concentration (specific activity), total 
      amount, and its chemical concentration. Order and use only as much as 
      needed. Always wear appropriate gloves, lab coat, and safety goggles when 
      handling radioactive material. X-rays and gamma rays are electromagnetic 
      waves of very short wavelengths either generated by technical devices or 
      emitted by radioactive materials. They might be emitted isotropically from 
      the source or may be focused into a beam. Their potential dangers depend 
      on the time period of exposure, the intensity experienced, and the 
      wavelengths used. Be aware that appropriate shielding is usually made of 
      lead or other similar material. The thickness of the shielding is 
      determined by the energy(s) of the X-rays or gamma rays. Consult the local 
      safety office for further guidance in the appropriate use and disposal of 
      radioactive materials. Always monitor thoroughly after using 
      radioisotopes. A convenient calculator to perform routine radioactivity 
      calculations can be found at: <A target=_blank 
      href="http://www.graphpad.com/calculators/radcalc.cfm">http://www.graphpad.com/calculators/radcalc.cfm</A>. 
      </FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>X-gal</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">X-gal may be toxic to the eyes and skin. Observe general cautions 
      when handling the powder. Note that stock solutions of X-gal are prepared 
      in DMF, an organic solvent. For details, see DMF. See also BCIG. 
  </FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>REFERENCES</STRONG> 
</FONT><BR>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Anderson S. 
  1981. Shotgun DNA sequencing using cloned DNase I-generated fragments. 
  <I>Nucleic Acids Res.</I> 9:3015-3027. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Baer R., 
  Bankier A.T., Biggin M.D., Deininger P.L., Farrell P.J., Gibson T.J., Hatfull 
  G., Hudson G.S., Satchwell S.C., Seguin C., et al. 1984. DNA sequence and 
  expression of the B95-8 Epstein-Barr virus genome. <I>Nature</I> 310:207-211. 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Bankier A.T. 
  and Barrell B.G. 1983. Shotgun DNA-sequencing. <I>Techniques in life sciences: 
  Techniques in nucleic acid biochemistry</I> vol. B5:1-34. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Deininger P.L. 
  1983. Approaches to rapid DNA sequence analysis. <I>Anal. Biochem</I> 
  135:247-263. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Messing J. 
  1983. New M13 vectors for cloning. <I>Methods Enzymol.</I> 101:20-78. 
  </FONT></LI></OL><BR><BR><BR><BR><BR><FONT face=Arial,Verdana,Helvetica 
size=1><STRONG>Copyright © 2000 by Cold Spring Harbor Laboratory Press. All 
rights reserved.</STRONG><BR>No part of these pages, either text or image may be 
used for any purpose other than personal use. Therefore, reproduction 
modification, storage in a retrieval system or retransmission, in any form or by 
any means, electronic, mechanical, or otherwise, for reasons other than personal 
use, is strictly prohibited without prior written 
permission.</FONT><BR><BR>
</td></tr></table>
</BODY></HTML>
