<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Molecular Cloning | Protocol 3 Print Version</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1251"><!--================== Designed and Developed by WriteDesign =================--><!--======================= Powered by RomeBlack, Inc. =======================--><!--=== Copyright 2000, 2001 Cold Spring Harbor Labs. All Rights Reserved. ===-->
<META content="" name=keywords>
<META 
content="This protocol describes DNA-sequencing of single-stranded DNA templates in reactions catalyzed by Sequenase (version 2.0). All the materials required for sequencing with Sequenase version 2.0 are contained in kits, sold by USB/Amersham Life Science, which include a detailed protocol describing the reagents and steps used in the sequencing reaction. This kit is invaluable for first-time or occasional sequencers, but it is expensive. After gaining experience, it may turn out to be more economical to purchase Sequenase version 2.0 in bulk and then assemble the remaining reagents locally." 
name=description>
<META content="MSHTML 5.50.4134.600" name=GENERATOR></HEAD>
<BODY bgColor=#ffffff background=../izo/fon.gif>
<table width=600 border="0" cellspacing="0" cellpadding="0"><tr><td width=600><IMG height=49 hspace=4 src="../izo/logo_csh.gif" 
width=169 align=left vspace=4><IMG height=49 hspace=4 
src="../izo/print_header.gif" width=293 align=left vspace=4><BR 
clear=all><BR><FONT face="arial,verdana, helvetica" size=4><STRONG>Chapter 12, 
Protocol 3</STRONG></FONT><BR><BR clear=all><FONT face=arial,verdana,helvetica 
size=3><STRONG>Dideoxy-mediated Sequencing Reactions Using Bacteriophage T7 DNA 
Polymerase (Sequenase)</STRONG></FONT><BR><BR><FONT 
face="arial,verdana, helvetica" size=2>This protocol describes DNA-sequencing of 
single-stranded DNA templates in reactions catalyzed by Sequenase (version 2.0). 
All the materials required for sequencing with Sequenase version 2.0 are 
contained in kits, sold by USB/Amersham Life Science, which include a detailed 
protocol describing the reagents and steps used in the sequencing reaction. This 
kit is invaluable for first-time or occasional sequencers, but it is expensive. 
After gaining experience, it may turn out to be more economical to purchase 
Sequenase version 2.0 in bulk and then assemble the remaining reagents 
locally.</FONT> <BR><BR>
<TABLE border=1>
  <TBODY>
  <TR>
    <TD width=700><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" 
      width=11 border=0><FONT face=arial,verdana,helvetica 
      size=2><STRONG>CAUTION</STRONG></FONT><BR><IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0><FONT 
      face="arial,verdana, helvetica" size=2> <STRONG>RECIPE</STRONG></FONT> 
  </TD></TR></TBODY></TABLE><BR><BR><FONT face=arial,verdana,helvetica 
size=3><STRONG>MATERIALS</STRONG></FONT><BR><BR>
<TABLE cellPadding=0 cellborder="0">
  <TBODY>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Buffers and Solutions</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Deionized 
      distilled H<SUB>2</SUB>O (ice cold)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>ddNTP solution 
      (0.5 mM) of each of the four ddNTPs</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>dNTP solution 
      (0.5 mM) of each of the four dNTPs</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>DTT 
      (dithiothreitol) (100 mM)</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>5x Labeling 
      mixture and ddNTP extension/termination mixtures (ddCTP, ddTTP, ddATP, and 
      ddGTP)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>These 
      reaction mixtures can be assembled by mixing the volumes of the solutions 
      shown in the table below.</I><BR><IMG 
      src="../izo/chapt12_3.gif"><BR><I>The 5x labeling mixture contains 
      unlabeled dGTP, dCTP, and dTTP, each at a concentration of 7.5 mM. 
      Radiolabeled dATP is added at Step 7 to a final concentration of approx. 
      30 µM. Each of the ddNTP extension/termination mixtures contains all four 
      dNTPs at a concentration of 160 µM and a single ddNTP at a concentration 
      of 0.16 µM. The 5x labeling mixture and extension/termination mixtures 
      should be dispensed in 50-µl aliquots and stored frozen at 
      -20°C.</I><BR><I>When sequencing templates that are rich in GC structure, 
      use base analogs such as 7-deaza-2'-dGTP or dITP in place of dGTP. If 
      using 7-deaza-dGTP, simply replace the GTP in all the ddNTP 
      extension/termination mixtures with an equimolar amount of the base 
      analog.</I><BR><I>If using dITP, substitute 30 µl of a 0.5 µM stock 
      solution of dITP for dGTP in the 5x labeling mixture and decrease the 
      amount of H<SUB>2</SUB>O to 940 µl. In the ddNTP extension/termination 
      mixtures containing ddCTP, ddTTP, and ddATP, substitute 160 µl of a 0.5 mM 
      stock dITP solution for dGTP. In the extension/termination mixture 
      containing ddGTP, substitute 320 µl of 0.5 mM stock solution of dITP for 
      dGTP, decrease the amount of ddGTP to 3.2 µl, and decrease the volume of 
      H<SUB>2</SUB>O to 187 µl.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Formamide 
      loading buffer</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0><IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Sequenase 
      dilution buffer</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>5x Sequenase 
      reaction buffer with MgCl<SUB>2</SUB></FONT> <IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>5x Sequenase 
      reaction buffer with MnCl<SUB>2</SUB></FONT> <IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>TE (pH 
      7.6)</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Enzymes and Buffers</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Sequenase 
      (version 2.0)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Sequenase 
      is supplied by the manufacturer at a concentration of approx. 13 units/µl 
      (approx. 1 mg/ml). Store at -20°C (not in a frost-free 
      freezer).</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Yeast 
      inorganic pyrophosphatase</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Optional, 
      please see Step 6. This is supplied by the manufacturer (USB/Amersham Life 
      Science) at a concentration of 5 units/ml; it catalyzes the hydrolysis of 
      pyrophosphate to two molecules of 
      orthophosphate.</I><BR><I>Pyrophosphatase can be mixed with Sequenase at a 
      ratio of 1 unit of pyrophosphatase to 3 units of Sequenase. This is best 
      done by mixing equal volumes of the two enzyme preparations and diluting 
      the mixture with 6 volumes of enzyme dilution buffer. This dilutes 
      Sequenase to its working concentration of approx. 1.6 units/µl and adds 
      enough pyrophosphatase to prevent build-up of pyrophosphate; 2 µl of the 
      mixture of diluted enzymes is required for each sequencing 
      reaction.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Nucleic Acids and Oligonucleotides</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Oligonucleotide primer</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Use at a 
      concentration of 0.5 pmole/µl (approx. 3.3 ng/µl) in H<SUB>2</SUB>O or TE 
      (pH 7.6).</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Template DNAs 
      (1 µg) in TE (pH 7.6)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2><I>Single-stranded DNA (1 µg) or 2.0 µg of denatured 
      double-stranded plasmid DNA is required for each set of four sequencing 
      reactions. Linear double-stranded DNA can be denatured and annealed to the 
      primer by mixing the template with an excess of primer and heating the 
      mixture in a boiling water bath for approx. 2 minutes and then plunging 
      the tube into an ice-water bath. The mixture should not be allowed to warm 
      up and should be used without delay.</I><BR><I>The concentration of 
      single-stranded DNA in small-scale preparations of bacteriophage M13 
      recombinants varies from 0.05 µg/ml to 0.5 µg/ml, depending on the growth 
      rate of the particular bacteriophage. Under the conditions normally used 
      for sequencing, the template DNA is present in excess. Small variations in 
      the amounts of template added to different sets of sequencing reactions do 
      not generally affect the quality of the results.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Radioactive Compounds</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>[<IMG 
      src="../izo/alpha.gif" border=0>-<SUP>35</SUP>S]dATP (1000 Ci/mmole, 10 
      mCi/ml)</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><BR>or</FONT> 
    </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>[<IMG 
      src="../izo/alpha.gif" border=0>-<SUP>33</SUP>P]dATP (1000-3000 
      Ci/mmole, approx. 20 mCi/ml)</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><BR>or</FONT> 
    </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>[<IMG 
      src="../izo/alpha.gif" border=0>-<SUP>32</SUP>P]dATP (1000 Ci/mmole, 10 
      mCi/ml)</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Instead of 
      internal labeling with [<SUP>32</SUP>P]dATP, carry out sequencing with a 
      5&acute; <SUP>32</SUP>P-labeled oligonuclotide primer. In this case, 2 µl 
      (approx. 5 x 10<SUP>5</SUP> cpm; approx. 0.5 ng) of the radiolabeled 
      primer and 2 µl of H<SUB>2</SUB>O are included in the reaction mixture in 
      place of the unlabeled primer (Step 1) and [<SUP>32</SUP>P]dATP (Step 7). 
      All other steps remain the same. The 5&acute; termini of oligonucleotides are 
      generally labeled by transfer of [<IMG 
      src="../izo/gamma.gif">-<SUP>32</SUP>P] from [<IMG 
      src="../izo/gamma.gif">-<SUP>32</SUP>P]ATP in a reaction catalyzed by 
      polynucleotide kinase. For details of the labeling reaction, please see <A href="../10/02.htm">Chapter 
      10, Protocol 2</A>.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 
  border=0></TD></TR></TBODY></TABLE><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" 
size=3><STRONG>METHOD</STRONG></FONT> <BR><FONT face="arial,verdana, helvetica" 
color=#000000 size=2>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>To a 0.5-ml 
  microfuge tube or well of a microtiter plate, add: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>single-stranded 
        template DNA (approx. 1 µg/µl)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>oligonucleotide 
        primer (approx. 1 ng/µl)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>3 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5x Sequenase 
        reaction buffer containing</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>MgCl<SUB>2</SUB> or MnCl<SUB>2</SUB></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2></FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 10 
      µl</FONT></TD></TR></TBODY></TABLE><BR><B>IMPORTANT</B><I> When sequencing 
  double-stranded plasmid DNAs that have already been denatured and annealed to 
  a sequencing primer (<A href="02.htm">Chapter 
  12, Protocol 2</A>), ignore the first two steps of this protocol and begin at 
  Step 3.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Incubate the 
  tightly closed tube for 2 minutes at 65°C. Remove the tube from the water bath 
  and allow it to cool to room temperature over the course of 3-5 minutes. 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>While the 
  primer and template are cooling, thaw the 5x labeling and ddNTP 
  extension/termination mixtures and radiolabeled dATP. Store the thawed 
  solutions on ice. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Transfer 2.5 µl 
  of each ddNTP extension/termination mixture into separate 0.5-ml microfuge 
  tubes or into individual wells of a microtiter plate color-coded or labeled C, 
  T, A, and G. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Make a fivefold 
  dilution of the 5x labeling mixture in ice-cold H<SUB>2</SUB>O. A volume of 2 
  µl of the diluted labeling mixture is required for each template sequenced. 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Dilute 
  Sequenase in ice-cold Sequenase dilution buffer, with or without addition of 
  yeast pyrophosphatase, as described above in Materials.<BR><I>A volume of 2.0 
  µl, containing approx. 3.0 units of Sequenase enzyme, is required for each 
  template sequenced. Store the diluted enzyme on ice at all times.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>To carry out 
  the labeling reaction, add the following to the 10-µl annealing reaction of 
  Step 2: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>diluted 
        labeling mixture (from Step 5)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>0.1 M 
        dithiothreitol</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>[<IMG 
        src="../izo/alpha.gif" border=0>-<SUP>33</SUP>P]dATP, [<IMG 
        src="../izo/alpha.gif" border=0>-<SUP>32</SUP>P]dATP, or [<IMG 
        src="../izo/alpha.gif" border=0>-<SUP>35</SUP>S]dATP</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>0.5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>diluted 
        Sequenase (approx. 1.6 units/µl)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2.0 
    µl</FONT></TD></TR></TBODY></TABLE><BR>Mix the components of the reaction by 
  gently tapping the sides of the tube or microtiter plate and then incubate the 
  reaction for 2-5 minutes at 20°C.<BR><I>Store the diluted Sequenase in ice; do 
  not allow it to warm to ambient room temperature. Store the concentrated stock 
  of enzyme supplied by the manufacturer at -20°C. It may lose activity if 
  stored in an ice bucket for hours.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Toward the end 
  of the labeling reaction, prepare to set up the termination reactions by 
  prewarming the labeled microfuge tubes or microtiter plates to 37°C. <I>This 
  step is important!</I> Then transfer the 3.5 µl of the labeling reaction to 
  the walls of each of the prewarmed labeled microfuge tubes or to the sides of 
  prewarmed microtiter wells containing the appropriate dideoxy terminator 
  mixtures (Step 4 above). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Place the 
  microfuge tubes in a microfuge <I>at room temperature</I> (use appropriate 
  rotor or adaptors for 0.5-ml tubes or place inside decapitated 1.5-ml tubes), 
  or place the microtiter plates in a centrifuge <I>at room temperature</I> 
  equipped with an appropriate adaptor. Centrifuge the C, T, A, and G tubes or 
  plates for a few seconds at 2000 rpm to mix the components of the reactions. 
  Immediately transfer the reactions to a heating block or a water bath for 3-5 
  minutes at 37°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Stop the 
  reactions by adding 4 µl of formamide loading buffer. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>The reactions 
  may be stored for up to 5 days at -20°C or analyzed directly by denaturing gel 
  electrophoresis (<A href="08.htm">Chapter 
  12, Protocol 8</A> or <A href="09.htm">Chapter 
  12, Protocol 9</A>). After heat denaturation (2 minutes at 100°C) and quick 
  cooling on ice, load 3 µl of each of the C, T, A, and G reactions into 
  individual wells of a sequencing gel. </FONT></LI></OL></FONT><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>RECIPES</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/circle.gif" width=12 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>DTT</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 1 M 
      solution: Dissolve 3.09 g of dithiothreitol in 20 ml of 0.01 M sodium 
      acetate (pH 5.2) and sterilize by filtration. Dispense into 1-ml aliquots 
      and store at -20°C. Under these conditions, dithiothreitol is stable to 
      oxidation by air.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>EDTA</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare EDTA at 0.5 M 
      (pH 8.0): Add 186.1 g of disodium EDTA•2H<SUB>2</SUB>O to 800 ml of 
      H<SUB>2</SUB>O. Stir vigorously on a magnetic stirrer. Adjust the pH to 
      8.0 with NaOH (approx. 20 g of NaOH pellets). Dispense into aliquots and 
      sterilize by autoclaving. The disodium salt of EDTA will not go into 
      solution until the pH of the solution is adjusted to approx. 8.0 by the 
      addition of NaOH.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Formamide</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Deionize the formamide by 
      stirring on a magnetic stirrer with Dowex XG8 mixed bed resin for 1 hour 
      and filtering it twice through Whatman No. 1 paper. Store deionized 
      formamide in small aliquots under nitrogen at 
-70°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Formamide Loading Buffer</STRONG></FONT> 
  </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>80% (w/v) deionized 
      formamide</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>10 mM EDTA (pH 
    8.0)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>1 mg/ml xylene cyanol 
      FF</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>1 mg/ml bromophenol 
      blue</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Purchase a distilled 
      deionized preparation of formamide and store in small aliquots under 
      nitrogen at -20°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>NaCl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 5 M 
      solution: Dissolve 292 g of NaCl in 800 ml of H<SUB>2</SUB>O. Adjust the 
      volume to 1 liter with H<SUB>2</SUB>O. Dispense into aliquots and 
      sterilize by autoclaving. Store the NaCl solution at room 
      temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Sequenase Dilution Buffer</STRONG></FONT> 
  </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>10 mM Tris-Cl (pH 
      7.5)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>5 mM 
    dithiothreitol</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.5 mg/ml bovine serum 
      albumin</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Store the solution at 
      -20°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Sequenase reaction buffer with 
      MgCl<SUB>2</SUB></STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>200 mM Tris-Cl (pH 
      7.5)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM 
      MgCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>125 mM NaCl</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>For a 5x buffer. Store at 
      -20°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Sequenase reaction buffer with 
      MnCl<SUB>2</SUB></STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>200 mM Tris-Cl (pH 
      7.5)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>25 mM 
      MnCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>125 mM NaCl</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>For a 5x buffer. Store at -20°C.</P>
      <P>The presence of Mn<SUP>2+</SUP> in the Sequenase reaction buffer 
      increases the efficiency with which Sequenase will utilize ddNTPs, even if 
      Mg<SUP>2+</SUP> is also present. The net result is a more efficient 
      termination of sequencing reactions closer to the starting primer. This 
      modification is useful for improving the strength of the bands that run 
      close to the oligonucleotide primer or when using a base analog such as 
      dITP to iron out compressions in GC-rich regions of sequence.</P>
      <P>To use Mn<SUP>2+</SUP>, add 1 µl of 5x Sequenase reaction buffer with 
      MnCl<SUB>2</SUB> to the labeling reaction before adding Sequenase. 
      Alternatively, add 0.01 volume of 1 M MnCl<SUB>2</SUB> to the dideoxy-dNTP 
      extension/termination mixtures. Do not use manganese-containing buffers 
      that develop a slight yellowish color or that contain a yellow-brown 
      precipitate.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>TE</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM Tris-Cl (desired 
      pH)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>10 mM EDTA (pH 
    8.0)</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>(10x Tris EDTA) Sterilize 
      solutions by autoclaving for 20 minutes at 15 psi (1.05 kg/cm 
      <SUP>2</SUP>) on liquid cycle. Store the buffer at room 
    temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Cl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve 121.1 g of Tris base in 800 ml of H<SUB>2</SUB>O. Adjust the 
      pH to the desired value by adding concentrated HCl.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2><I>pH</I></FONT> </TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2><I>HCl</I></FONT> 
        </TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.4 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>70 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.6 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>8.0 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>42 ml 
        </FONT></TD></TR></TBODY></TABLE>
      <P>(1 M) Allow the solution to cool to room temperature before making 
      final adjustments to the pH. Adjust the volume of the solution to 1 liter 
      with H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
      autoclaving.</P>
      <P>If the 1 M solution has a yellow color, discard it and obtain Tris of 
      better quality. The pH of Tris solutions is temperature-dependent and 
      decreases approx. 0.03 pH units for each 1°C increase in temperature. For 
      example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5°C, 
      25°C, and 37°C, respectively.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>dNTP Solution</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve each dNTP (deoxyribonucleoside triphosphates) in 
      H<SUB>2</SUB>O at an approximate concentration of 100 mM. Use 0.05 M Tris 
      base and a micropipette to adjust the pH of each of the solutions to 7.0 
      (use pH paper to check the pH). Dilute an aliquot of the neutralized dNTP 
      appropriately, and read the optical density at the wavelengths given in 
      the table below. Calculate the actual concentration of each dNTP. Dilute 
      the solutions with H<SUB>2</SUB>O to a final concentration of 50 mM dNTP. 
      Store each separately at -70°C in small aliquots. For polymerase chain 
      reactions (PCRs), adjust the dNTP solution to pH 8.0 with 2 N NaOH. 
      Commercially available solutions of PCR-grade dNTPs require no 
      adjustment.</P>
      <P>
      <TABLE width=400>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>Base</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana 
            size=2>wavelength<BR>(nm)</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Extinction 
            Coefficient<BR>(E)&nbsp;(M<SUP>-1</SUP>cm<SUP>-1</SUP>)</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>A</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>259</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.54 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>G</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>253</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.37 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>C</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>271</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.10 x 
            10<SUP>3</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>T</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>267</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.60 x 
            10<SUP>3</SUP></FONT></TD></TR></TBODY></TABLE></P>
      <P>For a cuvette with a path length of 1 cm, absorbance = EM. 100 mM stock 
      solutions of each dNTP are commercially available 
    (Pharmacia).</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>ddNTP Solution</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve each ddNTP (dideoxyribonucleoside triphosphates) in 
      H<SUB>2</SUB>O at an approximate concentration of 100 mM. Use 0.05 M Tris 
      base and a micropipette to adjust the pH of each of the solutions to 7.0 
      (use pH paper to check the pH). Dilute an aliquot of the neutralized ddNTP 
      appropriately, and read the optical density at the wavelengths given in 
      the table below. Calculate the actual concentration of each ddNTP. Dilute 
      the solutions with H<SUB>2</SUB>O to a final concentration of 50 mM ddNTP. 
      Store each separately at -70&amp;#176;C in small aliquots.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>Base</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Wavelength 
          (nm)</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Extinction Coefficient 
            (E)(M-1cm-1)</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>A</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>259</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.54 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>G</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>253</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.37 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>C</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>271</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.10 x 
            10<SUP>3</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>T</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>267</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.60 x 
            10<SUP>3</SUP></FONT></TD></TR></TBODY></TABLE>
      <P>For a cuvette with a path length of 1 cm, absorbance = 
  Em.</P></FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;     helvetica" size=3><STRONG>CAUTIONS</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" width=11 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Formamide </STRONG></FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Formamide is teratogenic. The vapor is irritating to the eyes, 
      skin, mucous membranes, and upper respiratory tract. It may be harmful by 
      inhalation, ingestion, or skin absorption. Wear appropriate gloves and 
      safety glasses. Always use in a chemical fume hood when working with 
      concentrated solutions of formamide. Keep working solutions covered as 
      much as possible. </FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Radioactive substances</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Radioactive substances: When planning an experiment that involves 
      the use of radioactivity, consider the physico-chemical properties of the 
      isotope (half-life, emission type, and energy), the chemical form of the 
      radioactivity, its radioactive concentration (specific activity), total 
      amount, and its chemical concentration. Order and use only as much as 
      needed. Always wear appropriate gloves, lab coat, and safety goggles when 
      handling radioactive material. X-rays and gamma rays are electromagnetic 
      waves of very short wavelengths either generated by technical devices or 
      emitted by radioactive materials. They might be emitted isotropically from 
      the source or may be focused into a beam. Their potential dangers depend 
      on the time period of exposure, the intensity experienced, and the 
      wavelengths used. Be aware that appropriate shielding is usually made of 
      lead or other similar material. The thickness of the shielding is 
      determined by the energy(s) of the X-rays or gamma rays. Consult the local 
      safety office for further guidance in the appropriate use and disposal of 
      radioactive materials. Always monitor thoroughly after using 
      radioisotopes. A convenient calculator to perform routine radioactivity 
      calculations can be found at: <A target=_blank 
      href="http://www.graphpad.com/calculators/radcalc.cfm">http://www.graphpad.com/calculators/radcalc.cfm</A>. 
      </FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>REFERENCES</STRONG> 
</FONT><BR>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Tabor S. and 
  Richardson C.C. 1987. DNA sequence analysis with a modified bacteriophage T7 
  DNA polymerase. <I>Proc. Natl. Acad. Sci.</I> 84:4767-4771. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Tabor S. and 
  Richardson C.C. 1989. Selective inactivation of the exonuclease activity of 
  bacteriophage T7 DNA polymerase by in vitro mutagenesis. <I>J. Biol. Chem.</I> 
  264:6447-6458. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Tabor S. and 
  Richardson C.C. 1990. DNA sequence analysis with a modified bacteriophage T7 
  DNA polymerase. Effect of pyrophosphorolysis and metal ions. <I>J. Biol. 
  Chem.</I> 265:8322-8328. </FONT></LI></OL><BR><BR><BR><BR><BR><FONT 
face=Arial,Verdana,Helvetica size=1><STRONG>Copyright © 2000 by Cold Spring 
Harbor Laboratory Press. All rights reserved.</STRONG><BR>No part of these 
pages, either text or image may be used for any purpose other than personal use. 
Therefore, reproduction modification, storage in a retrieval system or 
retransmission, in any form or by any means, electronic, mechanical, or 
otherwise, for reasons other than personal use, is strictly prohibited without 
prior written permission.</FONT><BR><BR>
</td></tr></table>
</BODY></HTML>
