<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Molecular Cloning | Protocol 5 Print Version</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1251"><!--================== Designed and Developed by WriteDesign =================--><!--======================= Powered by RomeBlack, Inc. =======================--><!--=== Copyright 2000, 2001 Cold Spring Harbor Labs. All Rights Reserved. ===-->
<META content="" name=keywords>
<META 
content="Because sequencing reactions catalyzed by thermostable DNA polymerases such as Taq are carried out at elevated temperatures, problems caused by mismatched annealing of primers or templates rich in secondary structure are greatly alleviated." 
name=description>
<META content="MSHTML 5.50.4134.600" name=GENERATOR></HEAD>
<BODY bgColor=#ffffff background=../izo/fon.gif>
<table width=600 border="0" cellspacing="0" cellpadding="0"><tr><td width=600><IMG height=49 hspace=4 src="../izo/logo_csh.gif" 
width=169 align=left vspace=4><IMG height=49 hspace=4 
src="../izo/print_header.gif" width=293 align=left vspace=4><BR 
clear=all><BR><FONT face="arial,verdana, helvetica" size=4><STRONG>Chapter 12, 
Protocol 5</STRONG></FONT><BR><BR clear=all><FONT face=arial,verdana,helvetica 
size=3><STRONG>Dideoxy-mediated Sequencing of DNA Using Taq DNA 
Polymerase</STRONG></FONT><BR><BR><FONT face="arial,verdana, helvetica" 
size=2>Because sequencing reactions catalyzed by thermostable DNA polymerases 
such as <I>Taq</I> are carried out at elevated temperatures, problems caused by 
mismatched annealing of primers or templates rich in secondary structure are 
greatly alleviated.</FONT> <BR><BR>
<TABLE border=1>
  <TBODY>
  <TR>
    <TD width=700><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" 
      width=11 border=0><FONT face=arial,verdana,helvetica 
      size=2><STRONG>CAUTION</STRONG></FONT><BR><IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0><FONT 
      face="arial,verdana, helvetica" size=2> <STRONG>RECIPE</STRONG></FONT> 
  </TD></TR></TBODY></TABLE><BR><BR><FONT face=arial,verdana,helvetica 
size=3><STRONG>MATERIALS</STRONG></FONT><BR><BR>
<TABLE cellPadding=0 cellborder="0">
  <TBODY>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Buffers and Solutions</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Deionized 
      distilled H<SUB>2</SUB>O (ice cold)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>ddNTP solution 
      (5 mM) of each of the four ddNTPs</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>dNTP solution 
      (1 mM) of each of the four dNTPs</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Formamide 
      loading buffer</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0><IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>10x Labeling 
      mixture and ddNTP chain extension/termination mixtures</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>When 
      sequencing templates that are rich in secondary structure, replace dGTP in 
      the 10x labeling mixture and the ddNTP extension/termination mixtures with 
      an equimolar amount of 7-deaza-2&acute;-dGTP. Assemble the mixtures according to 
      the table below.</I><BR><IMG src="../izo/chapt12_5.gif"><BR><I>The 10x 
      labeling mixture contains dGTP, dCTP, and dTTP, each at a concentration of 
      1.5 µM.</I><BR><I>The ratios of ddNTP:dNTP in the four ddNTP 
      extension/termination mixtures are 30:1, 80:1, 80:1, and 3:1 for C, T, A, 
      and G, respectively.</I><BR><I>The 10x labeling mixture and ddNTP 
      extension/termination mixtures should be dispensed in 50-µl aliquots and 
      stored frozen at -20°C.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Enzymes and Buffers</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>5x Dideoxy 
      reaction buffer</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Taq</I> (5 
      units/µl) or a similar thermostable DNA polymerase</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Taq</I> 
      dilution buffer</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2><I>Thermostable enzymes purified from other organisms may require 
      slightly different reaction and dilution buffers. For optimal buffers of 
      individual enzymes, please see the specifications provided by the 
      manufacturer. 1 unit of thermostable DNA polymerase activity is usually 
      defined as that amount which will incorporate 10 nmoles of nucleotide into 
      an acid-precipitable form in 30 minutes at 70-80°C.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Nucleic Acids and Oligonucleotides</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Oligonucleotide primer</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Use at a 
      concentration of 1.0 pmole/µl (approx. 6.6 ng/µl) in TE (pH 
      7.6).</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Template DNAs 
      (100 ng/µl) in TE (pH 7.6)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2><I>Single-stranded DNA (500 ng) or 1.0 µg of denatured 
      double-stranded plasmid DNA is required for each set of four sequencing 
      reactions. Denature and anneal the linear double-stranded DNA to primer by 
      mixing the native template DNA with an excess of primer, heating the 
      mixture in a boiling water bath for approx. 2 minutes, and then plunging 
      the tube into an ice-water bath. Do not allow the mixture to warm up; use 
      immediately.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Radioactive Compounds</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>[<IMG 
      src="../izo/alpha.gif" border=0>-<SUP>32</SUP>P]dATP (3000 Ci/mmole, 10 
      mCi/ml)</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><BR>or</FONT> 
    </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>[<IMG 
      src="../izo/alpha.gif" border=0>-<SUP>35</SUP>S]dATP (1000 Ci/mmole, 10 
      mCi/ml)</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><BR>or</FONT> 
    </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>[<IMG 
      src="../izo/alpha.gif" border=0>-<SUP>33</SUP>P]dATP (2000-4000 
      Ci/mmole, approx. 10 mCi/ml)</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><BR>or</FONT> 
    </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>5' 
      <SUP>32</SUP>P-labeled oligonucleotide primer</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Instead of 
      internal labeling with radiolabeled dATP, carry out sequencing with an 
      oligonucleotide primer labeled at the 5&acute; end with <SUP>32</SUP>P or 
      <SUP>33</SUP>P. In this case, 1.0-1.5 pmoles of the radiolabeled primer 
      and 2 µl of H<SUB>2</SUB>O are included in the reaction mixture in place 
      of the unlabeled primer (Step 1) and radiolabeled dATP (Step 6). All other 
      steps remain the same. The 5&acute; termini of oligonucleotides are generally 
      labeled by transfer of [<IMG src="../izo/gamma.gif">-<SUP>32</SUP>P] 
      from [<IMG src="../izo/gamma.gif">-<SUP>32</SUP>P]ATP in a reaction 
      catalyzed by polynucleotide kinase. For details, please see <A href="../10/02.htm">Chapter 
      10, Protocol 2</A>.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 
  border=0></TD></TR></TBODY></TABLE><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" 
size=3><STRONG>METHOD</STRONG></FONT> <BR><FONT face="arial,verdana, helvetica" 
color=#000000 size=2>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>To a 0.5-ml 
  microfuge tube or well of a microtiter plate, add: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>single-stranded 
        template DNA</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5.0 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>(250 fmoles) 
        (100 ng/µl)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2></FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>oligonucleotide 
        primer (0.5 pmole)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>3.0 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>(approx. 3.3 
        ng/µl)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2></FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5x dideoxy 
        reaction buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2.0 
    µl</FONT></TD></TR></TBODY></TABLE></FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Incubate the 
  tightly closed tube for 2 minutes at 65°C. Remove the tube from the water bath 
  and allow it to cool to room temperature over the course of 3-5 minutes. 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>While the 
  primer and template are cooling, thaw the 10x labeling and ddNTP 
  extension/termination mixtures and radiolabeled dATP. Store the thawed 
  solutions on ice. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>For each 
  annealing reaction, transfer 4 µl of each ddNTP termination/extension mixture 
  into color-coded 0.5-ml microfuge tubes or into individual wells of a 
  microtiter plate prelabeled C, T, A, and G (i.e., 4 µl of ddCTP mixture to 
  tube/well labeled C, 4 µl of ddTTP mixture into tube/well labeled T, etc.). 
  Store the tubes/microtiter plates on ice. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Dilute enough 
  thermostable DNA polymerase enzyme 1:8 for all templates to be sequenced, for 
  example, 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2><I>Taq</I> DNA 
        polymerase (5-10 units/µl)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2><I>Taq</I> 
        dilution buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>7 
    µl</FONT></TD></TR></TBODY></TABLE><I>2 µl (2 units) of diluted enzyme is 
  needed for each set of four sequencing reactions. The final concentration of 
  the enzyme should be approx. 1 unit/µl. Store the diluted enzyme on ice at all 
  times.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Add the 
  following to each annealing reaction (Step 2 above): 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x labeling 
        mixture</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>radiolabeled 
        dATP</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>0.5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>diluted DNA 
        polymerase enzyme</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>8 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>(approx. 1 
        unit/µl)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
    size=2></FONT></TD></TR></TBODY></TABLE>Mix the contents of the tube by 
  vortexing and then incubate the reactions for 5 minutes at 45°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Transfer 4 µl 
  of the labeling reaction to the sides of each of the C, T, A, and G tubes or 
  to the sides of microtiter wells containing the appropriate dideoxy terminator 
  mixtures (Step 4 above). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Place the 
  microfuge tubes in a microfuge (use appropriate rotor or adaptors for 0.5-ml 
  tubes or place them inside decapitated 1.5-ml tubes), or place the microtiter 
  plates in a centrifuge equipped with an appropriate adaptor. Centrifuge the C, 
  T, A, and G tubes or plates for a few seconds at 2000 rpm to mix the 
  components of the reactions. Incubate the reactions for 5 minutes at 
  72°C.<BR><I>Incubate the tubes/plates in an efficient heating block or place 
  them in contact with H<SUB>2</SUB>O, oil, or other efficient conductor of 
  heat. Tubes/plates placed in an air incubator set at 72°C will not reach 
  optimum temperature for </I>Taq<I> DNA polymerase during the course of the 
  brief incubation.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Stop the 
  reactions by adding 4 µl of formamide loading buffer. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>The reactions 
  may be stored at -20°C for up to 5 days or analyzed directly by denaturing gel 
  electrophoresis (<A href="08.htm">Chapter 
  12, Protocol 8</A> or <A href="09.htm">Chapter 
  12, Protocol 9</A>). After heat denaturation (2 minutes at 100°C) and quick 
  cooling on ice, load 3 µl of each of the C, T, A, and G reactions into 
  individual wells of a sequencing gel. </FONT></LI></OL></FONT><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>RECIPES</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/circle.gif" width=12 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Dideoxy Reaction buffer</STRONG></FONT> 
</TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>200 mM Tris-Cl (pH 
      8.8)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>25 mM MgCl<SUB>2</SUB> 
      </FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>For a 5x 
  buffer.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>EDTA</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare EDTA at 0.5 M 
      (pH 8.0): Add 186.1 g of disodium EDTA•2H<SUB>2</SUB>O to 800 ml of 
      H<SUB>2</SUB>O. Stir vigorously on a magnetic stirrer. Adjust the pH to 
      8.0 with NaOH (approx. 20 g of NaOH pellets). Dispense into aliquots and 
      sterilize by autoclaving. The disodium salt of EDTA will not go into 
      solution until the pH of the solution is adjusted to approx. 8.0 by the 
      addition of NaOH.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Formamide</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Deionize the formamide by 
      stirring on a magnetic stirrer with Dowex XG8 mixed bed resin for 1 hour 
      and filtering it twice through Whatman No. 1 paper. Store deionized 
      formamide in small aliquots under nitrogen at 
-70°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Formamide Loading Buffer</STRONG></FONT> 
  </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>80% (w/v) deionized 
      formamide</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>10 mM EDTA (pH 
    8.0)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>1 mg/ml xylene cyanol 
      FF</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>1 mg/ml bromophenol 
      blue</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Purchase a distilled 
      deionized preparation of formamide and store in small aliquots under 
      nitrogen at -20°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Cl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve 121.1 g of Tris base in 800 ml of H<SUB>2</SUB>O. Adjust the 
      pH to the desired value by adding concentrated HCl.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2><I>pH</I></FONT> </TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2><I>HCl</I></FONT> 
        </TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.4 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>70 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.6 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>8.0 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>42 ml 
        </FONT></TD></TR></TBODY></TABLE>
      <P>(1 M) Allow the solution to cool to room temperature before making 
      final adjustments to the pH. Adjust the volume of the solution to 1 liter 
      with H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
      autoclaving.</P>
      <P>If the 1 M solution has a yellow color, discard it and obtain Tris of 
      better quality. The pH of Tris solutions is temperature-dependent and 
      decreases approx. 0.03 pH units for each 1°C increase in temperature. For 
      example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5°C, 
      25°C, and 37°C, respectively.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>dNTP Solution</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve each dNTP (deoxyribonucleoside triphosphates) in 
      H<SUB>2</SUB>O at an approximate concentration of 100 mM. Use 0.05 M Tris 
      base and a micropipette to adjust the pH of each of the solutions to 7.0 
      (use pH paper to check the pH). Dilute an aliquot of the neutralized dNTP 
      appropriately, and read the optical density at the wavelengths given in 
      the table below. Calculate the actual concentration of each dNTP. Dilute 
      the solutions with H<SUB>2</SUB>O to a final concentration of 50 mM dNTP. 
      Store each separately at -70°C in small aliquots. For polymerase chain 
      reactions (PCRs), adjust the dNTP solution to pH 8.0 with 2 N NaOH. 
      Commercially available solutions of PCR-grade dNTPs require no 
      adjustment.</P>
      <P>
      <TABLE width=400>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>Base</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana 
            size=2>wavelength<BR>(nm)</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Extinction 
            Coefficient<BR>(E)&nbsp;(M<SUP>-1</SUP>cm<SUP>-1</SUP>)</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>A</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>259</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.54 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>G</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>253</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.37 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>C</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>271</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.10 x 
            10<SUP>3</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>T</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>267</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.60 x 
            10<SUP>3</SUP></FONT></TD></TR></TBODY></TABLE></P>
      <P>For a cuvette with a path length of 1 cm, absorbance = EM. 100 mM stock 
      solutions of each dNTP are commercially available 
    (Pharmacia).</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>ddNTP Solution</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve each ddNTP (dideoxyribonucleoside triphosphates) in 
      H<SUB>2</SUB>O at an approximate concentration of 100 mM. Use 0.05 M Tris 
      base and a micropipette to adjust the pH of each of the solutions to 7.0 
      (use pH paper to check the pH). Dilute an aliquot of the neutralized ddNTP 
      appropriately, and read the optical density at the wavelengths given in 
      the table below. Calculate the actual concentration of each ddNTP. Dilute 
      the solutions with H<SUB>2</SUB>O to a final concentration of 50 mM ddNTP. 
      Store each separately at -70&amp;#176;C in small aliquots.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>Base</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Wavelength 
          (nm)</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Extinction Coefficient 
            (E)(M-1cm-1)</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>A</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>259</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.54 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>G</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>253</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.37 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>C</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>271</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.10 x 
            10<SUP>3</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>T</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>267</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.60 x 
            10<SUP>3</SUP></FONT></TD></TR></TBODY></TABLE>
      <P>For a cuvette with a path length of 1 cm, absorbance = 
  Em.</P></FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;     helvetica" size=3><STRONG>CAUTIONS</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" width=11 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Formamide </STRONG></FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Formamide is teratogenic. The vapor is irritating to the eyes, 
      skin, mucous membranes, and upper respiratory tract. It may be harmful by 
      inhalation, ingestion, or skin absorption. Wear appropriate gloves and 
      safety glasses. Always use in a chemical fume hood when working with 
      concentrated solutions of formamide. Keep working solutions covered as 
      much as possible. </FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Radioactive substances</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Radioactive substances: When planning an experiment that involves 
      the use of radioactivity, consider the physico-chemical properties of the 
      isotope (half-life, emission type, and energy), the chemical form of the 
      radioactivity, its radioactive concentration (specific activity), total 
      amount, and its chemical concentration. Order and use only as much as 
      needed. Always wear appropriate gloves, lab coat, and safety goggles when 
      handling radioactive material. X-rays and gamma rays are electromagnetic 
      waves of very short wavelengths either generated by technical devices or 
      emitted by radioactive materials. They might be emitted isotropically from 
      the source or may be focused into a beam. Their potential dangers depend 
      on the time period of exposure, the intensity experienced, and the 
      wavelengths used. Be aware that appropriate shielding is usually made of 
      lead or other similar material. The thickness of the shielding is 
      determined by the energy(s) of the X-rays or gamma rays. Consult the local 
      safety office for further guidance in the appropriate use and disposal of 
      radioactive materials. Always monitor thoroughly after using 
      radioisotopes. A convenient calculator to perform routine radioactivity 
      calculations can be found at: <A target=_blank 
      href="http://www.graphpad.com/calculators/radcalc.cfm">http://www.graphpad.com/calculators/radcalc.cfm</A>. 
      </FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>REFERENCES</STRONG> 
</FONT><BR>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Brow M.A.D. 
  1990. Sequencing with <I>Taq</I> DNA polymerase: <I>PCR protocols: A guide to 
  methods and applications</I> ed. M.A. Innis et al.:189-196. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Innis M.A., 
  Myambo K.B., Gelfand D.H., and Brow M.A. 1988. DNA sequencing with <I>Thermus 
  aquaticus </I>DNA polymerase and direct sequencing of polymerase chain 
  reaction-amplified DNA. <I>Proc. Natl. Acad. Sci.</I> 85:9436-9440. 
  </FONT></LI></OL><BR><BR><BR><BR><BR><FONT face=Arial,Verdana,Helvetica 
size=1><STRONG>Copyright © 2000 by Cold Spring Harbor Laboratory Press. All 
rights reserved.</STRONG><BR>No part of these pages, either text or image may be 
used for any purpose other than personal use. Therefore, reproduction 
modification, storage in a retrieval system or retransmission, in any form or by 
any means, electronic, mechanical, or otherwise, for reasons other than personal 
use, is strictly prohibited without prior written 
permission.</FONT><BR><BR>
</td></tr></table>
</BODY></HTML>
