<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Molecular Cloning | Protocol 4 Print Version</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1251"><!--================== Designed and Developed by WriteDesign =================--><!--======================= Powered by RomeBlack, Inc. =======================--><!--=== Copyright 2000, 2001 Cold Spring Harbor Labs. All Rights Reserved. ===-->
<META content="" name=keywords>
<META 
content="A synthetic oligonucleotide annealed to single-stranded DNA derived from a recombinant bacteriophage M13 or phagemid template is used to prime the synthesis of complementary radiolabeled DNA. Synthesis is catalyzed by the Klenow fragment of E. coli DNA polymerase I, which extends the annealed primer for various distances along the single-stranded template DNA. The products of the reaction, which are heterogeneous both in length and in the amount of incorporated radiolabeled dNTPs, are digested with a restriction enzyme to create double-stranded DNA fragments of uniform length, which are subsequently purified by agarose gel electrophoresis.    The oligonucleotide primer is usually complementary to a region of the lac gene immediately 3&acute; to the polycloning site in the mp series of bacteriophage M13 vectors. This " 
name=description sequence.? cloned the of portion a only represent that probes 
prepare to used be also can DNA within sequences specific complementary 
oligonucleotides custom-made However, vector. polylinker in site restriction any 
into been has segment it advantage companies, several by sold is which primer, 
universal?>
<META content="MSHTML 5.50.4134.600" name=GENERATOR></HEAD>
<BODY bgColor=#ffffff background=../izo/fon.gif>
<table width=600 border="0" cellspacing="0" cellpadding="0"><tr><td width=600><IMG height=49 hspace=4 src="../izo/logo_csh.gif" 
width=169 align=left vspace=4><IMG height=49 hspace=4 
src="../izo/print_header.gif" width=293 align=left vspace=4><BR 
clear=all><BR><FONT face="arial,verdana, helvetica" size=4><STRONG>Chapter 9, 
Protocol 4</STRONG></FONT><BR><BR clear=all><FONT face=arial,verdana,helvetica 
size=3><STRONG>Synthesis of Single-stranded DNA Probes of Defined Length from 
Bacteriophage M13 Templates</STRONG></FONT><BR><BR><FONT 
face="arial,verdana, helvetica" size=2>A synthetic oligonucleotide annealed to 
single-stranded DNA derived from a recombinant bacteriophage M13 or phagemid 
template is used to prime the synthesis of complementary radiolabeled DNA. 
Synthesis is catalyzed by the Klenow fragment of <I>E. coli</I> DNA polymerase 
I, which extends the annealed primer for various distances along the 
single-stranded template DNA. The products of the reaction, which are 
heterogeneous both in length and in the amount of incorporated radiolabeled 
dNTPs, are digested with a restriction enzyme to create double-stranded DNA 
fragments of uniform length, which are subsequently purified by agarose gel 
electrophoresis.<BR>The oligonucleotide primer is usually complementary to a 
region of the <I>lac</I> gene immediately 3&acute; to the polycloning site in the mp 
series of bacteriophage M13 vectors. This "universal" primer, which is sold by 
several companies, has the advantage that it can be used to prepare probes 
complementary to any segment of DNA that has been cloned into any restriction 
site in the polylinker of the vector. However, custom-made oligonucleotides 
complementary to specific sequences within the cloned DNA can also be used to 
prepare probes that represent only a portion of the cloned sequence.</FONT> 
<BR><BR>
<TABLE border=1>
  <TBODY>
  <TR>
    <TD width=700><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" 
      width=11 border=0><FONT face=arial,verdana,helvetica 
      size=2><STRONG>CAUTION</STRONG></FONT><BR><IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0><FONT 
      face="arial,verdana, helvetica" size=2> <STRONG>RECIPE</STRONG></FONT> 
  </TD></TR></TBODY></TABLE><BR><BR><FONT face=arial,verdana,helvetica 
size=3><STRONG>MATERIALS</STRONG></FONT><BR><BR>
<TABLE cellPadding=0 cellborder="0">
  <TBODY>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Buffers and Solutions</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Ammonium 
      acetate (10 M)</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0><IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>DTT 
      (Dithiothreitol) (1 M)</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>EDTA (0.5 M, 
      pH 8.0)</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Ethanol</FONT> 
    </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>NaCl (5 
      M)</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Phenol:chloroform (1:1, v/v)</FONT> <IMG height=11 alt=caution 
      hspace=4 src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>5x TBE</FONT> 
      <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" width=12 
      border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Optional, 
      please see Step 16.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Tris-Cl (1 M, 
      pH 7.6)</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Optional, 
      please see Step 16.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Enzymes and Buffers</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>10x Klenow 
      basic buffer</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Klenow 
      fragment of <I>E. coli</I> DNA polymerase I</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Restriction 
      enzyme(s), as appropriate for generation of the desired probe 
</FONT></TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>The 
      restriction enzyme used to cleave the products of the labeling reaction 
      should cleave between 200 bp and 1 kb downstream from the primer-binding 
      site.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Nucleic Acids and Oligonucleotides</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>dNTP solution 
      containing unlabeled dCTP, dGTP, and dTTP, each at 20 mM</FONT> <IMG 
      height=12 alt="" hspace=4 src="../izo/circle.gif" width=12 border=0> 
  </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>The 
      composition of this solution depends on the [<IMG src="../izo/alpha.gif" 
      border=0>-<SUP>32</SUP>P]dNTP to be used. If, as in this protocol, 
      radiolabeled dATP is used, the mix should contain dCTP, dTTP, and dGTP 
      each at a concentration of 20 mM. If two radiolabeled dNTPs are used, this 
      solution should contain the other two dNTPs.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>dATP (40 µM 
      and 20 mM)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Oligonucleotide primer</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>The best 
      results are obtained with a purified oligonucleotide (bacteriophage M13 
      Universal Primer or custom-synthesized oligonucleotide) (please see the 
      purification strategies in <A href="../10/01.htm">Chapter 
      10, Protocol 1 </A>). Resuspend the oligonucleotide at a concentration of 
      5 pmoles/ml in TE (pH 7.6).</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Template 
      DNA</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Purify the 
      single-stranded bacteriophage M13 or phagemid DNA as described in <A href="../03/04.htm">Chapter 
      3, Protocol 4 </A>or <A href="../03/08.htm">Chapter 
      3, Protocol 8 </A>. Measure the concentration of the DNA by absorption 
      spectrophotometry or staining with SYBR Gold. Approximately 1 µg of 
      template DNA in a volume of 15 µl or less is required for each synthetic 
      reaction.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Radioactive Compounds</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>[<IMG 
      src="../izo/alpha.gif" border=0>-<SUP>32</SUP>P]dATP (10 mCi/ml, sp. 
      act. 800-3000 Ci/mmole)</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Additional Reagents</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 17 of 
      this protocol requires the reagents listed in <A href="../05/04.htm">Chapter 
      5, Protocol 4 </A>or <A href="../05/12.htm">Chapter 
      5, Protocol 12 </A>.</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 
  border=0></TD></TR></TBODY></TABLE><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" 
size=3><STRONG>METHOD</STRONG></FONT> <BR><FONT face="arial,verdana, helvetica" 
color=#000000 size=2>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>In a 0.5-ml 
  microfuge tube, mix: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>single-stranded 
        template <BR>(bacteriophage M13 or phagemid DNA) (approx. 0.5 
        pmole)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µg</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>oligonucleotide 
        primer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 
      pmoles</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x Klenow 
        basic buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>3 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 20 
      µl</FONT></TD></TR></TBODY></TABLE></FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Heat the 
  mixture to 85°C for 5 minutes and then let it cool slowly to 37°C.<BR><I>This 
  slow cooling can be achieved by floating the microfuge tube in a piece of 
  Styrofoam in a 250-ml beaker filled with water equilibrated to 85°C. Place the 
  beaker containing the primer-template mix at room temperature until the 
  temperature of the water falls to 37°C (approx. 30 minutes). The same result 
  can be achieved using a thermal cycler programmed to heat at 85°C for 5 
  minutes before ramping down to 37°C over a 30-minute period.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>To the tube of 
  annealed primer and template, add: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>0.1 M 
        dithiothreitol</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10 mCi/ml [<IMG 
        src="../izo/alpha.gif" border=0>-<SUP>32</SUP>P]dATP (sp. act. 3000 
        Ci/mmole)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>40 µM 
        dATP</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 mM solution 
        of dTTP, dCTP, and dGTP</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR></TBODY></TABLE>Mix the reagents by gently tapping the side 
  of the tube. Centrifuge the tube at maximum speed for 1-2 seconds in a 
  microfuge to transfer all of the liquid to the bottom.<BR><I>As an 
  alternative, substitute [<IMG src="../izo/alpha.gif" 
  border=0>-<SUP>32</SUP>P]dNTPs of sp. act. 800 Ci/mmole in this reaction, in 
  which case, omit the 1 µl of 40 µM dATP.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Transfer 0.5 µl 
  of the mixture to a microfuge tube containing 15 µl of 20 mM EDTA (pH 8.0). 
  Store the tube on ice. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Add 1 µl (5 
  units) of the Klenow fragment to the remainder of the mixture. Mix the 
  components of the reaction by gently tapping the side of the tube. Incubate 
  the reaction for 30 minutes at room temperature.<BR><I>Approximately 5 units 
  of the Klenow fragment are required for each labeling reaction.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Transfer 0.5 µl 
  of the reaction to a fresh microfuge tube containing 20 µl of 0.5 M EDTA (pH 
  8.0). Store the tube on ice. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Add 1 µl of 20 
  mM unlabeled dATP to the remainder of the reaction. Mix by gently tapping the 
  side of the tube. Centrifuge the tube at maximum speed for 1-2 seconds in a 
  microfuge to transfer all of the liquid to the bottom. Incubate the reaction 
  mixture for a further 20 minutes at room temperature. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>During the 
  20-minute incubation (Step 7), measure the fraction of radioactivity in the 
  samples stored in Steps 4 and 6 that either has become insoluble in 10% TCA or 
  that adheres to a DE-81 filter. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Heat the 
  reaction to 68°C for 10 minutes to inactivate the Klenow fragment. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Adjust the 
  concentration of NaCl in the reaction to achieve optimal conditions for 
  cleavage of the product by the selected restriction enzyme.<BR><I>The 
  concentration of NaCl in the primer-extension reaction is 50 mM.</I><BR><I>If 
  the restriction enzyme works best in the absence of NaCl, or if the enzyme 
  requires unusual conditions (e.g., high concentrations of Tris or the presence 
  of detergent), transfer the radiolabeled DNA to the appropriate buffer.</I> 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Add 20 units of 
  the desired restriction enzyme, and incubate the reaction for 1 hour at the 
  appropriate temperature. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Purify the DNA 
  by standard extraction with phenol:chloroform, and remove unincorporated dNTPs 
  by spun-column chromatography or differential precipitation with 2.5 M 
  ammonium acetate and ethanol (please see Appendix 8 in the print version of 
  the manual). Add 0.5 M EDTA (pH 8.0) to a final concentration of 10 
  mM.<BR><B>IMPORTANT</B><I> It is essential that free magnesium ions not be 
  present when the DNA is exposed to alkali in the next step. Otherwise, the DNA 
  will form an insoluble complex with Mg(OH)<SUB>2</SUB>.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Isolate the 
  radiolabeled DNA by electrophoresis through a denaturing polyacrylamide gel 
  (please see <A href="../12/08.htm">Chapter 
  12, Protocol 8</A>) or an alkaline agarose gel (please see <A href="../05/08.htm">Chapter 
  5, Protocol 8 </A>), depending on the size of the fragment. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>After 
  electrophoresis, prepare the gel for autoradiography. Expose the gel to X-ray 
  film for 5-10 minutes. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Align the 
  developed film with the gel, using the images from the phosphorescent dots 
  (Glogos) or radioactive ink spots. Tape the film to the gel and mark the 
  position of the primer-extended radioactive DNA fragment on the back of the 
  gel plate. Remove the film and cut out the segment of the gel containing the 
  desired fragment of DNA. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>If a 
  polyacrylamide gel was used to separate the DNA fragment from the template, 
  proceed to Step 17. If an alkaline agarose gel was used, neutralize the gel by 
  gentle shaking for 45 minutes in 0.5 M Tris-Cl (pH 7.6), followed by shaking 
  for an additional 45 minutes in 0.5x TBE before the radiolabeled DNA is 
  eluted. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Extract the DNA 
  from the gel by electroelution or by crushing and soaking the polyacrylamide 
  gel slice in an appropriate buffer (please see <A href="../05/04.htm">Chapter 
  5, Protocol 4 </A>and <A href="../05/12.htm">Chapter 
  5, Protocol 12 </A>, respectively).<BR><I>Check the efficiency of the elution 
  process with a hand-held minimonitor.</I><BR><I>The eluted DNA is ready for 
  use as a probe and does not need to be denatured.</I> 
</FONT></LI></OL></FONT><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>RECIPES</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/circle.gif" width=12 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Ammonium Acetate</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 10 M 
      solution in 1 liter, dissolve 770 g of ammonium acetate in 800 ml of 
      H<SUB>2</SUB>O. Adjust volume to 1 liter with H<SUB>2</SUB>O. Sterilize by 
      filtration. Alternatively, to prepare a 100-ml solution, dissolve 77 g of 
      ammonium acetate in 70 ml of H<SUB>2</SUB>O at room temperature. Adjust 
      the volume to 100 ml with H<SUB>2</SUB>O. Sterilize the solution by 
      passing it through a 0.22-µm filter. Store the solution in tightly sealed 
      bottles at 4°C or at room temperature. Ammonium acetate decomposes in hot 
      H<SUB>2</SUB>O and solutions containing it should not be 
    autoclaved.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>DTT</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 1 M 
      solution: Dissolve 3.09 g of dithiothreitol in 20 ml of 0.01 M sodium 
      acetate (pH 5.2) and sterilize by filtration. Dispense into 1-ml aliquots 
      and store at -20°C. Under these conditions, dithiothreitol is stable to 
      oxidation by air.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>EDTA</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare EDTA at 0.5 M 
      (pH 8.0): Add 186.1 g of disodium EDTA•2H<SUB>2</SUB>O to 800 ml of 
      H<SUB>2</SUB>O. Stir vigorously on a magnetic stirrer. Adjust the pH to 
      8.0 with NaOH (approx. 20 g of NaOH pellets). Dispense into aliquots and 
      sterilize by autoclaving. The disodium salt of EDTA will not go into 
      solution until the pH of the solution is adjusted to approx. 8.0 by the 
      addition of NaOH.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Klenow basic buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>500 mM Tris-Cl (pH 
      7.5)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM 
      MgSO<SUB>4</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Sterilize the 10x buffer 
      by filtration and store it in small aliquots at 
-20°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>MgSO<SUB>4</SUB></STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 1 M 
      solution: Dissolve 12 g of MgSO<SUB>4</SUB> in a final volume of 100 ml of 
      H<SUB>2</SUB>O. Sterilize by autoclaving or filter sterilization. Store at 
      room temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>NaCl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 5 M 
      solution: Dissolve 292 g of NaCl in 800 ml of H<SUB>2</SUB>O. Adjust the 
      volume to 1 liter with H<SUB>2</SUB>O. Dispense into aliquots and 
      sterilize by autoclaving. Store the NaCl solution at room 
      temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>TBE</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Prepare a 5x stock solution in 1 liter of H<SUB>2</SUB>O:</P>
      <P>54 g of Tris base </P>
      <P>27.5 g of boric acid </P>
      <P>20 ml of 0.5 M EDTA (pH 8.0)</P><BR>
      <P>The 0.5x working solution is 45 mM Tris-borate/1 mM EDTA.</P>
      <P>TBE is usually made and stored as a 5x or 10x stock solution. The pH of 
      the concentrated stock buffer should be approx. 8.3. Dilute the 
      concentrated stock buffer just before use and make the gel solution and 
      the electrophoresis buffer from the same concentrated stock solution. Some 
      investigators prefer to use more concentrated stock solutions of TBE (10x 
      as opposed to 5x). However, 5x stock solution is more stable because the 
      solutes do not precipitate during storage. Passing the 5x or 10x buffer 
      stocks through a 0.22-µm filter can prevent or delay formation of 
      precipitates.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Cl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve 121.1 g of Tris base in 800 ml of H<SUB>2</SUB>O. Adjust the 
      pH to the desired value by adding concentrated HCl.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2><I>pH</I></FONT> </TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2><I>HCl</I></FONT> 
        </TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.4 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>70 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.6 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>8.0 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>42 ml 
        </FONT></TD></TR></TBODY></TABLE>
      <P>(1 M) Allow the solution to cool to room temperature before making 
      final adjustments to the pH. Adjust the volume of the solution to 1 liter 
      with H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
      autoclaving.</P>
      <P>If the 1 M solution has a yellow color, discard it and obtain Tris of 
      better quality. The pH of Tris solutions is temperature-dependent and 
      decreases approx. 0.03 pH units for each 1°C increase in temperature. For 
      example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5°C, 
      25°C, and 37°C, respectively.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>dNTP Solution</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve each dNTP (deoxyribonucleoside triphosphates) in 
      H<SUB>2</SUB>O at an approximate concentration of 100 mM. Use 0.05 M Tris 
      base and a micropipette to adjust the pH of each of the solutions to 7.0 
      (use pH paper to check the pH). Dilute an aliquot of the neutralized dNTP 
      appropriately, and read the optical density at the wavelengths given in 
      the table below. Calculate the actual concentration of each dNTP. Dilute 
      the solutions with H<SUB>2</SUB>O to a final concentration of 50 mM dNTP. 
      Store each separately at -70°C in small aliquots. For polymerase chain 
      reactions (PCRs), adjust the dNTP solution to pH 8.0 with 2 N NaOH. 
      Commercially available solutions of PCR-grade dNTPs require no 
      adjustment.</P>
      <P>
      <TABLE width=400>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>Base</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana 
            size=2>wavelength<BR>(nm)</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Extinction 
            Coefficient<BR>(E)&nbsp;(M<SUP>-1</SUP>cm<SUP>-1</SUP>)</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>A</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>259</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.54 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>G</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>253</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.37 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>C</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>271</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.10 x 
            10<SUP>3</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>T</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>267</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.60 x 
            10<SUP>3</SUP></FONT></TD></TR></TBODY></TABLE></P>
      <P>For a cuvette with a path length of 1 cm, absorbance = EM. 100 mM stock 
      solutions of each dNTP are commercially available 
    (Pharmacia).</P></FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;     helvetica" size=3><STRONG>CAUTIONS</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" width=11 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Ammonium acetate</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Ammonium acetate H<SUB>3</SUB>CCOONH<SUB>4</SUB> may be harmful by 
      inhalation, ingestion, or skin absorption. Wear appropriate gloves and 
      safety glasses. Use in a chemical fume hood. 
</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Phenol:chloroform</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Phenol:chloroform, see Phenol; Chloroform 
</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Radioactive substances</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Radioactive substances: When planning an experiment that involves 
      the use of radioactivity, consider the physico-chemical properties of the 
      isotope (half-life, emission type, and energy), the chemical form of the 
      radioactivity, its radioactive concentration (specific activity), total 
      amount, and its chemical concentration. Order and use only as much as 
      needed. Always wear appropriate gloves, lab coat, and safety goggles when 
      handling radioactive material. X-rays and gamma rays are electromagnetic 
      waves of very short wavelengths either generated by technical devices or 
      emitted by radioactive materials. They might be emitted isotropically from 
      the source or may be focused into a beam. Their potential dangers depend 
      on the time period of exposure, the intensity experienced, and the 
      wavelengths used. Be aware that appropriate shielding is usually made of 
      lead or other similar material. The thickness of the shielding is 
      determined by the energy(s) of the X-rays or gamma rays. Consult the local 
      safety office for further guidance in the appropriate use and disposal of 
      radioactive materials. Always monitor thoroughly after using 
      radioisotopes. A convenient calculator to perform routine radioactivity 
      calculations can be found at: <A target=_blank 
      href="http://www.graphpad.com/calculators/radcalc.cfm">http://www.graphpad.com/calculators/radcalc.cfm</A>. 
      </FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>REFERENCES</STRONG> 
</FONT><BR>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Ley T.J., 
  Anagnou N.P., Pepe G., and Nienhuis A.W. 1982. RNA processing errors in 
  patients with <IMG src="../izo/beta.gif" border=0>-thalassemia. <I>Proc. 
  Natl. Acad. Sci.</I> 79:4775-4779. </FONT></LI></OL><BR><BR><BR><BR><BR><FONT 
face=Arial,Verdana,Helvetica size=1><STRONG>Copyright © 2000 by Cold Spring 
Harbor Laboratory Press. All rights reserved.</STRONG><BR>No part of these 
pages, either text or image may be used for any purpose other than personal use. 
Therefore, reproduction modification, storage in a retrieval system or 
retransmission, in any form or by any means, electronic, mechanical, or 
otherwise, for reasons other than personal use, is strictly prohibited without 
prior written permission.</FONT><BR><BR>
</td></tr></table>
</BODY></HTML>
