<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Molecular Cloning | Protocol 2 Print Version</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1251"><!--================== Designed and Developed by WriteDesign =================--><!--======================= Powered by RomeBlack, Inc. =======================--><!--=== Copyright 2000, 2001 Cold Spring Harbor Labs. All Rights Reserved. ===-->
<META content="" name=keywords>
<META 
content="Here, the DNA-RNA hybrids synthesized in Stage 1 are converted into full-length double-stranded cDNAs. The primers for synthesis of second-strand cDNA are created by RNase H, which introduces nicks into the RNA moiety of the cDNA-mRNA hybrids. E. coli DNA polymerase I extends the newly created 3'-hydroxyl termini, using the first-strand cDNA as a template. In this way, the remaining segments of mRNA in the cDNA-mRNA hybrid are replaced with the newly synthesized second strand of DNA. Residual nicks are then repaired by E. coli DNA ligase, and the frayed termini of the double-stranded cDNA are polished by a DNA polymerase such as bacteriophage T4 DNA polymerase or Pfu. Finally, bacteriophage T4 polynucleotide kinase is used to catalyze the phosphorylation of 5'-hydroxyl groups on the ends of the cDNAs in preparation for ligation of linkers or adaptors (Chapter 11, Protocol 4)." 
name=description>
<META content="MSHTML 5.50.4134.600" name=GENERATOR></HEAD>
<BODY bgColor=#ffffff background=../izo/fon.gif>
<table width=600 border="0" cellspacing="0" cellpadding="0"><tr><td width=600><IMG height=49 hspace=4 src="../izo/logo_csh.gif" 
width=169 align=left vspace=4><IMG height=49 hspace=4 
src="../izo/print_header.gif" width=293 align=left vspace=4><BR 
clear=all><BR><FONT face="arial,verdana, helvetica" size=4><STRONG>Chapter 11, 
Protocol 2</STRONG></FONT><BR><BR clear=all><FONT face=arial,verdana,helvetica 
size=3><STRONG>Construction of cDNA Libraries<BR>Stage 2: Second-strand 
Synthesis</STRONG></FONT><BR><BR><FONT face="arial,verdana, helvetica" 
size=2>Here, the DNA-RNA hybrids synthesized in Stage 1 are converted into 
full-length double-stranded cDNAs. The primers for synthesis of second-strand 
cDNA are created by RNase H, which introduces nicks into the RNA moiety of the 
cDNA-mRNA hybrids. <I>E. coli</I> DNA polymerase I extends the newly created 
3'-hydroxyl termini, using the first-strand cDNA as a template. In this way, the 
remaining segments of mRNA in the cDNA-mRNA hybrid are replaced with the newly 
synthesized second strand of DNA. Residual nicks are then repaired by <I>E. 
coli</I> DNA ligase, and the frayed termini of the double-stranded cDNA are 
polished by a DNA polymerase such as bacteriophage T4 DNA polymerase or 
<I>Pfu</I>. Finally, bacteriophage T4 polynucleotide kinase is used to catalyze 
the phosphorylation of 5'-hydroxyl groups on the ends of the cDNAs in 
preparation for ligation of linkers or adaptors (<A href="04.htm">Chapter 
11, Protocol 4</A>).</FONT> <BR><BR>
<TABLE border=1>
  <TBODY>
  <TR>
    <TD width=700><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" 
      width=11 border=0><FONT face=arial,verdana,helvetica 
      size=2><STRONG>CAUTION</STRONG></FONT><BR><IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0><FONT 
      face="arial,verdana, helvetica" size=2> <STRONG>RECIPE</STRONG></FONT> 
  </TD></TR></TBODY></TABLE><BR><BR><FONT face=arial,verdana,helvetica 
size=3><STRONG>MATERIALS</STRONG></FONT><BR><BR>
<TABLE cellPadding=0 cellborder="0">
  <TBODY>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Buffers and Solutions</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Ammonium 
      sulfate (NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB> (1 M)</FONT> <IMG 
      height=12 alt="" hspace=4 src="../izo/circle.gif" width=12 border=0> 
  </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>10x 
      Bacteriophage T4 polynucleotide kinase buffer</FONT> <IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Chloroform</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>EDTA (0.5 M, 
      pH 8.0)</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Ethanol</FONT> 
    </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>MgCl<SUB>2</SUB> (1 M)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><IMG 
      src="../izo/beta.gif" border=0>-Nicotinamide adenine dinucleotide (<IMG 
      src="../izo/beta.gif" border=0>-NAD) (50 mM)</FONT> <IMG height=12 
      alt="" hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I><IMG 
      src="../izo/beta.gif" border=0>-NAD is a cofactor for </I>E. coli<I> DNA 
      ligase, as well as the electron acceptor; do not confuse it with <IMG 
      src="../izo/alpha.gif" border=0>-NAD.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Phenol:chloroform (1:1, v/v)</FONT> <IMG height=11 alt=caution 
      hspace=4 src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>RNase H buffer 
      (optional)</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Use this 
      buffer only if carrying out the alternative to Step 1.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Sodium acetate 
      (3 M, pH 5.2)</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>TE (pH 
      7.6)</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Tris-Cl (2 M, 
      pH 7.4)</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Enzymes and Buffers</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Bacteriophage 
      T4 DNA polymerase (2.5 units/µl)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Bacteriophage 
      T4 polynucleotide kinase (30 units/µl)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>E. coli</I> 
      DNA ligase</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>E. coli</I> 
      DNA polymerase I</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>RNase H</FONT> 
    </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>The RNase H 
      purified from </I>E. coli<I> is sold by several manufacturers and is 
      required at a specific activity of approx. 1000 units/ml.</I></FONT> 
</TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Nucleic Acids and Oligonucleotides</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>dNTP solution 
      containing all four dNTPs, each at 10 mM</FONT> <IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>First-strand 
      cDNA</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Use the 
      large-scale, first-strand reaction mixture prepared in <A href="01.htm">Chapter 
      11, Protocol 1</A>.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Marker 
      DNA</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Radioactive Compounds</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>[<IMG 
      src="../izo/alpha.gif" border=0>-<SUP>32</SUP>P]dCTP (10 mCi/ml, 400 
      Ci/mmole)</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Do not use 
      [<IMG src="../izo/alpha.gif" border=0>-<SUP>32</SUP>P]dTTP as a 
      radioactive tracer to monitor synthesis of the second strands of cDNA; it 
      may be incorporated preferentially into the region of the second strand 
      corresponding to the poly(A)<SUP>+</SUP> tract at the 3' terminus of the 
      mRNA.</I><BR><I>Thaw the [<IMG src="../izo/alpha.gif" 
      border=0>-<SUP>32</SUP>P]dCTP just before starting the protocol. Store the 
      thawed radiolabeled dCTP on ice until needed in Step 1. Return the 
      radiolabeled dCTP to the freezer immediately after use.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 
  border=0></TD></TR></TBODY></TABLE><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" 
size=3><STRONG>METHOD</STRONG></FONT> <BR><FONT face="arial,verdana, helvetica" 
color=#000000 size=2>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Add the 
  following reagents directly to the large-scale first-strand reaction mixture 
  (<A href="01.htm">Chapter 
  11, Protocol 1</A>): 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10 mM 
        MgCl<SUB>2</SUB></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>70 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 M Tris-Cl (pH 
        7.4)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10 mCi/ml [<IMG 
        src="../izo/alpha.gif" border=0>-<SUP>32</SUP>P]dCTP (400 
        Ci/mmole)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 M 
        (NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1.5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>RNase H (1000 
        units/ml)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2><I>E. coli</I> 
        DNA polymerase I (10,000 units/ml)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>4.5 
    µl</FONT></TD></TR></TBODY></TABLE><BR>Mix the reagents by gently vortexing, 
  and centrifuge the reaction mixture briefly in a microfuge to eliminate any 
  bubbles. Incubate the reaction for 2-4 hours at 16°C.<BR><I>Second-strand 
  synthesis catalyzed by RNase H and DNA polymerase I results in loss of 
  sequences (approx. 20 nucleotides) from the extreme 5&acute; end of the mRNA 
  template. The alternative to Step 1 reduces the risk of losing 5&acute;-terminal 
  regions of cDNA clones. Unfortunately, the alternative method results in a 
  reduced yield of double-stranded cDNA, and most investigators will therefore 
  opt for the standard Step 1 method.</I><BR><BR><I>Incubate the second-strand 
  synthetic reaction (above) for 2 hours at 16°C </I>in the absence of RNase 
  H<I>. Purify the resulting double-stranded cDNA by extraction with 
  phenol:chloroform and precipitation with ethanol in the presence of 0.3 M 
  sodium acetate (pH 5.2). Dissolve the precipitated DNA in 20 µl of 20 mM 
  Tris-Cl (pH 7.6), 20 mM KCl, 0.1 mM EDTA (pH 8.0), 0.1 mM dithiothreitol. 
  Digest the DNA for 20 minutes at 37°C with RNase H (0.5 unit). Proceed to Step 
  2.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>At the end of 
  the incubation, add the following reagents to the reaction mixture: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2><IMG 
        src="../izo/beta.gif" border=0>-NAD (50 mM)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2><I>E. coli</I> 
        DNA ligase (1000-4000 units/ml)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR></TBODY></TABLE>Incubate the reaction for 15 minutes at 
  room temperature. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>At the end of 
  the incubation, add 1 µl of a mixture containing all four dNTPs each at a 
  concentration of 10 mM and 2 µl (5 units) of bacteriophage T4 DNA polymerase. 
  Incubate the reaction mixture for 15 minutes at room temperature. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Remove a small 
  aliquot (3 µl) of the reaction. Measure the mass of second-strand DNA in the 
  aliquot as described in Steps 7 and 8. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>To the 
  remainder of the reaction, add 5 µl of 0.5 M EDTA (pH 8.0). Extract the 
  mixture once with phenol:chloroform and once with chloroform. Recover the DNA 
  by precipitation with ethanol in the presence of 0.3 M sodium acetate (pH 
  5.2). Dissolve the DNA in 90 µl of TE (pH 7.6). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>To the DNA, 
  add: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x T4 
        polynucleotide kinase buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>T4 
        polynucleotide kinase (3000 units/ml)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR></TBODY></TABLE>Incubate the reaction at room temperature 
  for 15 minutes. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Use the small 
  aliquot from Step 4 to determine the total amount of radioactivity and the 
  TCA-precipitable counts in 1 µl of the second-strand synthesis reaction as 
  described in Appendix 8 in the print version of the manual. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Use the 
  following equation to calculate the weight of the cDNA synthesized in the 
  second-strand reaction, taking into account the amount of dNTPs already 
  incorporated into the first strand of cDNA:<BR><U>cpm incorporated in the 
  second-strand reaction</U> x (66 µg - <I>x</I> µg)<BR>total cpm<BR>= µg of 
  second-strand cDNA synthesized<BR>where <I>x</I> is the weight of the first 
  strand of cDNA. The amount of second-strand cDNA synthesized is usually 70-80% 
  of the weight of the first strand. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Extract the 
  reaction containing phosphorylated cDNA (from Step 6) with an equal volume of 
  phenol:chloroform. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Separate the 
  unincorporated dNTPs from the cDNA by spun-column chromatography through 
  Sephadex G-50 equilibrated in TE (pH 7.6) containing 10 mM NaCl (please see 
  Appendix 8 in the print version of the manual). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Precipitate the 
  eluted cDNA by adding 0.1 volume of 3 M sodium acetate (pH 5.2) and 2 volumes 
  of ethanol. Store the sample on ice for at least 15 minutes. Recover the 
  precipitated DNA by centrifugation at maximum speed for 15 minutes at 4°C in a 
  microfuge. Use a hand-held minimonitor to check that all of the radioactivity 
  has been precipitated. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Wash the pellet 
  with 70% ethanol and centrifuge again. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Gently aspirate 
  all of the fluid (check to see that none of the radioactivity is in the 
  aspirated fluid), and allow the pellet to dry in the air. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Dissolve the 
  cDNA in 80 µl of TE (pH 7.6) if it is to be methylated by <I>Eco</I>RI 
  methylase (please see <A href="03.htm">Chapter 
  11, Protocol 3</A>). Alternatively, if the cDNA is to be ligated directly to 
  <I>Not</I>I or <I>Sal</I>I linkers, or an adaptor oligonucleotide (please see 
  <A href="04.htm">Chapter 
  11, Protocol 4</A>), resuspend the cDNA in 29 µl of TE (pH 7.6). 
</FONT></LI></OL></FONT><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>RECIPES</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/circle.gif" width=12 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG><IMG src="../izo/beta.gif" 
      border=0>-Nicotinamide adenine dinucleotide (<IMG src="../izo/beta.gif" 
      border=0>-NAD)</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Make a 50 mM stock 
      solution by dissolving 33.2 mg of <IMG src="../izo/beta.gif" 
      border=0>-nicotinamide adenine dinucleotide (<IMG src="../izo/beta.gif" 
      border=0>-NAD) in 1 ml of H<SUB>2</SUB>O. Store the solution in small 
      aliquots at -70°C. <IMG src="../izo/beta.gif" border=0>-NAD is a 
      cofactor for <I>E. coli</I> DNA ligase, as well as the electron acceptor; 
      do not confuse it with <IMG src="../izo/alpha.gif" 
    border=0>-NAD.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Ammonium sulfate 
      (NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB></STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 1 M 
      solution, dissolve 1.3 g of solid 
      (NH<SUB>4</SUB>)<SUB>2</SUB>SO<SUB>4</SUB> in a final volume of 10 ml of 
      H<SUB>2</SUB>O. Filter-sterilize the solution and store the solution at 
      room temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Bacteriophage T4 Polynucleotide Kinase 
      Buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>700 mM Tris-Cl (pH 
      7.6)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM 
      MgCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>50 mM 
      dithiothreitol</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Divide the 10x stock into 
      small aliquots and store frozen at -20°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>EDTA</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare EDTA at 0.5 M 
      (pH 8.0): Add 186.1 g of disodium EDTA•2H<SUB>2</SUB>O to 800 ml of 
      H<SUB>2</SUB>O. Stir vigorously on a magnetic stirrer. Adjust the pH to 
      8.0 with NaOH (approx. 20 g of NaOH pellets). Dispense into aliquots and 
      sterilize by autoclaving. The disodium salt of EDTA will not go into 
      solution until the pH of the solution is adjusted to approx. 8.0 by the 
      addition of NaOH.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>KCl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Dissolve an appropriate 
      amount of solid KCl in H<SUB>2</SUB>O, autoclave for 20 minutes on liquid 
      cycle and store at room temperature. Ideally, this 4 M solution should be 
      divided into small (approx. 100 µl) aliquots in sterile tubes and each 
      aliquot thereafter used one time.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>RNase H Buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>20 mM Tris-Cl (pH 
      7.6)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>20 mM KCl</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.1 mM EDTA (pH 
      8.0)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.1 mM 
      dithiothreitol</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Prepare fresh just before 
      use.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Sodium Acetate</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 3 M 
      solution: Dissolve 408.3 g of sodium acetate•3H<SUB>2</SUB>O in 800 ml of 
      H<SUB>2</SUB>O. Adjust the pH to 5.2 with glacial acetic acid or adjust 
      the pH to 7.0 with dilute acetic acid. Adjust the volume to 1 liter with 
      H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
    autoclaving.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>TE</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM Tris-Cl (desired 
      pH)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>10 mM EDTA (pH 
    8.0)</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>(10x Tris EDTA) Sterilize 
      solutions by autoclaving for 20 minutes at 15 psi (1.05 kg/cm 
      <SUP>2</SUP>) on liquid cycle. Store the buffer at room 
    temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Cl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve 121.1 g of Tris base in 800 ml of H<SUB>2</SUB>O. Adjust the 
      pH to the desired value by adding concentrated HCl.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2><I>pH</I></FONT> </TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2><I>HCl</I></FONT> 
        </TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.4 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>70 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.6 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>8.0 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>42 ml 
        </FONT></TD></TR></TBODY></TABLE>
      <P>(1 M) Allow the solution to cool to room temperature before making 
      final adjustments to the pH. Adjust the volume of the solution to 1 liter 
      with H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
      autoclaving.</P>
      <P>If the 1 M solution has a yellow color, discard it and obtain Tris of 
      better quality. The pH of Tris solutions is temperature-dependent and 
      decreases approx. 0.03 pH units for each 1°C increase in temperature. For 
      example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5°C, 
      25°C, and 37°C, respectively.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>dNTP Solution</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve each dNTP (deoxyribonucleoside triphosphates) in 
      H<SUB>2</SUB>O at an approximate concentration of 100 mM. Use 0.05 M Tris 
      base and a micropipette to adjust the pH of each of the solutions to 7.0 
      (use pH paper to check the pH). Dilute an aliquot of the neutralized dNTP 
      appropriately, and read the optical density at the wavelengths given in 
      the table below. Calculate the actual concentration of each dNTP. Dilute 
      the solutions with H<SUB>2</SUB>O to a final concentration of 50 mM dNTP. 
      Store each separately at -70°C in small aliquots. For polymerase chain 
      reactions (PCRs), adjust the dNTP solution to pH 8.0 with 2 N NaOH. 
      Commercially available solutions of PCR-grade dNTPs require no 
      adjustment.</P>
      <P>
      <TABLE width=400>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>Base</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana 
            size=2>wavelength<BR>(nm)</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Extinction 
            Coefficient<BR>(E)&nbsp;(M<SUP>-1</SUP>cm<SUP>-1</SUP>)</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>A</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>259</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.54 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>G</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>253</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.37 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>C</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>271</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.10 x 
            10<SUP>3</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>T</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>267</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.60 x 
            10<SUP>3</SUP></FONT></TD></TR></TBODY></TABLE></P>
      <P>For a cuvette with a path length of 1 cm, absorbance = EM. 100 mM stock 
      solutions of each dNTP are commercially available 
    (Pharmacia).</P></FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;     helvetica" size=3><STRONG>CAUTIONS</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" width=11 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Chloroform</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Chloroform CHCl<SUB>3</SUB> is irritating to the skin, eyes, 
      mucous membranes, and respiratory tract. It is a carcinogen and may damage 
      the liver and kidneys. It is also volatile. Avoid breathing the vapors. 
      Wear appropriate gloves and safety glasses. Always use in a chemical fume 
      hood. </FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Phenol:chloroform</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Phenol:chloroform, see Phenol; Chloroform 
</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Radioactive substances</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Radioactive substances: When planning an experiment that involves 
      the use of radioactivity, consider the physico-chemical properties of the 
      isotope (half-life, emission type, and energy), the chemical form of the 
      radioactivity, its radioactive concentration (specific activity), total 
      amount, and its chemical concentration. Order and use only as much as 
      needed. Always wear appropriate gloves, lab coat, and safety goggles when 
      handling radioactive material. X-rays and gamma rays are electromagnetic 
      waves of very short wavelengths either generated by technical devices or 
      emitted by radioactive materials. They might be emitted isotropically from 
      the source or may be focused into a beam. Their potential dangers depend 
      on the time period of exposure, the intensity experienced, and the 
      wavelengths used. Be aware that appropriate shielding is usually made of 
      lead or other similar material. The thickness of the shielding is 
      determined by the energy(s) of the X-rays or gamma rays. Consult the local 
      safety office for further guidance in the appropriate use and disposal of 
      radioactive materials. Always monitor thoroughly after using 
      radioisotopes. A convenient calculator to perform routine radioactivity 
      calculations can be found at: <A target=_blank 
      href="http://www.graphpad.com/calculators/radcalc.cfm">http://www.graphpad.com/calculators/radcalc.cfm</A>. 
      </FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>REFERENCES</STRONG> 
</FONT><BR>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Gubler U. 1987. 
  Second-strand cDNA synthesis: mRNA fragments as primers. <I>Methods 
  Enzymol.</I> 152:330-335. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Gubler U. and 
  Hoffman B.J. 1983. A simple and very efficient method for generating cDNA 
  libraries. <I>Gene</I> 25:263-269. </FONT></LI></OL><BR><BR><BR><BR><BR><FONT 
face=Arial,Verdana,Helvetica size=1><STRONG>Copyright © 2000 by Cold Spring 
Harbor Laboratory Press. All rights reserved.</STRONG><BR>No part of these 
pages, either text or image may be used for any purpose other than personal use. 
Therefore, reproduction modification, storage in a retrieval system or 
retransmission, in any form or by any means, electronic, mechanical, or 
otherwise, for reasons other than personal use, is strictly prohibited without 
prior written permission.</FONT><BR><BR>
</td></tr></table>
</BODY></HTML>
