<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Molecular Cloning | Protocol 14 Print Version</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1251"><!--================== Designed and Developed by WriteDesign =================--><!--======================= Powered by RomeBlack, Inc. =======================--><!--=== Copyright 2000, 2001 Cold Spring Harbor Labs. All Rights Reserved. ===-->
<META content="" name=keywords>
<META 
content="The goal of this method is to identify transcriptionally active genes in cloned segments of genomic DNA. The protocol uses hybridization and affinity purification to recover biotin-labeled cDNAs that bind to a 500-kb segment of human DNA cloned in a BAC vector. However, the method can be easily adapted to other clones of genomic DNAs cloned in high-capacity vectors. This protocol was provided by Michael Lovett (Morgan et al. 1992; Simmons and Lovett 1999)." 
name=description>
<META content="MSHTML 5.50.4134.600" name=GENERATOR></HEAD>
<BODY bgColor=#ffffff background=../izo/fon.gif>
<table width=600 border="0" cellspacing="0" cellpadding="0"><tr><td width=600><IMG height=49 hspace=4 src="../izo/logo_csh.gif" 
width=169 align=left vspace=4><IMG height=49 hspace=4 
src="../izo/print_header.gif" width=293 align=left vspace=4><BR 
clear=all><BR><FONT face="arial,verdana, helvetica" size=4><STRONG>Chapter 11, 
Protocol 14</STRONG></FONT><BR><BR clear=all><FONT face=arial,verdana,helvetica 
size=3><STRONG>Direct Selection of cDNAs with Large Genomic DNA 
Clones</STRONG></FONT><BR><BR><FONT face="arial,verdana, helvetica" size=2>The 
goal of this method is to identify transcriptionally active genes in cloned 
segments of genomic DNA. The protocol uses hybridization and affinity 
purification to recover biotin-labeled cDNAs that bind to a 500-kb segment of 
human DNA cloned in a BAC vector. However, the method can be easily adapted to 
other clones of genomic DNAs cloned in high-capacity vectors. This protocol was 
provided by Michael Lovett (Morgan et al. 1992; Simmons and Lovett 1999).</FONT> 
<BR><BR>
<TABLE border=1>
  <TBODY>
  <TR>
    <TD width=700><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" 
      width=11 border=0><FONT face=arial,verdana,helvetica 
      size=2><STRONG>CAUTION</STRONG></FONT><BR><IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0><FONT 
      face="arial,verdana, helvetica" size=2> <STRONG>RECIPE</STRONG></FONT> 
  </TD></TR></TBODY></TABLE><BR><BR><FONT face=arial,verdana,helvetica 
size=3><STRONG>MATERIALS</STRONG></FONT><BR><BR>
<TABLE cellPadding=0 cellborder="0">
  <TBODY>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Buffers and Solutions</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>10x 
      Amplification buffer</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Include 
      0.01% (w/v) gelatin in the buffer.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>ATP (10 
      mM)</FONT> <IMG height=11 alt=caution hspace=4 src="../izo/triangle.gif" 
      width=12 border=0><IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>2x 
      Hybridization solution</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>NaOH (0.1 
      M)</FONT> <IMG height=11 alt=caution hspace=4 src="../izo/triangle.gif" 
      width=12 border=0><IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>10x Nick 
      translation buffer</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>SDS 
      (10%)</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0><IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>20x SSC</FONT> 
      <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" width=12 
      border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Streptavidin 
      bead-binding buffer</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Tris-HCl (1 M, 
      pH 7.5)</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Enzymes and Buffers</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>DNA 
      polymerase/DNase I (5 units/µl; Boehringer Mannheim)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Restriction 
      endonucleases</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>T4 DNA 
      ligase</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Thermostable 
      DNA polymerase</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Nucleic Acids and Oligonucleotides</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>BAC DNA 
      encompassing the target genomic region</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Purify as 
      described in <A href="../04/08.htm">Chapter 
      4, Protocol 8</A> or <A href="../04/09.htm">Chapter 
      4, Protocol 9</A>.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Blunt-ended, 
      random-primed cDNAs</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Please see 
      <A href="01.htm">Chapter 
      11, Protocol 1</A>.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>C<I><SUB>0</SUB>t</I>1 genomic DNA</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Purchased 
      from Life Technologies.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Cytoplasmic 
      poly(A)<SUP>+</SUP> RNA, purified from tissue or cell line of 
      interest</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Please see 
      <A href="01.htm">Chapter 
      11, Protocol 1</A>.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>DNA size 
      markers</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>dNTP solution 
      for nick translation containing all four dNTPs, each at 0.4 mM (pH 
      8.0)</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>dNTP solution 
      for PCR containing all four dNTPs, each at 2.5 mM (pH 8.0)</FONT> <IMG 
      height=12 alt="" hspace=4 src="../izo/circle.gif" width=12 border=0> 
  </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Oligonucleotides (10 mM in TE [pH 8.0]):</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana 
size=2><I>Oligo3</I></FONT></TD>
          <TD width=5></TD>
          <TD><FONT face=arial,helvetica,verdana 
            size=2><I>5'-CTCGAGAATTCTGGATCCTC-3'</I></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana 
size=2><I>Oligo4</I></FONT></TD>
          <TD width=5></TD>
          <TD><FONT face=arial,helvetica,verdana 
            size=2><I>5'-GAGGATCCAGAATTCTCGAGTT-3'</I></FONT></TD></TR></TBODY></TABLE></FONT></TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Positive-control DNA</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>The 
      positive-control DNA should be a part of a gene or expressed sequence tag 
      (EST) known to be present in the starting segment of DNA. If no genes are 
      known, then a single-copy control DNA can be seeded into the genomic 
      target before labeling (at a 1:1 molar ratio) and diluted into the cDNA 
      (at a 1:10<SUP>6</SUP> molar ratio). Label the DNA sample using one of the 
      protocols given in Chapter 9; the choice of labeling protocol is 
      determined by the experimental goal.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Labeled Compounds</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>[<IMG 
      src="../izo/alpha.gif" border=0>-<SUP>32</SUP>P]dCTP 
      (3000Ci/mmole)</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Biotin-16-dUTP 
      (0.4 mmole)</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Additional Reagents</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 1 of this 
      protocol requires reagents listed in <A href="../10/02.htm">Chapter 
      10, Protocol 2</A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 2 of this 
      protocol requires reagents for the synthesis of random primed 
      double-stranded cDNAs or PCR-amplified inserts from a cDNA library listed 
      in <A href="01.htm">Chapter 
      11, Protocol 1</A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 2 of this 
      protocol requires reagents for labeling listed in <A href="../09/03.htm">Chapter 
      9, Protocol 3</A>, <A href="../09/05.htm">Chapter 
      9, Protocol 5</A>, and <A href="../09/06.htm">Chapter 
      9, Protocol 6</A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Steps 19 and 
      20 require reagents listed in <A href="../06/08.htm">Chapter 
      6, Protocol 8</A> and <A href="../06/10.htm">Chapter 
      6, Protocol 10</A>.</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 
  border=0></TD></TR></TBODY></TABLE><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" 
size=3><STRONG>METHOD</STRONG></FONT> <BR><FONT face="arial,verdana, helvetica" 
color=#000000 size=2>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Label 
  oligonucleotides 3 and 4 separately by phosphorylation at their 5' termini 
  using polynucleotide kinase, as described in <A href="../10/02.htm">Chapter 
  10, Protocol 2</A>. Mix the labeled complementary oligos in equal molar ratios 
  (approx. 2 µg of each), denature them for 10 minutes at 100°C, and slowly cool 
  them to room temperature. During this process, the oligonucleotides anneal to 
  form an adaptor. Adjust the concentration of the adaptor to 1 µg/ml. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Prepare at 
  least 2 µg of double-stranded cDNA from cytoplasmic polyadenylated RNA by 
  random priming (please see <A href="../11/01.htm">Chapter 
  11, Protocol 1</A>). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Prepare the 
  following ligation reaction in a sterile 0.5-ml microfuge tube: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>double-stranded 
        blunt-ended cDNA (Step 2)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
    µg</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>oligonucleotide 
        amplification cassette mixture at 1 µg/µl, from Step 1</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>3 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x T4 DNA 
        ligase buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>3 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10 mM 
      ATP</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>3 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>T4 DNA ligase 
        at 1 unit/µl</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>3 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 30 
      µl</FONT></TD></TR></TBODY></TABLE>Incubate the ligation for 16 hours at 14°C. 
  Inactivate the T4 DNA ligase by a 10-minute incubation at 65°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Purify the 
  products of the ligation reaction by phenol:chloroform extraction, by 
  spun-column chromatography through Sephadex G-50, and by standard ethanol 
  precipitation. Dry the pellet and resuspend it in 10 µl TE (pH 7.6). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Incorporate 
  biotinylated residues into the BAC genomic DNA clone using nick translation. 
  Prepare the following nick translation reaction in a sterile 0.5-ml microfuge 
  tube (label each BAC DNA separately): 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>purified BAC 
        DNA (0.1 mg/ml)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>biotin-16-dUTP 
        (0.04 mM)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x nick 
        translation buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>dNTP mix for 
        nick translation (0.4 mM)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>[<IMG 
        src="../izo/alpha.gif" border=0>-<SUP>32</SUP>P]dCTP (3000 
        Ci/mmole)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>DNA 
        polymerase/DNase I (5 units/µl)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 20 
      µl</FONT></TD></TR></TBODY></TABLE>Incubate the reaction for 2 hours at 4°C. 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Purify the 
  radiolabeled and biotinylated products of the nick translation reaction by 
  spun-column chromatography through Sephadex G-50 and standard ethanol 
  precipitation. Resuspend the pellet in 10 µl of TE and store it at -20°C. 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>In a 1.5-ml 
  sterile microfuge tube, wash 3 mg (300 µl) of beads with 500 µl of 
  streptavidin bead-binding buffer three times. Following each wash, remove the 
  beads from the binding buffer using a magnetic separator. Resuspend the beads 
  at a concentration of 10 mg/ml in streptavidin bead-binding buffer. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Test an aliquot 
  of each labeling reaction (from Step 6) for the ability to bind to 
  streptavidin beads. Prepare the following binding reaction in a sterile 0.5-ml 
  microfuge tube: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>washed 
        streptavidin-coated beads (Step 7)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>labeled genomic 
        DNA contig* (Step 6)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>streptavidin 
        bead-binding buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>29 
    µl</FONT></TD></TR></TBODY></TABLE><I>*Mix equimolar amounts of each 
  separately labeled BAC in the contig to yield a mixture at a concentration of 
  10 ng/µl.</I> <BR>Incubate the reaction for 15 minutes at room temperature 
  with occasional gentle mixing. Remove the beads from the binding buffer using 
  a magnetic separator, and transfer the supernatant to a fresh sterile 0.5-ml 
  microfuge tube. Use a Geiger counter or Cerenkov counter to measure the 
  radioactivity present on the beads and in the supernatant. If the ratio of 
  bound to free cpm is &gt;8:1, then proceed with the selection.<BR><BR><I>If 
  the ratio of bound to free cpm is &lt;8:1, it is likely that the DNA was 
  resistant to proper labeling in Step 5. Before labeling, try purifying the BAC 
  DNA further by several rounds of extraction with phenol:chloroform and passing 
  it through a Sephadex G-50 spun column.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Block or 
  "repeat suppress" repetitive sequences in the pool of cDNA (from Step 4) using 
  C<I><SUB>0</SUB>t</I>1 DNA as follows: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>a.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Prepare the 
        following annealing reaction in a sterile 0.5-ml microfuge 
      tube:</FONT></TD></TR>
    <TR>
      <TD colSpan=2>
        <TABLE>
          <TBODY>
          <TR>
            <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>cDNA 
              carrying linkers (Step 4)</FONT></TD>
            <TD width=5></FONT></TD>
            <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 µl (1 
              µg)</FONT></TD></TR>
          <TR>
            <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>human 
              genomic C<SUB>0</SUB><I>t</I>1 DNA</FONT></TD>
            <TD width=5></FONT></TD>
            <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 µl (1 
              µg)</FONT></TD></TR></TBODY></TABLE></FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>b.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Overlay the 
        reaction mixture with light mineral oil (approx. 50 µl) to prevent 
        evaporation, and denature the DNA by heating for 10 minutes at 100°C. 
        Cool the reaction mixture to 65°C, deliver 10 µl of 2x hybridization 
        solution under the oil. Mix the components gently. Incubate the reaction 
        mixture for 4 hours at 65°C.</FONT></TD></TR></TBODY></TABLE></FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>After 
  hybridization of the cDNA pools to the C<I><SUB>0</SUB>t</I>1 DNA is complete, 
  set up the primary direct selection. Deliver 5 µl (50 ng) of biotinylated BAC 
  DNAs from Step 6 into a fresh microfuge tube, and overlay the solution with 
  light mineral oil (approx. 50 µl ) to prevent evaporation. Denature the BAC 
  DNA by heating to 100°C for 10 minutes. Cool the reaction to 65°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Prepare the 
  following annealing reaction in a sterile 0.5-ml microfuge tube: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>biotinylated 
        BAC DNA from Step 10</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 µl (50 
        ng)</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>blocked cDNA 
        from Step 9 (1 µg cDNA plus 1 µg C<I><SUB>0</SUB>t</I>1 DNA)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2x 
        hybridization solution</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 
    µl</FONT></TD></TR></TBODY></TABLE>Mix the reagents gently, and incubate the 
  reaction for &gt;54 hours at 65°C in a rotating hybridization oven or shaking 
  water bath. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>To capture and 
  wash the genomic DNA and hybridized cDNAs, prepare the following in a sterile 
  1.5-ml microfuge tube: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>washed 
        streptavidin-coated beads</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>100 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>annealing 
        reaction from Step 11</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>30 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>streptavidin 
        bead-binding buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>100 
    µl</FONT></TD></TR></TBODY></TABLE>Incubate the mixture for 15 minutes at room 
  temperature with occasional gentle mixing. Remove the beads from the binding 
  buffer using a magnetic separator, and then remove and discard the 
  supernatant. Wash the beads twice, for 15 minutes each time, in 1 ml of 1x 
  SSC/0.1% SDS at room temperature followed by three washes, 15 minutes each, in 
  1 ml of 0.1x SSC/0.1% SDS at 65°C. After the final wash, transfer the beads to 
  a fresh microfuge tube. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>To elute the 
  hybridizing cDNAs from the beads in Step 12: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>a.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Add 100 µl of 
        0.1 M NaOH, and incubate the reaction mixture for 10 minutes at room 
        temperature.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>b.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Add 100 µl of 1 
        M Tris-Cl (pH 7.5).</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>c.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Desalt the 
        mixture by spun-column chromatography through Sephadex G-50 (please see 
        Appendix 8 in the print version of the 
  manual).</FONT></TD></TR></TBODY></TABLE></FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Transfer three 
  aliquots (1 µl, 5 µl, and 10 µl, respectively) from the 200 µl of eluted cDNA 
  (Step 13) to 0.5-ml sterile amplification tubes. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>To each tube, 
  add the following: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>primer oligo3 
        (10 mM)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x 
        amplification buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2.5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>dNTP mixture 
        for PCR (2.5 mM)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2.5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2><I>Taq</I> DNA 
        polymerase (Perkin Elmer 5 units/µl)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>0.2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 25 
      µl</FONT></TD></TR></TBODY></TABLE>Set up two control reactions at the same 
  time as the above test reactions. For the negative control, include all of the 
  components listed above, but omit the template cDNA. For the second control, 
  include 10 ng of the starting cDNA (Step 4). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>If the thermal 
  cycler is not fitted with a heated lid, overlay the reaction mixtures with 1 
  drop (approx. 50 µl) of light mineral oil to prevent evaporation. 
  Alternatively, place a bead of wax into the tube if using a hot start PCR. 
  Place the tubes in the thermocycler. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Amplify the 
  nucleic acids using the denaturation, annealing, and polymerization times and 
  temperatures listed below. 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2><B>Cycle 
        Number</B></FONT></TD>
      <TD width=4></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Denaturation</B></FONT></TD>
      <TD width=4></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Annealing</B></FONT></TD>
      <TD width=4></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Polymerization</B></FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>30 
      cycles</FONT></TD>
      <TD width=4></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>30 sec at 
        94°C</FONT></TD>
      <TD width=4></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>30 sec at 
        55°C</FONT></TD>
      <TD width=4></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 min at 
        72°C</FONT></TD></TR></TBODY></TABLE><I>Times and temperatures may need to be 
  adapted to suit the particular reaction conditions.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Analyze 10% of 
  each amplification reaction on a 1% agarose gel cast and run in 0.5x TBE, 
  including DNA markers of an appropriate size. Stain the gel with ethidium 
  bromide or SYBR Gold to visualize the DNAs, estimate the concentration of the 
  amplified cDNA, and determine which input cDNA concentration produces the 
  highest yield of cDNA. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Onto a second 
  1% agarose gel, load the same amounts (approx. 0.5 µg per lane ) of the 
  amplifiied products of each PCR, as well as the appropriate size markers. Also 
  load 0.5 µg of randomly primed cDNA from Step 4. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Transfer the 
  separated DNA species to a membrane by Southern blotting (<A href="../06/08.htm">Chapter 
  6, Protocol 8</A>) and hybridize with the radiolabeled, positive control cDNA. 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Once the 
  positive control enrichment is confirmed, scale up the optimal amplification 
  reaction to yield at least 1.5 µg of selected cDNAs. Extract the pooled 
  reactions with phenol:chloroform and recover the DNA by standard ethanol 
  precipitation. Resuspend the dried cDNA pellet in 7.5 µl of TE (200 
  µg/ml).<BR>Proceed to carry out the secondary selection under the same 
  conditions as the primary selection, using 1 µg of the primary selected cDNA 
  and 50 ng of the target DNA (total length 500 kb in this example). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Block 
  repetitive sequences in the primary selected cDNA as described in Step 9 (with 
  1 µg of cDNA being used and 0.5 µg of cDNA being held in reserve for later 
  analysis). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Set up the 
  secondary selection as described in Steps 9 through 13. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Analyze the 
  products of the second amplification by electrophoresis through a 1% agarose 
  gel as in Step 18, including a lane with 0.5 µg of primary cDNA (held in 
  reserve) and a lane with 0.5 µg of the starting cDNA. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Analyze the gel 
  by Southern blotting and hybridization with the radiolabeled reporter probe as 
  described in Step 20. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Once enrichment 
  of the control is confirmed, clone the selected cDNAs into the appropriate 
  vector(s). The restriction enzyme sites in the amplification cassette 
  facilitate the cloning of the secondary selected cDNAs into bacteriophage or 
  plasmid vectors. </FONT></LI></OL></FONT><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>RECIPES</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/circle.gif" width=12 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>10x Nick translation buffer</STRONG></FONT> 
    </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>500 mM Tris-HCl (pH 7.5) 
      </FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM 
      MgCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>50 mM 
      dithiothreitol</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>2x Hybridization solution</STRONG></FONT> 
  </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>1.5 M NaCl</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>40 mM sodium phosphate 
      buffer (pH 7.2) </FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>10 mM EDTA (pH 8.0) 
      </FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>10x Denhardt's solution 
      </FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.2% 
  SDS</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>ATP</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve an appropriate amount of solid ATP in 25 mM Tris-Cl (pH 
      8.0).</P>
      <P>Store the 10 mM ATP solution in small aliquots at 
  -20°C.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Amplification Buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>500 mM KCl</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM Tris-Cl (pH 8.3 at 
      room temperature)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>15 mM 
      MgCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Autoclave the 10x buffer 
      for 10 minutes at 15 psi (1.05 kg/cm <SUP>2</SUP>) on liquid cycle. Divide 
      the sterile buffer into aliquots and store them at 
  -20°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Denhardt's Reagent</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>1% (w/v) Ficoll 
    400</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>1% (w/v) 
      polyvinylpyrrolidone</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>1% (w/v) bovine serum 
      albumin (Sigma, Fraction V) in H<SUB>2</SUB>O</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Denhardt's reagent is 
      usually made up as a 50x stock solution, which is filtered and stored at 
      -20°C. The stock solution is diluted tenfold into prehybridization buffer 
      (usually 6x SSC or 6x SSPE containing 1.0% SDS and 100 µ;g/ml denatured 
      salmon sperm DNA). 50x Denhardt's reagent contains in 
      H<SUB>2</SUB>O.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>EDTA</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare EDTA at 0.5 M 
      (pH 8.0): Add 186.1 g of disodium EDTA•2H<SUB>2</SUB>O to 800 ml of 
      H<SUB>2</SUB>O. Stir vigorously on a magnetic stirrer. Adjust the pH to 
      8.0 with NaOH (approx. 20 g of NaOH pellets). Dispense into aliquots and 
      sterilize by autoclaving. The disodium salt of EDTA will not go into 
      solution until the pH of the solution is adjusted to approx. 8.0 by the 
      addition of NaOH.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Ficoll 400 (20% w/v)</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Dissolve the Ficoll in 
      sterile H<SUB>2</SUB>O and store the solution frozen in 100-µl aliquots at 
      -20°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>KCl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Dissolve an appropriate 
      amount of solid KCl in H<SUB>2</SUB>O, autoclave for 20 minutes on liquid 
      cycle and store at room temperature. Ideally, this 4 M solution should be 
      divided into small (approx. 100 µl) aliquots in sterile tubes and each 
      aliquot thereafter used one time.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>NaCl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 5 M 
      solution: Dissolve 292 g of NaCl in 800 ml of H<SUB>2</SUB>O. Adjust the 
      volume to 1 liter with H<SUB>2</SUB>O. Dispense into aliquots and 
      sterilize by autoclaving. Store the NaCl solution at room 
      temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>NaOH</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>The preparation of 10 N 
      NaOH involves a highly exothermic reaction, which can cause breakage of 
      glass containers. Prepare this solution with extreme care in plastic 
      beakers. To 800 ml of H<SUB>2</SUB>O, slowly add 400g of NaOH pellets, 
      stirring continuously. As an added precaution, place the beaker on ice. 
      When the pellets have dissolved completely, adjust the volume to 1 liter 
      with H<SUB>2</SUB>O. Store the solution in a plastic container at room 
      temperature. Sterilization is not 
necessary.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Phosphate Buffers</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Gomori buffers, the most commonly used phosphate buffers, consist of a 
      mixture of monobasic dihydrogen phosphate and dibasic monohydrogen 
      phosphate. By varying the amount of each salt, a range of buffers can be 
      prepared that buffer well between pH 5.8 and pH 8.0 (please see the tables 
      below). Phosphates have a very high buffering capacity and are highly 
      soluble in water. However, they have a number of potential 
      disadvantages:</P>
      <P>
      <TABLE>
        <TBODY>
        <TR>
          <TD>•</TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Phosphates inhibit 
            many enzymatic reactions and procedures that are the foundation of 
            molecular cloning, including cleavage of DNA by many restriction 
            enzymes, ligation of DNA, and bacterial transformation.</FONT></TD></TR>
        <TR>
          <TD>•</TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Because phosphates 
            precipitate in ethanol, it is not possible to precipitate DNA and 
            RNA from buffers that contain significant quantities of phosphate 
            ions.</FONT> </TD></TR>
        <TR>
          <TD>•</TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Phosphates sequester 
            divalent cations such as Ca<SUP>2+</SUP> and Mg<SUP>2+</SUP>.</FONT> 
          </TD></TR></TBODY></TABLE>
      <TABLE>
        <TBODY>
        <TR>
          <TD><IMG height=412 src="../izo/phos_a.gif" width=339 
        border=0></TD></TR>
        <TR>
          <TD><IMG height=414 src="../izo/phos_b.gif" width=338 
        border=0></TD></TR></TBODY></TABLE></FONT></P></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>SDS</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Also called sodium lauryl 
      sulfate. To prepare a 20% (w/v) solution, dissolve 200 g of 
      electrophoresis-grade SDS in 900 ml of H<SUB>2</SUB>O. Heat to 68°C and 
      stir with a magnetic stirrer to assist dissolution. If necessary, adjust 
      the pH to 7.2 by adding a few drops of concentrated HCl. Adjust the volume 
      to 1 liter with H<SUB>2</SUB>O. Store at room temperature. Sterilization 
      is not necessary. Do not autoclave.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>SSC</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>For 20x solution: 
      Dissolve 175.3 g of NaCl and 88.2 g of sodium citrate in 800 ml of 
      H<SUB>2</SUB>O. Adjust the pH to 7.0 with a few drops of a 14 N solution 
      of HCl. Adjust the volume to 1 liter with H<SUB>2</SUB>O. Dispense into 
      aliquots. Sterilize by autoclaving. The final concentrations of the 
      ingredients are 3.0 M NaCl and 0.3 M sodium 
acetate.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Sodium Phosphate</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Phosphate 
    Buffers</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Streptavidin bead-binding 
      buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>10 mM Tris-HCl (pH 7.5) 
      </FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>1 mM EDTA (pH 8.0) 
    </FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>1 M NaCl 
  </FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Cl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve 121.1 g of Tris base in 800 ml of H<SUB>2</SUB>O. Adjust the 
      pH to the desired value by adding concentrated HCl.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2><I>pH</I></FONT> </TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2><I>HCl</I></FONT> 
        </TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.4 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>70 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.6 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>8.0 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>42 ml 
        </FONT></TD></TR></TBODY></TABLE>
      <P>(1 M) Allow the solution to cool to room temperature before making 
      final adjustments to the pH. Adjust the volume of the solution to 1 liter 
      with H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
      autoclaving.</P>
      <P>If the 1 M solution has a yellow color, discard it and obtain Tris of 
      better quality. The pH of Tris solutions is temperature-dependent and 
      decreases approx. 0.03 pH units for each 1°C increase in temperature. For 
      example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5°C, 
      25°C, and 37°C, respectively.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>dNTP Solution</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve each dNTP (deoxyribonucleoside triphosphates) in 
      H<SUB>2</SUB>O at an approximate concentration of 100 mM. Use 0.05 M Tris 
      base and a micropipette to adjust the pH of each of the solutions to 7.0 
      (use pH paper to check the pH). Dilute an aliquot of the neutralized dNTP 
      appropriately, and read the optical density at the wavelengths given in 
      the table below. Calculate the actual concentration of each dNTP. Dilute 
      the solutions with H<SUB>2</SUB>O to a final concentration of 50 mM dNTP. 
      Store each separately at -70°C in small aliquots. For polymerase chain 
      reactions (PCRs), adjust the dNTP solution to pH 8.0 with 2 N NaOH. 
      Commercially available solutions of PCR-grade dNTPs require no 
      adjustment.</P>
      <P>
      <TABLE width=400>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>Base</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana 
            size=2>wavelength<BR>(nm)</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Extinction 
            Coefficient<BR>(E)&nbsp;(M<SUP>-1</SUP>cm<SUP>-1</SUP>)</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>A</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>259</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.54 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>G</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>253</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.37 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>C</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>271</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.10 x 
            10<SUP>3</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>T</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>267</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.60 x 
            10<SUP>3</SUP></FONT></TD></TR></TBODY></TABLE></P>
      <P>For a cuvette with a path length of 1 cm, absorbance = EM. 100 mM stock 
      solutions of each dNTP are commercially available 
    (Pharmacia).</P></FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;     helvetica" size=3><STRONG>CAUTIONS</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" width=11 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>NaOH</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">NaOH, see Sodium hydroxide </FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Radioactive substances</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Radioactive substances: When planning an experiment that involves 
      the use of radioactivity, consider the physico-chemical properties of the 
      isotope (half-life, emission type, and energy), the chemical form of the 
      radioactivity, its radioactive concentration (specific activity), total 
      amount, and its chemical concentration. Order and use only as much as 
      needed. Always wear appropriate gloves, lab coat, and safety goggles when 
      handling radioactive material. X-rays and gamma rays are electromagnetic 
      waves of very short wavelengths either generated by technical devices or 
      emitted by radioactive materials. They might be emitted isotropically from 
      the source or may be focused into a beam. Their potential dangers depend 
      on the time period of exposure, the intensity experienced, and the 
      wavelengths used. Be aware that appropriate shielding is usually made of 
      lead or other similar material. The thickness of the shielding is 
      determined by the energy(s) of the X-rays or gamma rays. Consult the local 
      safety office for further guidance in the appropriate use and disposal of 
      radioactive materials. Always monitor thoroughly after using 
      radioisotopes. A convenient calculator to perform routine radioactivity 
      calculations can be found at: <A target=_blank 
      href="http://www.graphpad.com/calculators/radcalc.cfm">http://www.graphpad.com/calculators/radcalc.cfm</A>. 
      </FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>SDS</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">SDS (Sodium dodecyl sulfate) is toxic, an irritant, and poses a 
      risk of severe damage to the eyes. It may be harmful by inhalation, 
      ingestion, or skin absorption. Wear appropriate gloves and safety goggles. 
      Do not breathe the dust. </FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>REFERENCES</STRONG> 
</FONT><BR>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Morgan J.G., 
  Dolganov M., Robbins S.E., Hinton L.M., and Lovett M. 1992. The selective 
  isolation of novel cDNAs encoded by the regions surrounding the human 
  interleukin 4 and 5 gene. <I>Nucleic Acids Res.</I> 20:5173-5179. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Simmons A.D. 
  and Lovett M. 1999. Direct cDNA selection using large genomic DNA targets. 
  <I>Methods Enzymol</I> 303:111-126. </FONT></LI></OL><BR><BR><BR><BR><BR><FONT 
face=Arial,Verdana,Helvetica size=1><STRONG>Copyright © 2000 by Cold Spring 
Harbor Laboratory Press. All rights reserved.</STRONG><BR>No part of these 
pages, either text or image may be used for any purpose other than personal use. 
Therefore, reproduction modification, storage in a retrieval system or 
retransmission, in any form or by any means, electronic, mechanical, or 
otherwise, for reasons other than personal use, is strictly prohibited without 
prior written permission.</FONT><BR><BR>
</td></tr></table>
</BODY></HTML>
