<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Molecular Cloning | Protocol 10 Print Version</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1251"><!--================== Designed and Developed by WriteDesign =================--><!--======================= Powered by RomeBlack, Inc. =======================--><!--=== Copyright 2000, 2001 Cold Spring Harbor Labs. All Rights Reserved. ===-->
<META content="" name=keywords>
<META 
content="In this method, the nuclease BAL 31 is used to make uni- or bidirectional deletions in a segment of cloned DNA. BAL 31 is a complex enzyme and tends to digest a population of double-stranded DNA targets in an asynchronous fashion, Deletions created by BAL 31 are therefore far more heterogeneous in size than those created by processive enzymes such as exonuclease III. Warning! BAL 31 is a tricky enzyme. This protocol is not for the technically challenged! Please see the information panel on BAL31 on page 13.68 in the print version of the manual." 
name=description>
<META content="MSHTML 5.50.4134.600" name=GENERATOR></HEAD>
<BODY bgColor=#ffffff background=../izo/fon.gif>
<table width=600 border="0" cellspacing="0" cellpadding="0"><tr><td width=600><IMG height=49 hspace=4 src="../izo/logo_csh.gif" 
width=169 align=left vspace=4><IMG height=49 hspace=4 
src="../izo/print_header.gif" width=293 align=left vspace=4><BR 
clear=all><BR><FONT face="arial,verdana, helvetica" size=4><STRONG>Chapter 13, 
Protocol 10</STRONG></FONT><BR><BR clear=all><FONT face=arial,verdana,helvetica 
size=3><STRONG>Generation of Bidirectional Sets of Deletion Mutants by Digestion 
with BAL 31 Nuclease</STRONG></FONT><BR><BR><FONT 
face="arial,verdana, helvetica" size=2>In this method, the nuclease BAL 31 is 
used to make uni- or bidirectional deletions in a segment of cloned DNA. BAL 31 
is a complex enzyme and tends to digest a population of double-stranded DNA 
targets in an asynchronous fashion, Deletions created by BAL 31 are therefore 
far more heterogeneous in size than those created by processive enzymes such as 
exonuclease III. <B>Warning!</B> BAL 31 is a tricky enzyme. This protocol is not 
for the technically challenged! Please see the information panel on BAL31 on 
page 13.68 in the print version of the manual.</FONT> <BR><BR>
<TABLE border=1>
  <TBODY>
  <TR>
    <TD width=700><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" 
      width=11 border=0><FONT face=arial,verdana,helvetica 
      size=2><STRONG>CAUTION</STRONG></FONT><BR><IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0><FONT 
      face="arial,verdana, helvetica" size=2> <STRONG>RECIPE</STRONG></FONT> 
  </TD></TR></TBODY></TABLE><BR><BR><FONT face=arial,verdana,helvetica 
size=3><STRONG>MATERIALS</STRONG></FONT><BR><BR>
<TABLE cellPadding=0 cellborder="0">
  <TBODY>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Buffers and Solutions</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>5x BAL 31 
      buffer (13-10)</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>dNTP solution 
      of all four dNTPs, each at 0.5 mM</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>EGTA (0.5 M, 
      pH 8.0)</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Ethanol</FONT> 
    </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>6x Gel-loading 
      buffer</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Phenol:chloroform (1:1, v/v)</FONT> <IMG height=11 alt=caution 
      hspace=4 src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Sodium acetate 
      (3 M, pH 5.2)</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>TE (pH 
      7.6)</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Enzymes and Buffers</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Bacteriophage 
      T4 DNA polymerase</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>BAL 31 
      nuclease</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Klenow 
      fragment of <I>E. coli</I> DNA polymerase I</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Restriction 
      endonucleases</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Please see 
      Steps 3, 23, and 31.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Nucleic Acids and Oligonucleotides</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Marker DNA for 
      gel electrophoresis</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Additional Reagents</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 1 of this 
      protocol requires the reagents listed in <A href="../01/17.htm">Chapter 
      1, Protocol 17</A> to <A href="../01/19.htm">Chapter 
      1, Protocol 19</A>, or the reagents listed in <A href="../03/06.htm">Chapter 
      3, Protocol 6</A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 2 of this 
      protocol requires the reagents listed in <A href="../01/09.htm">Chapter 
      1, Protocol 9</A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 27 of 
      this protocol requires the reagents listed in <A href="../05/04.htm">Chapter 
      5, Protocol 4</A> to <A href="../05/07.htm">Chapter 
      5, Protocol 7</A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 30 of 
      this protocol requires the reagents listed in <A href="../01/23.htm">Chapter 
      1, Protocol 23</A> to <A href="../01/26.htm">Chapter 
      1, Protocol 26</A>, or the reagents listed in <A href="../03/06.htm">Chapter 
      3, Protocol 6</A> or <A href="../03/08.htm">Chapter 
      3, Protocol 8</A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 31 of 
      this protocol requires the reagents listed in <A href="../03/01.htm">Chapter 
      1, Protocol 1</A>, or the reagents listed in <A href="../03/03.htm">Chapter 
      3, Protocol 3</A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 34 of 
      this protocol requires the reagents listed in <A href="../12/03.htm">Chapter 
      12, Protocol 3</A>, <A href="../12/04.htm">Chapter 
      12, Protocol 4</A>, or <A href="../12/05.htm">Chapter 
      12, Protocol 5</A>.</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 
  border=0></TD></TR></TBODY></TABLE><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" 
size=3><STRONG>METHOD</STRONG></FONT> <BR><FONT face="arial,verdana, helvetica" 
color=#000000 size=2>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Clone the 
  target fragment into an appropriate plasmid or bacteriophage M13 
  vector.<BR><I>If deletion mutants are to be constructed from both termini of 
  the target DNA, it will be necessary to clone the parental target DNA in both 
  orientations with respect to the polycloning site in an appropriate 
  vector.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Purify the 
  closed circular recombinant DNA(s) by column chromatography on Qiagen columns 
  (or their equivalent) and precipitation with ethanol. Redissolve the DNA in 
  the smallest practical volume of Tris/EDTA.<BR><I>It is essential to use 
  highly purified closed circular DNA (i) to minimize the contribution of 
  contaminating RNA and small fragments of </I>E. coli<I> chromosomal DNA to the 
  total concentration of termini in the reaction and (ii) to eliminate nicked 
  circular molecules, which are degraded by BAL 31 from the site of the 
  nick.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Digest 30 µg of 
  the closed circular DNA to completion with a restriction endonuclease that 
  cleaves at one end of the target DNA. This site defines the common point from 
  which the nested deletions will begin. Use agarose gel electrophoresis to 
  verify that digestion with the restriction endonuclease is complete. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Purify the DNA 
  by extraction with an equal volume of phenol:chloroform. Separate the aqueous 
  and organic phases by centrifugation at maximum speed for 3 minutes at 0°C in 
  a microfuge, and then transfer the aqueous phase to a fresh microfuge tube. 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Add 0.1 volume 
  of 3 M sodium acetate (pH 5.2) and 2 volumes of ice-cold ethanol. Store the 
  tube for 10 minutes at 0°C, and then recover the DNA by centrifugation at 
  maximum speed for 10 minutes at 4°C in a microfuge. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Remove the 
  supernatant, and wash the pellet of DNA carefully with 70% ethanol at room 
  temperature. Dry the pellet at room temperature, and dissolve it in TE (pH 
  7.6) at a concentration of 1 µg/µl. Store the DNA at -20°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>In a microfuge 
  tube, mix: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>linearized DNA 
        (1 µg/µl)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>4 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>48 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5x BAL 31 
        buffer (13-10)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>13 
    µl</FONT></TD></TR></TBODY></TABLE>Dispense 9 µl of this mixture into each of 
  seven separate microfuge tubes.<BR><I>Because the ratio of fast and slow forms 
  of BAL 31 varies from preparation to preparation, it is essential to assay the 
  activity of the particular batch of enzyme that will be used to generate 
  deletions.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Make a series 
  of seven twofold dilutions of BAL 31 in 1x BAL 31 buffer (13-10). Enzyme 
  dilution is best carried out by placing seven aliquots (2 µl) of 1x BAL 31 
  buffer (13-10) on the surface of a piece of Parafilm lying on a bed of ice or 
  on a cold block. Use a disposable micropipette tip to mix 2 µl of the BAL 31 
  preparation under test with the first drop. Use a fresh tip to transfer 2 µl 
  of the mixture to the next drop, and again mix. Continue in this fashion until 
  the enzyme has been added to all of the drops. Working quickly, add 1 µl of 
  each of the last six dilutions to six of the microfuge tubes containing the 
  linear DNA being tested. Do not add enzyme to the seventh tube. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Incubate all of 
  the microfuge tubes (including the tube that received no enzyme) for 30 
  minutes at 30°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Add 1 µl of 200 
  mM EGTA (pH 8.0) to each tube, and then heat the tubes for 5 minutes at 65°C. 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Mix each of the 
  samples with 3 µl of agarose gel-loading buffer and analyze the size of the 
  DNAs by electrophoresis through a 0.8% agarose gel containing 0.5 µg/ml 
  ethidium bromide. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Examine the gel 
  by UV illumination, and determine the dilution of enzyme just sufficient to 
  digest the DNA to the point where only a smear of small (200-bp) fragments is 
  detectable. This dilution of BAL 31 will be used in the large-scale digestion 
  (Step 15). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Mix: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>linearized DNA 
        (1 µg/µl)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>240 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5x BAL 31 
        buffer (13-10)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>65 
    µl</FONT></TD></TR></TBODY></TABLE>Incubate the mixture in a water bath at 
  30°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>While the 
  mixture is warming to 30°C, prepare a set of eight microfuge tubes, each 
  containing 5 µl of 200 mM EGTA (pH 8.0). Label the tubes 1.5 minutes, 3.0 
  minutes, 4.5 minutes, etc. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Add 36 µl of 
  the appropriate dilution of BAL 31 (please see Step 12) to the reaction 
  mixture prepared in Step 13. Quickly mix the enzyme by tapping the side of the 
  tube, and then return the tube to the water bath set at 30°C and start a 
  stopwatch. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>At 1.5-minute 
  intervals, transfer 45 µl of the reaction mixture to the appropriately labeled 
  microfuge tube. Store the tubes on ice until all of the samples have been 
  collected. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Heat the tubes 
  for 5 minutes at 65°C to inactivate the BAL 31. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Add 5 µl of 3 M 
  sodium acetate (pH 5.2) to each tube, followed by 100 µl of ice-cold ethanol. 
  Mix the solution by vortexing, and store the tubes on ice for 20-30 minutes. 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Recover the 
  DNAs by centrifugation at maximum speed for 10 minutes at 4°C. Remove the 
  supernatants, and wash the pellets with 200 µl of ice-cold 70% ethanol. 
  Centrifuge for a further 2 minutes. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Carefully 
  remove the supernatants, and stand the open tubes at room temperature until 
  all of the ethanol has evaporated. Dissolve each of the pellets in 23 µl of TE 
  (pH 7.6). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Add to each of 
  the DNA preparations: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>0.5 mM dNTP 
        solution</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>3 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x polymerase 
        buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>3 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>bacteriophage 
        T4 DNA polymerase (approx. 5 units)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR></TBODY></TABLE>Incubate the reactions for 15 minutes at 
  room temperature, and then add approx. 1 µl (approx. 5 units) of the Klenow 
  fragment. Continue the incubation for a further 15 minutes at room 
  temperature. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Purify the DNAs 
  by extraction with phenol:chloroform, and then precipitate the DNAs with 
  ethanol as described in Steps 18-20. Dissolve each of the DNAs in 16 µl of TE 
  (pH 7.6). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>To each DNA, 
  add 2 µl of the appropriate 10x restriction enzyme buffer and 8 units of a 
  restriction enzyme that will separate the target DNA from the vector. Incubate 
  the reactions for 1 hour at the appropriate temperature. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>At the end of 
  the incubation, transfer an aliquot (3 µl) from each digest to a fresh 
  microfuge tube. Store the remainder of the digests on ice until needed in Step 
  27. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Add 1 µl of 
  sucrose gel-loading buffer to each 3-µl aliquot, and load the contents of each 
  tube into the wells of an agarose gel cast in 0.5x TBE and containing 0.5 
  µg/ml ethidium bromide. The wells at the sides of the gel should contain 
  markers of the appropriate size. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Separate the 
  target fragments from the vector DNA by electrophoresis. Examine the gel by UV 
  illumination, and determine which of the samples has been digested to an 
  appropriate size by BAL 31. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Pool the 
  samples (from Step 24) containing target DNA of the appropriate size, and 
  isolate the target fragments by preparative gel electrophoresis. Recover the 
  target DNA fragments from the gel using one of the methods described in <A href="../05/04.htm">Chapter 
  5, Protocol 4</A> to <A href="../05/07.htm">Chapter 
  5, Protocol 7</A>. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Estimate the 
  amount of purified target DNA from the intensity of ethidium-bromide-mediated 
  fluorescence. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Ligate the 
  deleted target fragments with a plasmid, phagemid, or bacteriophage M13 vector 
  (please see Chapter 1 or Chapter 3) that carries one blunt end and one 
  terminus that is compatible with the restriction enzyme used in Step 23. 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Transform 
  (plasmids or phagemids) or transfect (bacteriophage M13 replicative form DNA) 
  competent <I>E. coli</I> of an appropriate strain with small aliquots or 
  dilutions of the ligation mixture. The next day, grow small-scale cultures of 
  12 transformed colonies or bacteriophage M13 plaques, chosen at random. 
</FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Purify plasmid, 
  phagemid, or bacteriophage M13 replicative form DNA from each of the 12 
  cultures by using one of the methods described in Chapter 1 or Chapter 3. 
  Digest the DNAs with a restriction enzyme(s) that will liberate the target 
  fragment from the vector. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Analyze the 
  size of the target fragment liberated from each of the DNAs by agarose gel 
  electrophoresis, using size markers of an appropriate size. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>If the results 
  are satisfactory (i.e., if the target fragments fall within the desired size 
  range), pick a large number of individual transformed colonies or plaques and 
  determine the size of the inserts as described above. Preserve those cultures 
  that carry recombinants of the desired size. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Determine the 
  exact endpoints of the deletion in each mutant by DNA sequencing (please see 
  <A href="../12/03.htm">Chapter 
  12, Protocol 3</A>, <A href="../12/04.htm">Chapter 
  12, Protocol 4</A>, or <A href="../12/05.htm">Chapter 
  12, Protocol 5</A>). </FONT></LI></OL></FONT><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>RECIPES</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/circle.gif" width=12 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>6x Gel-loading Buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Alkaline Gel-loading 
      Buffer</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>6x Gel-loading Buffer 
      I</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>6x Gel-loading Buffer 
      II</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>6x Gel-loading Buffer 
      III</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>6x Gel-loading Buffer 
      IV</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>6x Gel-loading Buffer I</STRONG></FONT> 
</TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.25% (w/v) bromophenol 
      blue</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.25% (w/v) xylene cyanol 
      FF</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>40% (w/v) sucrose in 
      H<SUB>2</SUB>O</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Store at 
  4°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>6x Gel-loading Buffer II</STRONG></FONT> 
  </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.25% (w/v) bromophenol 
      blue</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.25% (w/v) xylene cyanol 
      FF</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>15% (w/v) Ficoll (Type 
      400; Pharmacia) in H<SUB>2</SUB>O</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Store at room 
      temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>6x Gel-loading Buffer III</STRONG></FONT> 
  </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.25% (w/v) bromophenol 
      blue</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.25% (w/v) xylene cyanol 
      FF</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>30% (v/v) glycerol in 
      H<SUB>2</SUB>O</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Store at 
  4°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>6x Gel-loading Buffer IV</STRONG></FONT> 
  </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.25% (w/v) bromophenol 
      blue</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>40% (w/v) sucrose in 
      H<SUB>2</SUB>O</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Store at 
  4°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Alkaline Gel-loading Buffer</STRONG></FONT> 
    </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>300 mM NaOH</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>6 mM EDTA</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>18% (w/v) Ficoll (Type 
      400, Pharmacia)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.15% (w/v) bromocresol 
      green</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.25% (w/v) xylene 
      cyanol</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>For a 6x 
  buffer.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>BAL 31 buffer (13-10)</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>2.5 M NaCl</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>62.5 mM 
      CaCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>62.5 mM 
      MgCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM Tris-Cl (pH 
      8.0)</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>For a 5x 
  buffer.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>CaCl<SUB>2</SUB></STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Dissolve 11 g of 
      CaCl<SUB>2</SUB>•6H<SUB>2</SUB>O in a final volume of 20 ml of distilled 
      H<SUB>2</SUB>O. Sterilize the 2.5 M solution by passing it through a 
      0.22-µm filter. Store in 1-ml aliquots at 
4°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>EDTA</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare EDTA at 0.5 M 
      (pH 8.0): Add 186.1 g of disodium EDTA•2H<SUB>2</SUB>O to 800 ml of 
      H<SUB>2</SUB>O. Stir vigorously on a magnetic stirrer. Adjust the pH to 
      8.0 with NaOH (approx. 20 g of NaOH pellets). Dispense into aliquots and 
      sterilize by autoclaving. The disodium salt of EDTA will not go into 
      solution until the pH of the solution is adjusted to approx. 8.0 by the 
      addition of NaOH.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>EGTA</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare EGTA at 0.5 M 
      (pH 8.0): EGTA is ethylene glycol bis(<IMG src="../izo/beta.gif" 
      border=0>-aminoethyl ether) <I>N,N,N',N'</I>-tetraacetic acid. A solution 
      of EGTA is made up essentially as described for EDTA above and sterilized 
      by either autoclaving or filtering. Store the sterile solution at room 
      temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Ficoll 400 (20% w/v)</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Dissolve the Ficoll in 
      sterile H<SUB>2</SUB>O and store the solution frozen in 100-µl aliquots at 
      -20°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Glycerol</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 10% (v/v) 
      solution: Dilute 1 volume of molecular-biology-grade glycerol in 9 volumes 
      of sterile pure H<SUB>2</SUB>O. Sterilize the solution by passing it 
      through a prerinsed 0.22-µm filter. Store in 200-ml aliquots at 
    4°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>NaCl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 5 M 
      solution: Dissolve 292 g of NaCl in 800 ml of H<SUB>2</SUB>O. Adjust the 
      volume to 1 liter with H<SUB>2</SUB>O. Dispense into aliquots and 
      sterilize by autoclaving. Store the NaCl solution at room 
      temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>NaOH</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>The preparation of 10 N 
      NaOH involves a highly exothermic reaction, which can cause breakage of 
      glass containers. Prepare this solution with extreme care in plastic 
      beakers. To 800 ml of H<SUB>2</SUB>O, slowly add 400g of NaOH pellets, 
      stirring continuously. As an added precaution, place the beaker on ice. 
      When the pellets have dissolved completely, adjust the volume to 1 liter 
      with H<SUB>2</SUB>O. Store the solution in a plastic container at room 
      temperature. Sterilization is not 
necessary.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Sodium Acetate</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 3 M 
      solution: Dissolve 408.3 g of sodium acetate•3H<SUB>2</SUB>O in 800 ml of 
      H<SUB>2</SUB>O. Adjust the pH to 5.2 with glacial acetic acid or adjust 
      the pH to 7.0 with dilute acetic acid. Adjust the volume to 1 liter with 
      H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
    autoclaving.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>TE</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM Tris-Cl (desired 
      pH)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>10 mM EDTA (pH 
    8.0)</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>(10x Tris EDTA) Sterilize 
      solutions by autoclaving for 20 minutes at 15 psi (1.05 kg/cm 
      <SUP>2</SUP>) on liquid cycle. Store the buffer at room 
    temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Cl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve 121.1 g of Tris base in 800 ml of H<SUB>2</SUB>O. Adjust the 
      pH to the desired value by adding concentrated HCl.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2><I>pH</I></FONT> </TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2><I>HCl</I></FONT> 
        </TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.4 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>70 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.6 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>8.0 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>42 ml 
        </FONT></TD></TR></TBODY></TABLE>
      <P>(1 M) Allow the solution to cool to room temperature before making 
      final adjustments to the pH. Adjust the volume of the solution to 1 liter 
      with H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
      autoclaving.</P>
      <P>If the 1 M solution has a yellow color, discard it and obtain Tris of 
      better quality. The pH of Tris solutions is temperature-dependent and 
      decreases approx. 0.03 pH units for each 1°C increase in temperature. For 
      example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5°C, 
      25°C, and 37°C, respectively.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>dNTP Solution</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve each dNTP (deoxyribonucleoside triphosphates) in 
      H<SUB>2</SUB>O at an approximate concentration of 100 mM. Use 0.05 M Tris 
      base and a micropipette to adjust the pH of each of the solutions to 7.0 
      (use pH paper to check the pH). Dilute an aliquot of the neutralized dNTP 
      appropriately, and read the optical density at the wavelengths given in 
      the table below. Calculate the actual concentration of each dNTP. Dilute 
      the solutions with H<SUB>2</SUB>O to a final concentration of 50 mM dNTP. 
      Store each separately at -70°C in small aliquots. For polymerase chain 
      reactions (PCRs), adjust the dNTP solution to pH 8.0 with 2 N NaOH. 
      Commercially available solutions of PCR-grade dNTPs require no 
      adjustment.</P>
      <P>
      <TABLE width=400>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>Base</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana 
            size=2>wavelength<BR>(nm)</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Extinction 
            Coefficient<BR>(E)&nbsp;(M<SUP>-1</SUP>cm<SUP>-1</SUP>)</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>A</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>259</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.54 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>G</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>253</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.37 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>C</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>271</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.10 x 
            10<SUP>3</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>T</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>267</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.60 x 
            10<SUP>3</SUP></FONT></TD></TR></TBODY></TABLE></P>
      <P>For a cuvette with a path length of 1 cm, absorbance = EM. 100 mM stock 
      solutions of each dNTP are commercially available 
    (Pharmacia).</P></FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;     helvetica" size=3><STRONG>CAUTIONS</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" width=11 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>NaOH</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">NaOH, see Sodium hydroxide </FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Phenol:chloroform</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Phenol:chloroform, see Phenol; Chloroform 
</FONT></TD></TR></TBODY></TABLE><BR><BR><BR>
<OL></OL><BR><BR><BR><BR><BR><FONT face=Arial,Verdana,Helvetica 
size=1><STRONG>Copyright © 2000 by Cold Spring Harbor Laboratory Press. All 
rights reserved.</STRONG><BR>No part of these pages, either text or image may be 
used for any purpose other than personal use. Therefore, reproduction 
modification, storage in a retrieval system or retransmission, in any form or by 
any means, electronic, mechanical, or otherwise, for reasons other than personal 
use, is strictly prohibited without prior written 
permission.</FONT><BR><BR>
</td></tr></table>
</BODY></HTML>
