<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Molecular Cloning | Protocol 6 Print Version</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1251"><!--================== Designed and Developed by WriteDesign =================--><!--======================= Powered by RomeBlack, Inc. =======================--><!--=== Copyright 2000, 2001 Cold Spring Harbor Labs. All Rights Reserved. ===-->
<META content="" name=keywords>
<META 
content="Pairs of oligonucleotide primers used in PCR are often designed with restriction sites in their 5' regions. In many cases, the sites are different in the two primers. In this case, amplification generates a target fragment whose termini now carry new restriction sites that can be used for directional cloning into plasmid vectors. The purified fragment and the vector are digested with the appropriate restriction enzymes, ligated together, and transformed into E. coli." 
name=description>
<META content="MSHTML 5.50.4134.600" name=GENERATOR></HEAD>
<BODY bgColor=#ffffff background=../izo/fon.gif>
<table width=600 border="0" cellspacing="0" cellpadding="0"><tr><td width=600><IMG height=49 hspace=4 src="../izo/logo_csh.gif" 
width=169 align=left vspace=4><IMG height=49 hspace=4 
src="../izo/print_header.gif" width=293 align=left vspace=4><BR 
clear=all><BR><FONT face="arial,verdana, helvetica" size=4><STRONG>Chapter 8, 
Protocol 6</STRONG></FONT><BR><BR clear=all><FONT face=arial,verdana,helvetica 
size=3><STRONG>Cloning PCR Products by Addition of Restriction Sites to the 
Termini of Amplified DNA</STRONG></FONT><BR><BR><FONT 
face="arial,verdana, helvetica" size=2>Pairs of oligonucleotide primers used in 
PCR are often designed with restriction sites in their 5' regions. In many 
cases, the sites are different in the two primers. In this case, amplification 
generates a target fragment whose termini now carry new restriction sites that 
can be used for directional cloning into plasmid vectors. The purified fragment 
and the vector are digested with the appropriate restriction enzymes, ligated 
together, and transformed into <I>E. coli.</I></FONT> <BR><BR>
<TABLE border=1>
  <TBODY>
  <TR>
    <TD width=700><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" 
      width=11 border=0><FONT face=arial,verdana,helvetica 
      size=2><STRONG>CAUTION</STRONG></FONT><BR><IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0><FONT 
      face="arial,verdana, helvetica" size=2> <STRONG>RECIPE</STRONG></FONT> 
  </TD></TR></TBODY></TABLE><BR><BR><FONT face=arial,verdana,helvetica 
size=3><STRONG>MATERIALS</STRONG></FONT><BR><BR>
<TABLE cellPadding=0 cellborder="0">
  <TBODY>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Buffers and Solutions</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Chloroform</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>EDTA (0.5 M, 
      pH 8.0)</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Ethanol</FONT> 
    </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Phenol:chloroform (1:1, v/v)</FONT> <IMG height=11 alt=caution 
      hspace=4 src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Sodium acetate 
      (3 M, pH 5.2)</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>TE (pH 
      7.5)</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Enzymes and Buffers</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Bacteriophage 
      T4 DNA ligase</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Restriction 
      endonucleases</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Nucleic Acids and Oligonucleotides</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Forward primer 
      (20 µM) in H<SUB>2</SUB>O</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Reverse primer 
      (20 µM) in H<SUB>2</SUB>O</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Target 
      DNA</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Vectors</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Plasmid DNA 
      cleaved with the appropriate restriction enzyme(s) and purified by gel 
      electrophoresis</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>If the 
      linearized plasmid DNA carries compatible termini that can be ligated to 
      each other, use alkaline phosphatase to remove the 5&acute;-phosphate groups and 
      suppress self-ligation (please see <A href="../01/20.htm">Chapter 
      1, Protocol 20</A>).</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Additional Reagents</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 1 of this 
      protocol requires the reagents listed in <A href="01.htm">Chapter 
      8, Protocol 1 </A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 10 of 
      this protocol requires the reagents listed in <A href="../01/25.htm">Chapter 
      1, Protocol 25 </A>or <A href="../01/26.htm">Chapter 
      1, Protocol 26 </A>and <A href="../01/27.htm">Chapter 
      1, Protocol 27 </A></FONT></TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 11 of 
      this protocol may require the reagents listed in <A href="12.htm">Chapter 
      8, Protocol 12 </A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 13 of 
      this protocol requires the reagents listed in <A href="../12/06.htm">Chapter 
      12, Protocol 6 </A>or <A href="../06/10.htm">Chapter 
      6, Protocol 10 </A>.</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 
  border=0></TD></TR></TBODY></TABLE><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" 
size=3><STRONG>METHOD</STRONG></FONT> <BR><FONT face="arial,verdana, helvetica" 
color=#000000 size=2>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Design and 
  synthesize the appropriate oligonucleotide primers. Use these forward and 
  reverse primers to set up and carry out four identical amplification reactions 
  (50-µl volume) to amplify the target fragment (please see <A href="01.htm">Chapter 
  8, Protocol 1</A>). Combine the four PCRs, which, in aggregate, should contain 
  200-500 ng of the desired amplification product.<BR><I>Design forward and 
  reverse primers carrying the appropriate restriction sites. The 3&acute; end of each 
  primer should be an exact complement of approx. 15 consecutive bases at a 
  selected site in the target DNA. The 5&acute; terminus of each primer serves as a 
  clamp to hold together the termini of the amplified DNA and to provide a 
  landing site for the restriction enzyme. The clamp should be 3-10 nucleotides 
  in length. The mid-portion of the primer contains the recognition site for the 
  restriction enzyme. Each primer should therefore be 24-31 nucleotides in 
  length and contain approximately equal numbers of the four bases, with a 
  balanced distribution of G and C residues and a low propensity to form stable 
  secondary structures.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>If the PCR 
  mixture contains more than one or two bands of amplified DNA, purify the 
  target fragment by electrophoresis through low melting/gelling temperature 
  agarose (please see <A href="../05/06.htm">Chapter 
  5, Protocol 6 </A>). If not purified by gel electrophoresis, prepare 
  PCR-amplified DNA for ligation by extraction with phenol:chloroform and 
  ultrafiltration through a Centricon-100 filter (please see <A href="03.htm">Chapter 
  8, Protocol 3</A>). Dissolve in TE (pH 7.5) at a concentration of 25 µg/ml. 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>In a reaction 
  volume of 20 µl, digest approx. 100 ng of purified PCR product with 1.0-2.0 
  units of the relevant restriction enzyme(s). Incubate the reactions for 1 hour 
  at the optimum temperature for digestion. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>At the end of 
  the digestion, adjust the volume to 100 µl with H<SUB>2</SUB>O, and add 0.5 M 
  EDTA to a final concentration of 5 mM. Extract once with phenol:chloroform and 
  once with chloroform. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Transfer the 
  aqueous phase to a fresh tube and add 3 M sodium acetate (pH 5.2) to achieve a 
  final concentration of 0.3 M. Add 2 volumes of ethanol. Store the mixture for 
  30 minutes at 0°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Recover the 
  precipitated DNA by centrifugation at maximum speed for 5 minutes at 4°C in a 
  microfuge. Discard the supernatant, and then wash the pellet with 70% ethanol. 
  Centrifuge again, remove the supernatant, and allow the DNA to dry. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Dissolve the 
  DNA in 10 µl of H<SUB>2</SUB>O. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>In a microfuge 
  tube, set up the following ligation mixture: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>25 µg/ml 
        amplified target DNA</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1.0 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>plasmid 
        DNA</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 
    ng</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x ligation 
        buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1.0 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>T4 DNA 
        ligase</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    unit</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 10 
      µl</FONT></TD></TR></TBODY></TABLE><I>If necessary, add ATP to a final 
  concentration of 1 mM.</I><BR><I>When directional cloning is used, the 
  ligation mixture should contain an approx. 1:1 molar ratio of purified target 
  DNA to cleaved plasmid vector.</I><BR>Set up a control reaction that contains 
  all the reagents listed above except the amplified target DNA. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Incubate the 
  ligation mixtures for 4 hours at 16°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Dilute 5 µl of 
  each of the two ligation mixtures with 10 µl of H<SUB>2</SUB>O and transform a 
  suitable strain of competent <I>E. coli</I> to antibiotic resistance as 
  described in <A href="../01/25.htm">Chapter 
  1, Protocol 25 </A>or <A href="../01/26.htm">Chapter 
  1, Protocol 26 </A>. Plate the transformed cultures on media containing IPTG 
  and X-gal (please see <A href="../01/27.htm">Chapter 
  1, Protocol 27 </A>) and the appropriate antibiotic. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Calculate the 
  number of colonies obtained from each of the ligation mixtures. Pick a number 
  of white colonies obtained by transformation with the ligation reaction 
  containing the target DNA. Confirm the presence of the amplified fragment by 
  (i) isolating the plasmid DNAs and digesting them with restriction enzymes 
  whose sites flank the insert in the multiple cloning site or (ii) colony PCR 
  (<A href="12.htm"> 
  Chapter 8, Protocol 12</A>).<BR><I>In different experiments, the ratio of 
  blue:white colonies can vary between 1:5 and 2:1.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Fractionate the 
  restricted DNA by electrophoresis through an agarose gel using appropriate DNA 
  size markers. Measure the size of the cloned fragments. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Confirm the 
  identity of the cloned fragments by DNA sequencing, restriction mapping, or 
  Southern hybridization. </FONT></LI></OL></FONT><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>RECIPES</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/circle.gif" width=12 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>EDTA</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare EDTA at 0.5 M 
      (pH 8.0): Add 186.1 g of disodium EDTA•2H<SUB>2</SUB>O to 800 ml of 
      H<SUB>2</SUB>O. Stir vigorously on a magnetic stirrer. Adjust the pH to 
      8.0 with NaOH (approx. 20 g of NaOH pellets). Dispense into aliquots and 
      sterilize by autoclaving. The disodium salt of EDTA will not go into 
      solution until the pH of the solution is adjusted to approx. 8.0 by the 
      addition of NaOH.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Sodium Acetate</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 3 M 
      solution: Dissolve 408.3 g of sodium acetate•3H<SUB>2</SUB>O in 800 ml of 
      H<SUB>2</SUB>O. Adjust the pH to 5.2 with glacial acetic acid or adjust 
      the pH to 7.0 with dilute acetic acid. Adjust the volume to 1 liter with 
      H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
    autoclaving.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>TE</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM Tris-Cl (desired 
      pH)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>10 mM EDTA (pH 
    8.0)</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>(10x Tris EDTA) Sterilize 
      solutions by autoclaving for 20 minutes at 15 psi (1.05 kg/cm 
      <SUP>2</SUP>) on liquid cycle. Store the buffer at room 
    temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Cl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve 121.1 g of Tris base in 800 ml of H<SUB>2</SUB>O. Adjust the 
      pH to the desired value by adding concentrated HCl.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2><I>pH</I></FONT> </TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2><I>HCl</I></FONT> 
        </TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.4 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>70 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.6 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>8.0 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>42 ml 
        </FONT></TD></TR></TBODY></TABLE>
      <P>(1 M) Allow the solution to cool to room temperature before making 
      final adjustments to the pH. Adjust the volume of the solution to 1 liter 
      with H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
      autoclaving.</P>
      <P>If the 1 M solution has a yellow color, discard it and obtain Tris of 
      better quality. The pH of Tris solutions is temperature-dependent and 
      decreases approx. 0.03 pH units for each 1°C increase in temperature. For 
      example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5°C, 
      25°C, and 37°C, 
respectively.</P></FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;     helvetica" size=3><STRONG>CAUTIONS</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" width=11 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Chloroform</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Chloroform CHCl<SUB>3</SUB> is irritating to the skin, eyes, 
      mucous membranes, and respiratory tract. It is a carcinogen and may damage 
      the liver and kidneys. It is also volatile. Avoid breathing the vapors. 
      Wear appropriate gloves and safety glasses. Always use in a chemical fume 
      hood. </FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Phenol:chloroform</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Phenol:chloroform, see Phenol; Chloroform 
</FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>REFERENCES</STRONG> 
</FONT><BR>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Kaufman D.L. 
  and Evans G.A. 1990. Restriction endonuclease cleavage at the termini of PCR 
  products. <I>BioTechniques</I> 9:304-306. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Scharf S.J., 
  Horn G.T., and Erlich H.A. 1986. Direct cloning and sequence analysis of 
  enzymatically amplified genomic sequences. <I>Science</I> 233:1076-1078. 
  </FONT></LI></OL><BR><BR><BR><BR><BR><FONT face=Arial,Verdana,Helvetica 
size=1><STRONG>Copyright © 2000 by Cold Spring Harbor Laboratory Press. All 
rights reserved.</STRONG><BR>No part of these pages, either text or image may be 
used for any purpose other than personal use. Therefore, reproduction 
modification, storage in a retrieval system or retransmission, in any form or by 
any means, electronic, mechanical, or otherwise, for reasons other than personal 
use, is strictly prohibited without prior written 
permission.</FONT><BR><BR>
</td></tr></table>
</BODY></HTML>
