<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Molecular Cloning | Protocol 9 Print Version</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1251"><!--================== Designed and Developed by WriteDesign =================--><!--======================= Powered by RomeBlack, Inc. =======================--><!--=== Copyright 2000, 2001 Cold Spring Harbor Labs. All Rights Reserved. ===-->
<META content="" name=keywords>
<META 
content="This method is used to extend partial cDNA clones by amplifying the 5&acute; sequences of the corresponding mRNAs. The technique requires knowledge of a small region of sequence within the partial cDNA clone. During PCR, the thermostable DNA polymerase is directed to the appropriate target RNA by a single primer derived from the region of known sequence; the second primer required for PCR is complementary to a general feature of the target -- in the case of 5&acute;-RACE, to a homopolymeric tail added (via terminal transferase) to the 3&acute; termini of cDNAs transcribed from a preparation of mRNA. This synthetic tail provides a primer-binding site upstream of the unknown 5&acute; sequence of the target mRNA. The products of the amplification reaction are cloned into a plasmid vector for sequencing and subsequent manipulation. To reduce the chance of contamination with exogenous DNAs, prepare and use a special set of reagents and solutions for PCR only. Bake all glassware for 6 hours at 150°C and autoclave all plasticware." name=description>
<META content="MSHTML 5.50.4134.600" name=GENERATOR></HEAD>
<BODY bgColor=#ffffff background=../izo/fon.gif>
<table width=600 border="0" cellspacing="0" cellpadding="0"><tr><td width=600><IMG height=49 hspace=4 src="../izo/logo_csh.gif" 
width=169 align=left vspace=4><IMG height=49 hspace=4 
src="../izo/print_header.gif" width=293 align=left vspace=4><BR 
clear=all><BR><FONT face="arial,verdana, helvetica" size=4><STRONG>Chapter 8, 
Protocol 9</STRONG></FONT><BR><BR clear=all><FONT face=arial,verdana,helvetica 
size=3><STRONG>Rapid Amplification of 5&acute; cDNA Ends 
(5&acute;-RACE)</STRONG></FONT><BR><BR><FONT face="arial,verdana, helvetica" 
size=2>This method is used to extend partial cDNA clones by amplifying the 5&acute; 
sequences of the corresponding mRNAs. The technique requires knowledge of a 
small region of sequence within the partial cDNA clone. During PCR, the 
thermostable DNA polymerase is directed to the appropriate target RNA by a 
single primer derived from the region of known sequence; the second primer 
required for PCR is complementary to a general feature of the target -- in the 
case of 5&acute;-RACE, to a homopolymeric tail added (via terminal transferase) to the 
3&acute; termini of cDNAs transcribed from a preparation of mRNA. This synthetic tail 
provides a primer-binding site upstream of the unknown 5&acute; sequence of the target 
mRNA. The products of the amplification reaction are cloned into a plasmid 
vector for sequencing and subsequent manipulation. To reduce the chance of 
contamination with exogenous DNAs, prepare and use a special set of reagents and 
solutions for PCR only. Bake all glassware for 6 hours at 150°C and autoclave 
all plasticware.</FONT> <BR><BR>
<TABLE border=1>
  <TBODY>
  <TR>
    <TD width=700><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" 
      width=11 border=0><FONT face=arial,verdana,helvetica 
      size=2><STRONG>CAUTION</STRONG></FONT><BR><IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0><FONT 
      face="arial,verdana, helvetica" size=2> <STRONG>RECIPE</STRONG></FONT> 
  </TD></TR></TBODY></TABLE><BR><BR><FONT face=arial,verdana,helvetica 
size=3><STRONG>MATERIALS</STRONG></FONT><BR><BR>
<TABLE cellPadding=0 cellborder="0">
  <TBODY>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Buffers and Solutions</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>10x 
      Amplification buffer</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Chloroform</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>dNTP solution 
      (20 mM) containing all four dNTPs (pH 8.0)</FONT> <IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Placental 
      RNase inhibitor (20 units/µl)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>TE (pH 
      7.6)</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Enzymes and Buffers</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>5x Reverse 
      transcriptase buffer</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Reverse 
      transcriptase (RNA-dependent DNA polymerase)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Terminal 
      deoxynucleotidyl transferase (terminal transferase)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>5x Terminal 
      transferase buffer</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Thermostable 
      DNA polymerase</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Taq<I> DNA 
      polymerase is the standard and appropriate enzyme for the amplification 
      stage of most forms of RT-PCR. However, where elongation from 
      3&acute;-mismatched primers is suspected, a thermostable DNA polymerase with 
      3&acute;-&gt;5&acute; proofreading activity may be preferred.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Nucleic Acids and Oligonucleotides</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Adaptor-primer 
      (10 µM) (5&acute; GACTCGAGTCGACATCG 3&acute;) in H<SUB>2</SUB>O (10 pmoles/µl)</FONT> 
    </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>The 
      adaptor-primer is used in conjunction with a gene-specific sense primer to 
      amplify a particular target cDNA.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>(dT)<SUB>17</SUB>-Adaptor-primer (10 µM) (5&acute; 
      GACTCGAGTCGACATCGA(T)<SUB>17</SUB> 3&acute;) in H<SUB>2</SUB>O (10 
      pmoles/µl)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>The 
      (dT)<SUB>17</SUB>-adaptor-primer binds to the poly(A)<SUP>+</SUP> tract 
      added to the 5&acute; terminus of cDNAs by terminal transferase. In the example 
      given in this protocol, one of the termini of the amplified DNAs becomes 
      equipped with recognition sites for </I>Xho<I>I, </I>Sal<I>I, </I>Acc<I>I, 
      </I>Hin<I>cII, and </I>Cla<I>I restriction enzymes.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Gene-specific 
      antisense oligonucleotide primers (10 µM) in H<SUB>2</SUB>O (10 
      pmoles/µl)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>The 
      gene-specific antisense primers should be complementary to the known 
      sequence of the target mRNA, should be 20-30 nucleotides in length, and 
      should contain approximately equal numbers of the four bases, with a 
      balanced distribution of G and C residues and a low propensity to form 
      stable secondary structures. Gene-specific primer 1 is used in the reverse 
      transcriptase reaction (Step 2) to prime synthesis of gene-specific 
      first-strand cDNA. Gene-specific primer 2 is complementary to a sequence 
      in the target mRNA that is 5&acute; to primer 1 and is used in the amplification 
      stage of the reaction. The presence of restriction sites simplifies 
      cloning of the amplified cDNAs.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Random 
      hexanucleotides (1 mg/ml) in TE (pH 8.0) (optional)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>In an 
      emergency, random hexanucleotides can be used in place of the 
      gene-specific antisense oligonucleotide to prime synthesis of cDNA. 
      Subsequent use of a gene-specific sense primer in the PCR phase of the 
      5&acute;-RACE reaction generates the desired product.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Template 
      RNA<BR><I>The template may be total RNA (100 µg/ml) in H<SUB>2</SUB>O or 
      poly(A)<SUP>+</SUP> RNA (10 µg/ml) in H<SUB>2</SUB>O. Total RNA extracted 
      from cells with chaotropic agents is generally the template of choice for 
      RT-PCR for mRNAs that are expressed at moderate to high abundance. 
      Poly(A)<SUP>+</SUP> is preferred as a template for all forms of RT-PCR 
      when the target mRNA is expressed at low abundance. </I></FONT></TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Additional Reagents</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 9 of this 
      protocol requires the reagents listed in <A href="../05/02.htm">Chapter 
      5, Protocol 2 </A>, <A href="../12/06.htm">Chapter 
      12, Protocol 6</A>, or <A href="../06/10.htm">Chapter 
      6, Protocol 10 </A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 11 of 
      this protocol requires the reagents listed in <A href="03.htm">Chapter 
      8, Protocol 3 </A>and <A href="04.htm">Chapter 
      8, Protocol 4, </A><A href="05.htm">Chapter 
      8, Protocol 5 </A>, or <A href="06.htm">Chapter 
      8, Protocol 6 </A>.</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 
  border=0></TD></TR></TBODY></TABLE><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" 
size=3><STRONG>METHOD</STRONG></FONT> <BR><FONT face="arial,verdana, helvetica" 
color=#000000 size=2>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Transfer 1 pg 
  to 100 ng of poly(A)<SUP>+</SUP> mRNA or 10 pg to 1 µg of total RNA to a fresh 
  microfuge tube. Adjust the volume to 9 µl with H<SUB>2</SUB>O. Denature the 
  RNA by heating for 5 minutes at 75°C, followed by rapid chilling on ice. 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>To the 
  denatured RNA add: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5x reverse 
        transcriptase buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>4 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 mM solution 
        of four dNTPs (pH 8.0)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10 µM 
        gene-specific antisense primer 1</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>4 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>approx. 20 
        units/µl placental RNase inhibitor</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>100-200 
        units/µl reverse transcriptase</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 20 
      µl</FONT></TD></TR></TBODY></TABLE>Incubate the reaction for 60 minutes at 
  37°C. Set up three negative control reactions. In one reaction, include all of 
  the components of the first-strand reaction, except the RNA template. In 
  another reaction, include all of the components, except the reverse 
  transcriptase. Omit primer from the third reaction. Carry the controls through 
  all subsequent steps of the protocol.<BR>Total<I> cDNA synthesis can be 
  estimated from the proportion of trichloroacetic acid (TCA)-precipitable 
  radioactivity incorporated in reverse transcription reactions supplemented 
  with 10-20 µCi of [<SUP>32</SUP>P]dCTP (sp. act. 3000 Ci/mmole). Be sure to 
  set up a control reaction that contains no oligonucleotide 
  primer.</I><BR><I>The success or failure of 5&acute;-RACE is determined here. If the 
  reverse transcription step works efficiently, the chance of isolating clones 
  that contain the 5&acute;-terminal sequences of the target mRNA is high. On the 
  other hand, no amount of work on the later steps of the protocol can 
  compensate for ineffective reverse transcription. It is therefore worthwhile 
  to take the time to optimize the reverse transcriptase reaction by determining 
  the optimum ratio of primer to template for each preparation of RNA and by 
  varying the concentration of Mg<SUP>2+</SUP> in the reaction.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Remove excess 
  primer by diluting the reverse transcriptase reaction to a final volume of 2 
  ml with H<SUB>2</SUB>O and then applying the solution to a Centricon-100 
  microconcentrator (please see <A href="03.htm">Chapter 
  8, Protocol 3</A>). Centrifuge the solution at 500-1100<I>g</I> (2000-3000 rpm 
  in a Sorvall SS-34 rotor) for 20 minutes at a temperature between 4°C and 
  25°C. Repeat the dilution step and centrifuge again. Transfer the retentate to 
  a fresh 0.5-ml microfuge tube and reduce the volume to approx. 10 µl in a 
  rotary vacuum evaporator.<BR><I>Alternatively, remove the dNTPs and unused 
  primers by precipitating the cDNA twice in 2.5 M ammonium acetate and 3 
  volumes of ethanol.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>To the cDNA in 
  a volume of 10 µl add: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5x terminal 
        transferase buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>4 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 mM 
      dATP</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>4 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>terminal 
        transferase</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10-25 
        units</FONT></TD></TR></TBODY></TABLE>Incubate the reaction for 15 minutes at 
  37°C.<BR><I>The tailing reaction can be optimized by setting up mock reactions 
  containing approx. 50 ng of a control DNA fragment, 100-200 nucleotides in 
  length. After tailing, the size of the fragment should increase by 20-100 
  nucleotides as measured by electrophoresis through a 1% neutral agarose 
  gel.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Inactivate the 
  terminal transferase by heating the reaction for 3 minutes at 80°C. Dilute the 
  dA-tailed cDNA to a final volume of 1 ml with TE (pH 7.6). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>In a sterile 
  0.5-ml microfuge tube, amplification tube, or the well of a sterile microtiter 
  plate, set up a series of PCRs containing the following: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>diluted 
        cDNA</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>0-20 
      µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x 
        amplification buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 mM solution 
        of four dNTPs</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10 µM 
        (dT)<SUB>17</SUB>-adaptor-primer (16 pmoles)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1.6 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10 µM 
        adaptor-primer (32 pmoles)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>3.2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10 µM 
        gene-specific primer 2 (32 pmoles)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>3.2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1-2 units of 
        thermostable DNA polymerase</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 50 
      µl</FONT></TD></TR></TBODY></TABLE><I>It is essential to set up a series of 
  amplification reactions to find the amount of tailed cDNA that generates the 
  largest quantity of amplified 5&acute; termini. The control reaction containing no 
  cDNA template serves as a control for contamination.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>If the thermal 
  cycler is not fitted with a heated lid, overlay the reaction mixtures with 1 
  drop (approx. 50 µl) of light mineral oil. Alternatively, place a bead of 
  paraffin wax into the tube if using a hot start protocol. Place the tubes or 
  the microtiter plate in the thermal cycler. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Amplify the 
  nucleic acids using the denaturation, annealing, and polymerization times and 
  temperatures listed in the table. 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2><B>Cycle 
        Number</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Denaturation</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Annealing</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Polymerization</B></FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>First 
        Cycle</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 min at 
        94°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 min at 
        50-58°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>40 min at 
        72°C</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Subsequent 
        cycles (30)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>40 sec at 
        94°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 min at 
        50-58°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>3 min at 
        72°C</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Last 
        cycle</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>40 sec at 
        94°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 min at 
        50-58°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>15 min at 
        72°C</FONT></TD></TR></TBODY></TABLE><I>Times and temperatures may need to be 
  adapted to suit the particular reaction conditions.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Withdraw a 
  sample (5-10 µl) from the test reaction mixture and the four control reactions 
  and analyze them by electrophoresis through an agarose or polyacrylamide gel. 
  Include DNA markers of an appropriate size. Stain the gel with ethidium 
  bromide or SYBR Gold to visualize the DNA.<BR><I>A successful amplification 
  reaction should yield a readily visible DNA fragment of the expected size. The 
  identity of the band can be confirmed by DNA sequencing (please see </I><A href="../12/06.htm">Chapter 
  12, Protocol 6</A>),<I> Southern hybridization (please see <A href="../06/10.htm">Chapter 
  6, Protocol 10</A>), and/or restriction mapping.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>If mineral oil 
  was used to overlay the reaction (in Step 7), remove the oil from the sample 
  before cloning by extraction with 150 µl of chloroform.<BR><I>The aqueous 
  phase, which contains the amplified DNA, will form a micelle near the 
  meniscus. The micelle can be transferred to a fresh tube with an automatic 
  micropipette.</I><BR><STRONG>IMPORTANT</STRONG> <I>Do not attempt chloroform 
  extractions in microtiter plates. The plastic used in these plates is not 
  resistant to organic solvents.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Separate the 
  amplified DNA from the residual thermostable DNA polymerase and dNTPs (please 
  see <A href="03.htm">Chapter 
  8, Protocol 3</A>).<BR><I>The DNA can now be ligated to a blunt-ended vector 
  or a T vector, or it can digested with restriction enzymes and ligated to a 
  vector with compatible termini (please see </I><A href="04.htm">Chapter 
  8, Protocol 4</A>, <A href="05.htm">Chapter 
  8, Protocol 5</A>, <I>and</I> <A href="06.htm">Chapter 
  8, Protocol 6)</A>. </FONT></LI></OL></FONT><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>RECIPES</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/circle.gif" width=12 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Amplification Buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>500 mM KCl</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM Tris-Cl (pH 8.3 at 
      room temperature)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>15 mM 
      MgCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Autoclave the 10x buffer 
      for 10 minutes at 15 psi (1.05 kg/cm <SUP>2</SUP>) on liquid cycle. Divide 
      the sterile buffer into aliquots and store them at 
  -20°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>EDTA</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare EDTA at 0.5 M 
      (pH 8.0): Add 186.1 g of disodium EDTA•2H<SUB>2</SUB>O to 800 ml of 
      H<SUB>2</SUB>O. Stir vigorously on a magnetic stirrer. Adjust the pH to 
      8.0 with NaOH (approx. 20 g of NaOH pellets). Dispense into aliquots and 
      sterilize by autoclaving. The disodium salt of EDTA will not go into 
      solution until the pH of the solution is adjusted to approx. 8.0 by the 
      addition of NaOH.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>KCl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Dissolve an appropriate 
      amount of solid KCl in H<SUB>2</SUB>O, autoclave for 20 minutes on liquid 
      cycle and store at room temperature. Ideally, this 4 M solution should be 
      divided into small (approx. 100 µl) aliquots in sterile tubes and each 
      aliquot thereafter used one time.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Reverse Transcriptase Buffer</STRONG></FONT> 
    </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>500 mM Tris-Cl (pH 
      8.3)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>750 mM KCl</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>30 mM 
      MgCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>10x 
  stock.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>TE</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM Tris-Cl (desired 
      pH)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>10 mM EDTA (pH 
    8.0)</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>(10x Tris EDTA) Sterilize 
      solutions by autoclaving for 20 minutes at 15 psi (1.05 kg/cm 
      <SUP>2</SUP>) on liquid cycle. Store the buffer at room 
    temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Terminal Transferase Buffer</STRONG></FONT> 
    </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>500 mM potassium 
      cacodylate (pH 7.2)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>10 mM 
      CoCl<SUB>2</SUB>•6H<SUB>2</SUB>O</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>1 mM 
    dithiothreitol</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>5x terminal transferase (or tailing) buffer may be prepared according 
      to the following method:</P>
      <P>
      <TABLE>
        <TBODY>
        <TR>
          <TD vAlign=top><FONT face=arial,helvetica,verdana 
size=2>1.</FONT></TD>
          <TD vAlign=top><FONT face=arial,helvetica,verdana size=2>Equilibrate 
            5 g of Chelex 100 (Bio-Rad) with 10 ml of 3 M potassium acetate at 
            room temperature.</FONT></TD></TR>
        <TR>
          <TD vAlign=top><FONT face=arial,helvetica,verdana 
size=2>2.</FONT></TD>
          <TD vAlign=top><FONT face=arial,helvetica,verdana size=2>After five 
            minutes, remove excess liquid by vacuum suction. Wash the Chelex 
            three times with 10 ml of deionized H<SUB>2</SUB>0.</FONT></TD></TR>
        <TR>
          <TD vAlign=top><FONT face=arial,helvetica,verdana 
size=2>3.</FONT></TD>
          <TD vAlign=top><FONT face=arial,helvetica,verdana size=2>Prepare a 1 
            M solution of potassium cacodylate. Equilibrate the cacodylate 
            solution with the treated Chelex resin.</FONT></TD></TR>
        <TR>
          <TD vAlign=top><FONT face=arial,helvetica,verdana 
size=2>4.</FONT></TD>
          <TD vAlign=top><FONT face=arial,helvetica,verdana size=2>Recover the 
            cacodylate solution by passing it through a Buchner funnel fitted 
            with Whatman No. 1 filter paper.</FONT></TD></TR>
        <TR>
          <TD vAlign=top><FONT face=arial,helvetica,verdana 
size=2>5.</FONT></TD>
          <TD vAlign=top><FONT face=arial,helvetica,verdana size=2>To the 
            recovered cacodylate add in order: H<SUB>2</SUB>0, dithiothreitol, 
            and cobalt chloride to make the final concentrations of 500 mM 
            potassium cacodylate, 1 mM dithiothreitol, and 20 mM 
            CoCl<SUB>2</SUB>.</FONT></TD></TR></TBODY></TABLE></P>
      <P>Store the buffer in aliquots at 
-20°C.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Cl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve 121.1 g of Tris base in 800 ml of H<SUB>2</SUB>O. Adjust the 
      pH to the desired value by adding concentrated HCl.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2><I>pH</I></FONT> </TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2><I>HCl</I></FONT> 
        </TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.4 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>70 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.6 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>8.0 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>42 ml 
        </FONT></TD></TR></TBODY></TABLE>
      <P>(1 M) Allow the solution to cool to room temperature before making 
      final adjustments to the pH. Adjust the volume of the solution to 1 liter 
      with H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
      autoclaving.</P>
      <P>If the 1 M solution has a yellow color, discard it and obtain Tris of 
      better quality. The pH of Tris solutions is temperature-dependent and 
      decreases approx. 0.03 pH units for each 1°C increase in temperature. For 
      example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5°C, 
      25°C, and 37°C, respectively.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>dNTP Solution</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve each dNTP (deoxyribonucleoside triphosphates) in 
      H<SUB>2</SUB>O at an approximate concentration of 100 mM. Use 0.05 M Tris 
      base and a micropipette to adjust the pH of each of the solutions to 7.0 
      (use pH paper to check the pH). Dilute an aliquot of the neutralized dNTP 
      appropriately, and read the optical density at the wavelengths given in 
      the table below. Calculate the actual concentration of each dNTP. Dilute 
      the solutions with H<SUB>2</SUB>O to a final concentration of 50 mM dNTP. 
      Store each separately at -70°C in small aliquots. For polymerase chain 
      reactions (PCRs), adjust the dNTP solution to pH 8.0 with 2 N NaOH. 
      Commercially available solutions of PCR-grade dNTPs require no 
      adjustment.</P>
      <P>
      <TABLE width=400>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>Base</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana 
            size=2>wavelength<BR>(nm)</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Extinction 
            Coefficient<BR>(E)&nbsp;(M<SUP>-1</SUP>cm<SUP>-1</SUP>)</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>A</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>259</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.54 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>G</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>253</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.37 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>C</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>271</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.10 x 
            10<SUP>3</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>T</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>267</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.60 x 
            10<SUP>3</SUP></FONT></TD></TR></TBODY></TABLE></P>
      <P>For a cuvette with a path length of 1 cm, absorbance = EM. 100 mM stock 
      solutions of each dNTP are commercially available 
    (Pharmacia).</P></FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;     helvetica" size=3><STRONG>CAUTIONS</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" width=11 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Chloroform</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Chloroform CHCl<SUB>3</SUB> is irritating to the skin, eyes, 
      mucous membranes, and respiratory tract. It is a carcinogen and may damage 
      the liver and kidneys. It is also volatile. Avoid breathing the vapors. 
      Wear appropriate gloves and safety glasses. Always use in a chemical fume 
      hood. </FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Potassium cacodylate</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Potassium cacodylate, see Cacodylate 
</FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>REFERENCES</STRONG> 
</FONT><BR>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Frohman M.A., 
  Dush M.K., and Martin G.R. 1988. Rapid production of full-length cDNAs from 
  rare transcripts: Amplification using a single gene-specific oligonucleotide 
  primer. <I>Proc. Natl. Acad. Sci.</I> 85:8998-9002. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Schaefer B.C. 
  1995. Revolutions in rapid amplification of cDNA ends: New strategies for 
  polymerase chain reaction cloning of full-length cDNA ends. <I>Anal. 
  Biochem.</I> 227:255-273. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Zhang Y. and 
  Frohman M.A. 1997. Using rapid amplification of cDNA ends (RACE) to obtain 
  full-length cDNAs. <I>Methods Mol. Biol</I> 69:61-87. 
</FONT></LI></OL><BR><BR><BR><BR><BR><FONT face=Arial,Verdana,Helvetica 
size=1><STRONG>Copyright © 2000 by Cold Spring Harbor Laboratory Press. All 
rights reserved.</STRONG><BR>No part of these pages, either text or image may be 
used for any purpose other than personal use. Therefore, reproduction 
modification, storage in a retrieval system or retransmission, in any form or by 
any means, electronic, mechanical, or otherwise, for reasons other than personal 
use, is strictly prohibited without prior written 
permission.</FONT><BR><BR>
</td></tr></table>
</BODY></HTML>
