<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Molecular Cloning | Protocol 16 Print Version</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1251"><!--================== Designed and Developed by WriteDesign =================--><!--======================= Powered by RomeBlack, Inc. =======================--><!--=== Copyright 2000, 2001 Cold Spring Harbor Labs. All Rights Reserved. ===-->
<META content="" name=keywords>
<META 
content="This method uses PCR to amplify and display many cDNAs derived from the mRNAs of a given cell or tissue type. The method relies on two different types of synthetic oligonucleotides: anchored antisense primers and arbitrary sense primers. A typical anchored primer is complementary to approx. 13 nucleotides of the poly(A) tail of mRNA and the adjacent two nucleotides of the transcribed sequence. Anchored primers therefore anneal to the junction between the poly(A) tail and the 3&acute;-untranslated region of mRNA templates, from where they can prime synthesis of first-strand cDNA. A second primer, an arbitrary sequence of approx. 10 nucleotides, is then added to the reaction mixture, and double-stranded cDNAs are produced by conventional PCR, carried out at low stringency. The products of the amplification reaction are separated by electrophoresis through a denaturing polyacrylamide gel and visualized by autoradiography. By comparing the banding patterns of cDNA products derived from two different cell types, or from the same cell type grown under different conditions, it is sometimes possible to identify the products of differentially expressed genes. Bands of interest can then be recovered from the gel, amplified further, and cloned and/or used as probes to screen northern blots, cDNA libraries, etc. To reduce the chance of contamination with exogenous DNAs, prepare and use a special set of reagents and solutions for PCR only. Bake all glassware for 6 hours at 150°C and autoclave all plasticware." 
name=description>
<META content="MSHTML 5.50.4134.600" name=GENERATOR></HEAD>
<BODY bgColor=#ffffff background=../izo/fon.gif>
<table width=600 border="0" cellspacing="0" cellpadding="0"><tr><td width=600><IMG height=49 hspace=4 src="../izo/logo_csh.gif" 
width=169 align=left vspace=4><IMG height=49 hspace=4 
src="../izo/print_header.gif" width=293 align=left vspace=4><BR 
clear=all><BR><FONT face="arial,verdana, helvetica" size=4><STRONG>Chapter 8, 
Protocol 16</STRONG></FONT><BR><BR clear=all><FONT face=arial,verdana,helvetica 
size=3><STRONG>Differential Display-PCR</STRONG></FONT><BR><BR><FONT 
face="arial,verdana, helvetica" size=2>This method uses PCR to amplify and 
display many cDNAs derived from the mRNAs of a given cell or tissue type. The 
method relies on two different types of synthetic oligonucleotides: anchored 
antisense primers and arbitrary sense primers. A typical anchored primer is 
complementary to approx. 13 nucleotides of the poly(A) tail of mRNA and the 
adjacent two nucleotides of the transcribed sequence. Anchored primers therefore 
anneal to the junction between the poly(A) tail and the 3&acute;-untranslated region 
of mRNA templates, from where they can prime synthesis of first-strand cDNA. A 
second primer, an arbitrary sequence of approx. 10 nucleotides, is then added to 
the reaction mixture, and double-stranded cDNAs are produced by conventional 
PCR, carried out at low stringency. The products of the amplification reaction 
are separated by electrophoresis through a denaturing polyacrylamide gel and 
visualized by autoradiography. By comparing the banding patterns of cDNA 
products derived from two different cell types, or from the same cell type grown 
under different conditions, it is sometimes possible to identify the products of 
differentially expressed genes. Bands of interest can then be recovered from the 
gel, amplified further, and cloned and/or used as probes to screen northern 
blots, cDNA libraries, etc. To reduce the chance of contamination with exogenous 
DNAs, prepare and use a special set of reagents and solutions for PCR only. Bake 
all glassware for 6 hours at 150°C and autoclave all plasticware.</FONT> 
<BR><BR>
<TABLE border=1>
  <TBODY>
  <TR>
    <TD width=700><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" 
      width=11 border=0><FONT face=arial,verdana,helvetica 
      size=2><STRONG>CAUTION</STRONG></FONT><BR><IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0><FONT 
      face="arial,verdana, helvetica" size=2> <STRONG>RECIPE</STRONG></FONT> 
  </TD></TR></TBODY></TABLE><BR><BR><FONT face=arial,verdana,helvetica 
size=3><STRONG>MATERIALS</STRONG></FONT><BR><BR>
<TABLE cellPadding=0 cellborder="0">
  <TBODY>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Buffers and Solutions</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>10x 
      Amplification buffer</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>5x DD-PCR 
      reverse transcriptase buffer</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>DTT 
      (dithiothreitol) (100 mM)</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>dNTP solution 
      (20 mM) containing all four dNTPs (pH 8.0)</FONT> <IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Placental 
      RNase inhibitor (20 units/ml)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>5x Formamide 
      loading buffer</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0><IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Enzymes and Buffers</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Reverse 
      transcriptase (RNA-dependent DNA polymerase). </FONT></TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>For DD-PCR, 
      a reverse transcriptase deficient in RNase H is required (e.g., 
      Superscript from Life Technologies or StrataScript from 
      Stratagene).</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Thermostable 
      DNA polymerase</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Taq<I> DNA 
      polymerase is the standard and appropriate enzyme for the amplification 
      stage of most forms of DD-PCR.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Nucleic Acids and Oligonucleotides</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Anchoring 3&acute; 
      oligonucleotide primers (300 µg/ml) in 10 mM Tris-Cl (pH 7.6), 0.1 mM 
      EDTA</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>The 
      anchoring 3&acute; oligonucleotides are a family of 12 primers with the general 
      structure 5&acute;-d(T)<SUB>12</SUB></I>VN<I>-3&acute;, where </I>V<I> is either C, A, 
      or G, and </I>N<I> is C, T, A, or G. For example, one primer in the series 
      is 5&acute;-d(T)<SUB>12</SUB>CC-3&acute;, the next is 5&acute;-d(T)<SUB>12</SUB>CT-3&acute;, 
      etc.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Arbitrary 5&acute; 
      oligodeoxynucleotide primers (50 µg/ml) in 10 mM Tris-Cl (pH 7.6), 0.1 mM 
      EDTA</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Sixteen 
      arbitrary 5&acute; oligonucleotide primers are required, each ten nucleotides in 
      length. The sequence of each primer is chosen at random, but it should 
      contain approximately equal numbers of the four bases, with a balanced 
      distribution of G and C residues, and a low propensity to form stable 
      secondary structures.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Total RNA (100 
      µg/ml)</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Total RNA 
      extracted from cells with chaotropic agents is generally the template of 
      choice for DD-PCR. RNAs to be compared by DD-PCR should be prepared in an 
      identical fashion. Poly(A)<SUP>+</SUP> RNA is not ideal as a template in 
      differential display.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Radioactive Compounds</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Radiolabeled 
      dATP (10 µCi/µl, sp. act. 3000 Ci/mmole)</FONT> <IMG height=11 alt=caution 
      hspace=4 src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>DNA labeled 
      with <IMG src="../izo/alpha.gif" border=0>-<SUP>35</SUP>S or <IMG 
      src="../izo/alpha.gif" border=0>-<SUP>33</SUP>P generates sharper bands 
      on autoradiographs than does DNA labeled with <IMG 
      src="../izo/alpha.gif" border=0>-<SUP>32</SUP>P.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Additional Reagents</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Steps 10 and 
      13 of this protocol require the reagents listed in <A href="../12/10.htm">Chapter 
      12, Protocol 10</A> and <A href="../12/11.htm">Chapter 
      12, Protocol 11</A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 22 of 
      this protocol requires the reagents listed in <A href="05.htm">Chapter 
      8, Protocol 5 </A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 23 of 
      this protocol requires the reagents listed in <A href="12.htm">Chapter 
      8, Protocol 12 </A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 24 of 
      this protocol requires the reagents listed in <A href="../07/08.htm">Chapter 
      7, Protocol 8 </A>or <A href="../07/11.htm">Chapter 
      7, Protocol 11</A>, and <A href="15.htm">Chapter 
      8, Protocol 15</A>.</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 
  border=0></TD></TR></TBODY></TABLE><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" 
size=3><STRONG>METHOD</STRONG></FONT> <BR><FONT face="arial,verdana, helvetica" 
color=#000000 size=2>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>In sterile 
  0.5-ml microfuge tubes, set up a series of trial reactions to establish the 
  optimum concentrations of "control" and "test" RNAs required to produce a 
  pattern of 100-300 amplified cDNA bands after gel electrophoresis and 
  autoradiography. Make fivefold serial dilutions in H<SUB>2</SUB>O of the RNA 
  preparations to produce concentrations of between 1 µg/ml and 100 µg/ml. 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Choose one or 
  more primers from the collection of anchored 3&acute; oligonucleotides and set up a 
  series of annealing reactions that contain different amounts of diluted RNA 
  templates: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>template 
        RNA</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>8.0 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>anchored 3&acute; 
        oligonucleotide primer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2.0 
    µl</FONT></TD></TR></TBODY></TABLE>Incubate the reactions for 10 minutes at 
  65°C and then place them in a 37°C water bath.<BR><I>The total amount of RNA 
  in the annealing reactions should vary between 8 ng and 800 ng.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Add the 
  following to the annealing reactions: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5x DD-PCR 
        reverse transcriptase buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>4 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>100 mM 
        dithiothreitol</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>200 µM solution 
        of four dNTPs</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>approx. 25 
        units/µl placental RNase inhibitor</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>0.25 
      µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>200 units/µl 
        reverse transcriptase</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>0.25 
      µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>to 20 
      µl</FONT></TD></TR></TBODY></TABLE>Incubate the tubes at 37°C for 1 
  hour.<BR><I>To test for contaminating genomic DNA, set up one or more control 
  reactions that contain no reverse transcriptase enzyme and carry these through 
  Step 10 of the protocol. If necessary, the RNA preparation can be treated with 
  RNase-free DNase I either as a separate step during purification or in the 
  same reaction tube that will later be used to synthesize cDNA.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Inactivate the 
  reverse transcriptase by incubating the reaction mixtures for 10 minutes at 
  94°C.<BR><I>Steps 3 and 4 can be carried out in a thermal cycler programmed 
  with a single cycle of 37°C for 1 hour/94°C for 10 minutes, followed by a 4°C 
  hold.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Set up two 
  series of eight 0.5-ml amplification tubes. Each tube should contain: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x 
        amplification buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>anchored 3&acute; 
        oligonucleotide primer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 mM solution 
        of four dNTPs (pH 8.0)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>[<IMG 
        src="../izo/alpha.gif" border=0>-<SUP>33</SUP>P]dATP or [<IMG 
        src="../izo/alpha.gif" border=0>-<SUP>35</SUP>S]dATP (3000 
        Ci/mmole)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>9 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 units/µl 
        thermostable DNA polymerase</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    unit</FONT></TD></TR></TBODY></TABLE>To each tube, add 2 µl of a different 
  arbitrary 5&acute; primer. Mix the contents by tapping the sides of the 
  tubes.<BR><I>Wherever possible, use the 10x amplification buffer supplied by 
  the manufacturer of the </I>Taq<I> DNA polymerase.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Into one series 
  of eight tubes, dispense approx. 3-µl aliquots of the reverse transcriptase 
  reaction containing the test RNA. Into the other series of eight tubes 
  dispense approx. 3-µl aliquots of the reverse transcriptase reaction 
  containing the preparation of control RNA. Close the tubes and mix the 
  contents gently. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>If the thermal 
  cycler is not fitted with a heated lid, overlay the reaction mixtures with 1 
  drop (approx. 50 µl) of light mineral oil. Place the tubes in the thermal 
  cycler. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Amplify the 
  nucleic acids using the denaturation, annealing, and polymerization times and 
  temperatures listed in the table. 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2><B>Cycle 
        Number</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Denaturation</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Annealing</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Polymerization</B></FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>30 
      cycles</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>15 sec at 
        94°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>30 sec at 
        42°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>15 sec at 
        72°C</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Last 
        cycle</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>15 sec at 
        94°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>30 sec at 
        42°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 min at 
        72°C</FONT></TD></TR></TBODY></TABLE><I>Times and temperatures may need to be 
  adapted to suit the particular reaction conditions.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>At the end of 
  the program, remove the tubes from the thermal cycler and add 5 µl of 5x 
  formamide loading buffer to each. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Separate the 
  radiolabeled products of the reactions by electrophoresis through an 
  electrolyte gradient polyacrylamide gel of the type used for DNA sequencing. 
  Electrophoresis is carried at constant electrical power until the xylene 
  cyanol tracking dye has migrated about two thirds of the length of the gel 
  (please see <A href="../12/10.htm">Chapter 
  12, Protocol 10</A> and <A href="../12/11.htm">Chapter 
  12, Protocol 11</A>). Dry the gel and expose it to autoradiographic film. 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Examine the 
  pattern of DNA bands arising from reactions containing different 
  concentrations of control and test RNAs. A good differential display contains 
  between 100 and 250 well-resolved bands. Select the concentration of test and 
  control RNAs that work well with the largest number of primer pairs. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Repeat the 
  annealing, reverse transcriptase, and amplification reactions using all 
  combinations of primer pairs and the optimum amount of RNA templates. Set up 
  the reactions in 96-well microtiter plates designed for use in a thermal 
  cycler. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Separate the 
  products of the amplification reactions by electrophoresis through 
  polyacrylamide sequencing gels, as in Steps 9 and 10.<BR><I>Load the reactions 
  generated with each primer pair in adjacent lanes on the gel, i.e., load the 
  reaction obtained with primer pair A + B from one RNA preparation next to the 
  reaction obtained with primer pair A + B from the other RNA preparation.</I> 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Compare the 
  patterns of bands obtained with each primer pair from the different RNA 
  populations.<BR><I>When a differentially expressed band is identified, it is 
  advisable to repeat the experiment to make sure that the initial finding is 
  reproducible. Ideally, different batches of the two RNAs should be used, 
  although this precaution may not always be practicable.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Recover target 
  bands from the dried polyacrylamide gel. Lay the autoradiogram on top of the 
  gel and use a soft pencil to lightly mark the position of the desired band on 
  the autoradiogram. Cutting through the autoradiogram with a clean razor blade, 
  excise each target band and the attached Whatman 3MM paper. Soak each sliver 
  of dried gel/paper overnight at room temperature in a separate 0.5-ml 
  microfuge tube containing 50 µl of sterile H<SUB>2</SUB>O. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Puncture the 
  bottom of each 0.5-ml tube with a small-gauge needle. Place each punctured 
  tube inside a 1.5-ml microfuge tube. Centrifuge for 20 seconds to transfer the 
  eluate to the larger tube. Discard the amplification tube containing the 
  residue of the Whatman 3MM paper and polyacrylamide. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Amplify the 
  eluted fragment in a reaction containing the following: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x 
        amplification buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>DNA eluted from 
        polyacrylamide gel</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>3 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>arbitrary 5&acute; 
        oligonucleotide primer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>anchoring 3&acute; 
        oligonucleotide primer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20 mM solution 
        of four dNTPs (pH 7.0)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>9.5 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5 units/µl 
        <I>Taq</I> thermostable DNA polymerase</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 
      units</FONT></TD></TR></TBODY></TABLE><I>Wherever possible, use the 10x 
  amplification buffer supplied by the manufacturer of the </I>Taq<I> DNA 
  polymerase.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>If the thermal 
  cycler is not fitted with a heated lid, overlay the reaction mixtures with 1 
  drop (approx. 50 µl) of light mineral oil. Place the tubes in a thermal 
  cycler. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Amplify the 
  nucleic acids using the denaturation, annealing, and polymerization times and 
  temperatures listed in the table. 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2><B>Cycle 
        Number</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Denaturation</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Annealing</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2><B>Polymerization</B></FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>30 
      cycles</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>15 sec at 
        94°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>30 sec at 
        42°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>15 sec at 
        72°C</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Last 
        cycle</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>15 sec at 
        94°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>30 sec at 
        42°C</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>2 min at 
        72°C</FONT></TD></TR></TBODY></TABLE><I>Polymerization should be carried out 
  for 1 minute for every 1000 bp of length of the target DNA.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Estimate the 
  concentration of the reamplified DNA fragment by electrophoresis of 5-10% of 
  the reaction through a 1% (w/v) agarose gel. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>If mineral oil 
  was used to overlay the reaction (Step 18), remove the oil from the sample by 
  extraction with 150 µl of chloroform.<BR><I>The aqueous phase, which contains 
  the amplified DNA, will form a micelle near the meniscus. The micelle can be 
  transferred to a fresh tube with an automatic 
  micropipette.</I><BR><B>IMPORTANT</B><I> Do not attempt chloroform extractions 
  in microtiter plates. The plastic used in these plates is not resistant to 
  organic solvents.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Ligate the DNA 
  into a vector that has been tailed with dT (e.g., pGEM T vector from Promega) 
  (please see <A href="05.htm">Chapter 
  8, Protocol 5 </A>) and transform <I>E. coli</I> with aliquots of the ligation 
  reaction. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Isolate plasmid 
  DNA from six or more recombinants and compare the sizes of the inserts 
  released by restriction enzyme digestion.<BR><I>The sequence of the insert DNA 
  can be established by using universal primers that bind to the flanking 
  regions of the vector. These oligonucleotides can also be used as primers to 
  check the size of the inserts by PCR (please see <A href="12.htm">Chapter 
  8, Protocol 12 </A>).</I><BR><I>It is important to isolate and sequence more 
  than one plasmid recombinant from the ligation reaction. Compare the cDNA 
  sizes and sequences to each other and to those in the various databases.</I> 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Confirm the 
  differential expression of a candidate cDNA/mRNA in as many ways as possible, 
  including northern hybridization (<A href="../07/08.htm"> 
  Chapter 7, Protocol 8 </A>), RNase protection (<A href="../07/11.htm"> 
  Chapter 7, Protocol 11 </A>), or quantitative PCR (<A href="15.htm"> 
  Chapter 8, Protocol 15 </A>). In situ mRNA hybridization can be used to 
  localize the transcript to a diseased or developing tissue. 
</FONT></LI></OL></FONT><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>RECIPES</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/circle.gif" width=12 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Amplification Buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>500 mM KCl</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM Tris-Cl (pH 8.3 at 
      room temperature)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>15 mM 
      MgCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Autoclave the 10x buffer 
      for 10 minutes at 15 psi (1.05 kg/cm <SUP>2</SUP>) on liquid cycle. Divide 
      the sterile buffer into aliquots and store them at 
  -20°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>DD-PCR reverse transcriptase 
      buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>250 mM Tris-Cl (pH 
      8.3)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>375 mM KCl</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>15 mM 
      MgCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Autoclave the 5x buffer 
      for 10 minutes at 15 psi (1.05 kg/cm<SUP>2</SUP>) on liquid cycle. Divide 
      the sterile buffer into aliquots. Store the aliquots at 
  -20&ordm;C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>DTT</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 1 M 
      solution: Dissolve 3.09 g of dithiothreitol in 20 ml of 0.01 M sodium 
      acetate (pH 5.2) and sterilize by filtration. Dispense into 1-ml aliquots 
      and store at -20°C. Under these conditions, dithiothreitol is stable to 
      oxidation by air.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>EDTA</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare EDTA at 0.5 M 
      (pH 8.0): Add 186.1 g of disodium EDTA•2H<SUB>2</SUB>O to 800 ml of 
      H<SUB>2</SUB>O. Stir vigorously on a magnetic stirrer. Adjust the pH to 
      8.0 with NaOH (approx. 20 g of NaOH pellets). Dispense into aliquots and 
      sterilize by autoclaving. The disodium salt of EDTA will not go into 
      solution until the pH of the solution is adjusted to approx. 8.0 by the 
      addition of NaOH.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Formamide</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Deionize the formamide by 
      stirring on a magnetic stirrer with Dowex XG8 mixed bed resin for 1 hour 
      and filtering it twice through Whatman No. 1 paper. Store deionized 
      formamide in small aliquots under nitrogen at 
-70°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Formamide Loading Buffer</STRONG></FONT> 
  </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>80% (w/v) deionized 
      formamide</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>10 mM EDTA (pH 
    8.0)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>1 mg/ml xylene cyanol 
      FF</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>1 mg/ml bromophenol 
      blue</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Purchase a distilled 
      deionized preparation of formamide and store in small aliquots under 
      nitrogen at -20°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>KCl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Dissolve an appropriate 
      amount of solid KCl in H<SUB>2</SUB>O, autoclave for 20 minutes on liquid 
      cycle and store at room temperature. Ideally, this 4 M solution should be 
      divided into small (approx. 100 µl) aliquots in sterile tubes and each 
      aliquot thereafter used one time.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Cl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve 121.1 g of Tris base in 800 ml of H<SUB>2</SUB>O. Adjust the 
      pH to the desired value by adding concentrated HCl.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2><I>pH</I></FONT> </TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2><I>HCl</I></FONT> 
        </TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.4 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>70 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.6 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>8.0 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>42 ml 
        </FONT></TD></TR></TBODY></TABLE>
      <P>(1 M) Allow the solution to cool to room temperature before making 
      final adjustments to the pH. Adjust the volume of the solution to 1 liter 
      with H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
      autoclaving.</P>
      <P>If the 1 M solution has a yellow color, discard it and obtain Tris of 
      better quality. The pH of Tris solutions is temperature-dependent and 
      decreases approx. 0.03 pH units for each 1°C increase in temperature. For 
      example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5°C, 
      25°C, and 37°C, respectively.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>dNTP Solution</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve each dNTP (deoxyribonucleoside triphosphates) in 
      H<SUB>2</SUB>O at an approximate concentration of 100 mM. Use 0.05 M Tris 
      base and a micropipette to adjust the pH of each of the solutions to 7.0 
      (use pH paper to check the pH). Dilute an aliquot of the neutralized dNTP 
      appropriately, and read the optical density at the wavelengths given in 
      the table below. Calculate the actual concentration of each dNTP. Dilute 
      the solutions with H<SUB>2</SUB>O to a final concentration of 50 mM dNTP. 
      Store each separately at -70°C in small aliquots. For polymerase chain 
      reactions (PCRs), adjust the dNTP solution to pH 8.0 with 2 N NaOH. 
      Commercially available solutions of PCR-grade dNTPs require no 
      adjustment.</P>
      <P>
      <TABLE width=400>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>Base</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana 
            size=2>wavelength<BR>(nm)</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>Extinction 
            Coefficient<BR>(E)&nbsp;(M<SUP>-1</SUP>cm<SUP>-1</SUP>)</FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>A</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>259</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.54 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>G</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>253</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>1.37 x 
            10<SUP>4</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>C</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>271</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.10 x 
            10<SUP>3</SUP></FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>T</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>267</FONT></TD>
          <TD><FONT face=arial,helvetica,verdana size=2>9.60 x 
            10<SUP>3</SUP></FONT></TD></TR></TBODY></TABLE></P>
      <P>For a cuvette with a path length of 1 cm, absorbance = EM. 100 mM stock 
      solutions of each dNTP are commercially available 
    (Pharmacia).</P></FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;     helvetica" size=3><STRONG>CAUTIONS</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" width=11 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Formamide </STRONG></FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Formamide is teratogenic. The vapor is irritating to the eyes, 
      skin, mucous membranes, and upper respiratory tract. It may be harmful by 
      inhalation, ingestion, or skin absorption. Wear appropriate gloves and 
      safety glasses. Always use in a chemical fume hood when working with 
      concentrated solutions of formamide. Keep working solutions covered as 
      much as possible. </FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Radioactive substances</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Radioactive substances: When planning an experiment that involves 
      the use of radioactivity, consider the physico-chemical properties of the 
      isotope (half-life, emission type, and energy), the chemical form of the 
      radioactivity, its radioactive concentration (specific activity), total 
      amount, and its chemical concentration. Order and use only as much as 
      needed. Always wear appropriate gloves, lab coat, and safety goggles when 
      handling radioactive material. X-rays and gamma rays are electromagnetic 
      waves of very short wavelengths either generated by technical devices or 
      emitted by radioactive materials. They might be emitted isotropically from 
      the source or may be focused into a beam. Their potential dangers depend 
      on the time period of exposure, the intensity experienced, and the 
      wavelengths used. Be aware that appropriate shielding is usually made of 
      lead or other similar material. The thickness of the shielding is 
      determined by the energy(s) of the X-rays or gamma rays. Consult the local 
      safety office for further guidance in the appropriate use and disposal of 
      radioactive materials. Always monitor thoroughly after using 
      radioisotopes. A convenient calculator to perform routine radioactivity 
      calculations can be found at: <A target=_blank 
      href="http://www.graphpad.com/calculators/radcalc.cfm">http://www.graphpad.com/calculators/radcalc.cfm</A>. 
      </FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>REFERENCES</STRONG> 
</FONT><BR>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Liang P. and 
  Pardee A.B. 1992. Differential display of eukaryotic messenger RNA by means of 
  the polymerase chain reaction. <I>Science</I> 257:967-971. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Liang P. and 
  Pardee A.B. 1997. Differential display. A general protocol. <I>Methods Mol. 
  Biol.</I> 85:3-11. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Welsh J., Chada 
  K., Dalal S.S., Cheng R., Ralph D., and McClelland M. 1992. Arbitrarily primed 
  PCR fingerprinting of RNA. <I>Nucleic Acids Res.</I> 20:4965-4970. 
</FONT></LI></OL><BR><BR><BR><BR><BR><FONT face=Arial,Verdana,Helvetica 
size=1><STRONG>Copyright © 2000 by Cold Spring Harbor Laboratory Press. All 
rights reserved.</STRONG><BR>No part of these pages, either text or image may be 
used for any purpose other than personal use. Therefore, reproduction 
modification, storage in a retrieval system or retransmission, in any form or by 
any means, electronic, mechanical, or otherwise, for reasons other than personal 
use, is strictly prohibited without prior written 
permission.</FONT><BR><BR>
</td></tr></table>
</BODY></HTML>
