<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Molecular Cloning | Protocol 7 Print Version</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1251"><!--================== Designed and Developed by WriteDesign =================--><!--======================= Powered by RomeBlack, Inc. =======================--><!--=== Copyright 2000, 2001 Cold Spring Harbor Labs. All Rights Reserved. ===-->
<META content="" name=keywords>
<META 
content="Plaques formed by M13 bacteriophages or bacterial colonies transformed by plasmids carrying specific mutations can be detected by hybridization, using a radiolabeled oligonucleotide that forms a perfect duplex with the mutant sequence. Hybridization is carried out under conditions of low stringency that allow the radiolabeled oligonucleotide to anneal to both mutant and wild-type DNAs. A hybrid between the radiolabeled oligonucleotide and the wild-type sequence will contain one or more mismatched base pairs, whereas the hybrid formed between the newly created mutant and the oligonucleotide probe will be perfectly matched. These two types of hybrids usually differ in their thermal stabilities, with mismatched hybrids dissociating at a lower temperature than the corresponding perfect hybrid. Plaques or colonies that continue to hybridize to the probe after several cycles of washing at progressively higher temperatures are likely to contain recombinants carrying the desired mutation." 
name=description>
<META content="MSHTML 5.50.4134.600" name=GENERATOR></HEAD>
<BODY bgColor=#ffffff background=../izo/fon.gif>
<table width=600 border="0" cellspacing="0" cellpadding="0"><tr><td width=600><IMG height=49 hspace=4 src="../izo/logo_csh.gif" 
width=169 align=left vspace=4><IMG height=49 hspace=4 
src="../izo/print_header.gif" width=293 align=left vspace=4><BR 
clear=all><BR><FONT face="arial,verdana, helvetica" size=4><STRONG>Chapter 13, 
Protocol 7</STRONG></FONT><BR><BR clear=all><FONT face=arial,verdana,helvetica 
size=3><STRONG>Screening Recombinant Clones for Site-directed Mutagenesis by 
Hybridization to Radiolabeled Oligonucleotides</STRONG></FONT><BR><BR><FONT 
face="arial,verdana, helvetica" size=2>Plaques formed by M13 bacteriophages or 
bacterial colonies transformed by plasmids carrying specific mutations can be 
detected by hybridization, using a radiolabeled oligonucleotide that forms a 
perfect duplex with the mutant sequence. Hybridization is carried out under 
conditions of low stringency that allow the radiolabeled oligonucleotide to 
anneal to both mutant and wild-type DNAs. A hybrid between the radiolabeled 
oligonucleotide and the wild-type sequence will contain one or more mismatched 
base pairs, whereas the hybrid formed between the newly created mutant and the 
oligonucleotide probe will be perfectly matched. These two types of hybrids 
usually differ in their thermal stabilities, with mismatched hybrids 
dissociating at a lower temperature than the corresponding perfect hybrid. 
Plaques or colonies that continue to hybridize to the probe after several cycles 
of washing at progressively higher temperatures are likely to contain 
recombinants carrying the desired mutation.</FONT> <BR><BR>
<TABLE border=1>
  <TBODY>
  <TR>
    <TD width=700><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" 
      width=11 border=0><FONT face=arial,verdana,helvetica 
      size=2><STRONG>CAUTION</STRONG></FONT><BR><IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0><FONT 
      face="arial,verdana, helvetica" size=2> <STRONG>RECIPE</STRONG></FONT> 
  </TD></TR></TBODY></TABLE><BR><BR><FONT face=arial,verdana,helvetica 
size=3><STRONG>MATERIALS</STRONG></FONT><BR><BR>
<TABLE cellPadding=0 cellborder="0">
  <TBODY>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Buffers and Solutions</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Ammonium 
      formate (0.2 M)</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Oligonucleotide hybridization solution (13-7)</FONT> <IMG height=12 
      alt="" hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" 
      size=2>Oligonucleotide prehybridization solution (13-7)</FONT> <IMG 
      height=12 alt="" hspace=4 src="../izo/circle.gif" width=12 border=0> 
  </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>6x SSC</FONT> 
      <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" width=12 
      border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>6x SSC 
      should be warmed to various temperatures. Please see Steps 11, 12, and 
      13.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>TE (pH 
      7.6)</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Enzymes and Buffers</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Bacteriophage 
      T4 polynucleotide kinase</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>10x 
      Bacteriophage T4 polynucleotide kinase buffer</FONT> <IMG height=12 alt="" 
      hspace=4 src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Restriction 
      enzymes</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Please see 
      Steps 18 and 19.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Nucleic Acids and Oligonucleotides</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Mutagenic 
      oligonucleotide (10 pmoles/µl)</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Radioactive Compounds</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>[<IMG 
      src="../izo/gamma.gif">-<SUP>32</SUP>P]ATP (&gt;5000 Ci/mmole, 10 
      mCi/ml)</FONT> <IMG height=11 alt=caution hspace=4 
      src="../izo/triangle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Media</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>YT agar 
      plates</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>2x YT top 
      agar</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Additional Reagents</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 4 of this 
      protocol requires the reagents listed in <A href="../10/04.htm">Chapter 
      10, Protocol 4</A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 14 of 
      this protocol requires the reagents listed in <A href="../03/04.htm">Chapter 
      3, Protocol 4</A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Steps 15 and 
      16 of this protocol require the reagents listed in <A href="../12/03.htm">Chapter 
      12, Protocol 3</A>, <A href="../12/04.htm">Chapter 
      12, Protocol 4</A>, or <A href="../12/05.htm">Chapter 
      12, Protocol 5</A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 17 of 
      this protocol requires the reagents listed in <A href="../03/03.htm">Chapter 
      3, Protocol 3</A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 20 of 
      this protocol requires the reagents listed in <A href="../06/08.htm">Chapter 
      6, Protocol 8</A> and <A href="../06/10.htm">Chapter 
      6, Protocol 10.</A></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Vectors and Bacterial Strains</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Bacteriophage 
      M13 recombinants, previously mutagenized</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Please use 
      plates containing bacteriophage M13 plaques generated in <A href="../13/02.htm">Chapter 
      13, Protocol 2</A>.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Bacteriophage 
      M13 recombinants, nonmutagenized</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>These serve 
      as a negative control; please see Step 5.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>E. coli</I> 
      strain TG1 or equivalent</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 
  border=0></TD></TR></TBODY></TABLE><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" 
size=3><STRONG>METHOD</STRONG></FONT> <BR><FONT face="arial,verdana, helvetica" 
color=#000000 size=2>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>In a sterile 
  microfuge tube, mix: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>mutagenic 
        oligonucleotide (10 pmoles/µl)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10x 
        bacteriophage T4 polynucleotide kinase buffer</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10 mCi/ml [<IMG 
        src="../izo/gamma.gif">-<SUP>32</SUP>P]ATP (10-50 pmoles)</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
        size=2>H<SUB>2</SUB>O</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>6 
    µl</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5-10 units/µl 
        bacteriophage T4 polynucleotide kinase</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1 
    µl</FONT></TD></TR></TBODY></TABLE>Incubate the reaction mixture for 30 
  minutes at 37°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Dilute the 
  reaction mixture to 100 µl by the addition of 90 µl of TE (pH 7.6) and 
  inactivate the polynucleotide kinase by heating for 10 minutes at 68°C. 
</FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Measure the 
  efficiency of transfer of <SUP>32</SUP>P to the oligonucleotide and estimate 
  its specific activity by chromatography on DE-81 paper as follows: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>a.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Cut a strip of 
        Whatman DE-81 paper approx. 1 cm wide and 7-10 cm long. With a soft-lead 
        pencil, draw a fine line across the strip approx. 1.5 cm from one end. 
        This line marks the origin of the chromatogram.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>b.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Spot 1.0 µl of 
        the diluted phosphorylation reaction at the origin. Fill a 250-ml beaker 
        to a depth of approx. 0.5 cm with approx. 25-50 ml of 0.2 M ammonium 
        formate. Place the DE-81 strip vertically in a beaker so that the 
        radioactive sample(s) at the origin is just above the buffer solution. 
        Cover the beaker with a glass plate or aluminum foil and allow the 
        chromatogram to develop until the solvent front has migrated almost to 
        the top of the beaker.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>c.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Wrap the strip 
        of DE-81 paper in Saran Wrap and subject it to a very brief period of 
        autoradiography. Use the developed X-ray film as a guide to cut out the 
        radioactive region at the solvent front, the origin, and any other 
        region that contains radioactivity. Measure the amount of radioactivity 
        in each section in a scintillation 
  counter.</FONT></TD></TR></TBODY></TABLE></FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 
  size=2>(<I>Optional</I>) Remove unincorporated radiolabel from the 
  oligonucleotide by precipitation with cetylpyridinium bromide as described in 
  <A href="../10/04.htm">Chapter 
  10, Protocol 4</A>.<BR><I>This step is necessary only when background 
  hybridization is a persistent problem. Under normal circumstances, the 
  unfractionated reaction mixture may be used as a probe.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Prepare 
  replicas of the bacteriophage M13 plaques that are to be screened with the 
  radiolabeled oligonucleotide as follows: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>a.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Transfer plates 
        containing 100-500 plaques to 4°C for at least 30 minutes.<BR><I>Include 
        at least one plate that contains plaques of the original wild-type 
        recombinant bacteriophage M13. This plate serves as a negative control 
        in the hybridization and washing steps.</I></FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>b.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>When the plates 
        are thoroughly chilled, remove them from the cold room and immediately 
        lay a numbered, dry nitrocellulose or nylon filter on the agar surface 
        of each plate. Use an 18-gauge hypodermic needle to make a series of 
        holes in each filter and the underlying agar. These holes will later 
        serve to key the filters to the plates.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>c.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>After 30 
        seconds to 4 minutes, use blunt-ended forceps to peel each filter 
        carefully from its plate. Spread out all of the filters (plaque side up) 
        on a pad of paper towels. Wrap the plates in Saran Wrap, and store them 
        at 4°C until they are needed.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>d.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>When the 
        filters have dried (approx. 30 minutes at room temperature), bake them 
        for 1 hour at 80°C in a vacuum oven.</FONT></TD></TR></TBODY></TABLE></FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Transfer all of 
  the filters to a heat-sealable plastic bag (e.g., Sears Seal-A-Meal), or to an 
  evaporating dish of the appropriate diameter, or to a hybridization roller 
  bottle. Add oligonucleotide prehybridization solution (13-7) (approx. 10 
  ml/82-mm filter in bags or dishes; 5 ml/82-mm filter in bottles). Seal the 
  bag, cover the evaporating dish with Saran Wrap, or cap the roller bottle, and 
  incubate the filters for 1-2 hours at 65°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Discard the 
  prehybridization solution, and replace it with oligonucleotide hybridization 
  solution (13-7) (approx. 5 ml/82-mm filter). Reseal the bag, recover the 
  evaporating dish, or recap the hybridization roller bottle, and incubate the 
  hybridization reaction for 4-6 hours at the appropriate 
  temperature.<BR><I>Perform hybridization with the radiolabeled oligonucleotide 
  at a temperature 5-10°C below the </I>T<I><SUB>m</SUB> estimated from the 
  following formula:</I><BR>T<I><SUB>m</SUB></I> = 4(<I>G</I>+<I>C</I>) + 
  2(<I>A</I>+<I>T</I>)</I><BR><I>where (<I>G</I>+<I>C</I>) is the sum of G and C 
  residues in the oligonucleotide, and where (<I>A</I>+<I>T</I>) is the sum of 
  the A and T residues in the oligonucleotide.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>At the end of 
  the hybridization period, quickly transfer the filters to a tray containing 
  200-300 ml of 6x SSC at room temperature. Cover the tray with Saran Wrap, and 
  place it on a rotating shaker for 15 minutes. Replace the washing fluid every 
  5 minutes. Meanwhile, transfer the remainder of the radioactive hybridization 
  solution from the bag, dish, or roller bottle to a disposable plastic tube. 
  Close the tube tightly, and store the radioactive solution at -20°C until it 
  is needed for rescreening positive plaques (Step 13). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>At the end of 
  the washing period, quickly transfer the filters to a piece of Saran Wrap 
  stretched on the bench. Cover the filters with another piece of Saran Wrap. 
  Fold the edges of the two pieces of Saran Wrap together to form a tight seal. 
  Apply adhesive dot labels marked with radioactive or chemiluminescent ink to 
  the outside of the package, and generate an autoradiograph by exposing the 
  package of filters to X-ray film for 1-2 hours at -70°C, using an intensifying 
  screen.<BR><B>IMPORTANT</B><I> Do not allow the filters to dry on the Saran 
  Wrap.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Compare the 
  pattern of hybridization with the distribution of plaques. At this stage, it 
  is normal to find that virtually every plaque hybridizes to the probe. 
  Typically, however, some plaques hybridize more strongly than others, and 
  these often turn out to be the plaques carrying the desired mutation. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Transfer the 
  filters to a plastic box containing 100-200 ml of 6x SSC that has been warmed 
  to 10°C below the <I>T<SUB>m</SUB></I>. Agitate the filters in the solution 
  for 2 minutes (please see note below), and then transfer them to a piece of 
  Saran Wrap as described in Step 9. Establish another autoradiograph. At this 
  stage, it is often possible to identify two types of plaques: those whose 
  radioactive signal has decreased in intensity and those that show no change in 
  intensity. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Repeat the 
  cycles of washing and autoradiography, increasing the temperature of the 
  washing solution by 2-10°C in each cycle. The aim is to find a temperature 
  that does not markedly affect perfect hybrids but causes dissociation of 
  mismatched hybrids (such as those formed between the mutagenic oligonucleotide 
  and the original wild-type sequence). </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Positively 
  hybridizing plaques usually contain a mixture of both mutant and wild-type 
  sequences. It is therefore <I>essential</I> to plaque-purify the 
  bacteriophages from positively hybridizing plaques as follows: 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>a.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Touch the blunt 
        end of a sterile, disposable wooden toothpick to the surface of a 
        positively hybridizing plaque.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>b.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Drop the 
        toothpick into a sterile tube containing 1 ml of sterile TE (pH 7.6). 
        Store the tube for 10-15 minutes at room temperature, shaking it from 
        time to time to dislodge bacteriophage particles.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>c.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Make a series 
        of tenfold dilutions of the bacteriophage suspension with TE (pH 7.6). 
        Mix 10 µl of the 10<SUP>-3</SUP>, 10<SUP>-4</SUP>, and 10<SUP>-5</SUP> 
        dilutions with 100-µl aliquots of an overnight culture of an appropriate 
        strain of <I>E. coli</I> (e.g., TG1).</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>d.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Add 2.5 ml of 
        2x YT top agar (melted and cooled to 45°C) to each culture, and plate 
        the entire mixture on a single YT agar plate. Incubate the plate for 16 
        hours at 37°C to allow plaques to form.</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>e.</FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Rescreen the 
        plaques with the radiolabeled oligonucleotide as described in Steps 
        5-12. In this second round of screening, there is no need to increase 
        the temperature of the washing solution in a stepwise fashion. Instead, 
        the filters can be transferred directly from the washing solution at 
        room temperature (Step 8) to 6x SSC previously warmed to the 
        discriminatory temperature found empirically in Step 
    12.</FONT></TD></TR></TBODY></TABLE></FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Pick two 
  plaques from each of three independent putative mutants. Prepare 
  single-stranded DNA from small-scale cultures infected with bacteriophages 
  derived from each of these plaques as described in <A href="../03/04.htm">Chapter 
  3, Protocol 4</A>. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Carry out DNA 
  sequencing by the dideoxy-mediated chain-termination method (please see <A href="../12/03.htm">Chapter 
  12, Protocol 3</A> or <A href="../12/04.htm">Chapter 
  12, Protocol 4</A>) through the region containing the target sequence. Use 
  either a universal sequencing primer or a custom-synthesized primer that binds 
  50-100 nucleotides upstream of the mutation site. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>When the 
  presence of the mutation has been confirmed, verify the sequence of the entire 
  region of the target DNA cloned in the bacteriophage M13 vector to ensure that 
  no adventitious mutations have been generated during propagation of the 
  recombinant in bacteriophage M13. Often, this requires the synthesis of 
  custom-designed sequencing primers that are complementary to segments of the 
  target DNA spaced approx. 200-400 bp apart. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Isolate 
  bacteriophage M13 replicative form DNA from a culture infected with 
  plaque-purified recombinant bacteriophages (Step 14) that carry the desired 
  mutation and show no other changes in sequence in the target region. For 
  methods to isolate and purify bacteriophage M13 replicative form DNA, please 
  see <A href="../03/03.htm">Chapter 
  3, Protocol 3</A>. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Recover the 
  mutated target sequence by digestion of bacteriophage M13 replicative form DNA 
  with the appropriate restriction enzyme(s) and preparative gel 
  electrophoresis. Clone the target DNA into the desired vector. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Use several 
  different restriction enzymes to digest aliquots of either a recombinant that 
  carries the original (nonmutagenized) target sequence or the recombinant that 
  carries the mutagenized target sequence. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Separate the 
  resulting fragments by gel electrophoresis, and transfer them to a solid 
  support (e.g., nitrocellulose or nylon membrane) as described in <A href="../06/08.htm">Chapter 
  6, Protocol 8</A>. Carry out Southern hybridization at 10°C below the 
  <I>T<SUB>m</SUB></I>, using the <SUP>32</SUP>P-labeled mutagenic 
  oligonucleotide as a probe. Wash the filter under the discriminatory 
  conditions and establish an autoradiograph.<BR><I>The final autoradiograph 
  should show hybridization only to the relevant fragments of the mutagenized 
  target DNA.</I> </FONT></LI></OL></FONT><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>RECIPES</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/circle.gif" width=12 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Bacteriophage T4 Polynucleotide Kinase 
      Buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>700 mM Tris-Cl (pH 
      7.6)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM 
      MgCl<SUB>2</SUB></FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>50 mM 
      dithiothreitol</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Divide the 10x stock into 
      small aliquots and store frozen at -20°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Denhardt's Reagent</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>1% (w/v) Ficoll 
    400</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>1% (w/v) 
      polyvinylpyrrolidone</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>1% (w/v) bovine serum 
      albumin (Sigma, Fraction V) in H<SUB>2</SUB>O</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Denhardt's reagent is 
      usually made up as a 50x stock solution, which is filtered and stored at 
      -20°C. The stock solution is diluted tenfold into prehybridization buffer 
      (usually 6x SSC or 6x SSPE containing 1.0% SDS and 100 µ;g/ml denatured 
      salmon sperm DNA). 50x Denhardt's reagent contains in 
      H<SUB>2</SUB>O.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>EDTA</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare EDTA at 0.5 M 
      (pH 8.0): Add 186.1 g of disodium EDTA•2H<SUB>2</SUB>O to 800 ml of 
      H<SUB>2</SUB>O. Stir vigorously on a magnetic stirrer. Adjust the pH to 
      8.0 with NaOH (approx. 20 g of NaOH pellets). Dispense into aliquots and 
      sterilize by autoclaving. The disodium salt of EDTA will not go into 
      solution until the pH of the solution is adjusted to approx. 8.0 by the 
      addition of NaOH.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Ficoll 400 (20% w/v)</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Dissolve the Ficoll in 
      sterile H<SUB>2</SUB>O and store the solution frozen in 100-µl aliquots at 
      -20°C.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Media Containing Agar or 
      Agarose</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Prepare liquid media according to the recipes given. Just before 
      autoclaving, add one of the following:</P>
      <P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face="arial, helvetica,verdana" size=2>Bacto Agar (for 
            plates)</FONT></TD>
          <TD><FONT face="arial, helvetica,verdana" size=2>15 
        g/liter</FONT></TD></TR>
        <TR>
          <TD><FONT face="arial, helvetica,verdana" size=2>Bacto Agar (for top 
            agar)</FONT></TD>
          <TD><FONT face="arial, helvetica,verdana" size=2>7 
        g/liter</FONT></TD></TR>
        <TR>
          <TD><FONT face="arial, helvetica,verdana" size=2>agarose (for 
            plates)</FONT></TD>
          <TD><FONT face="arial, helvetica,verdana" size=2>15 
        g/liter</FONT></TD></TR>
        <TR>
          <TD><FONT face="arial, helvetica,verdana" size=2>agarose (for top 
            agarose)</FONT></TD>
          <TD><FONT face="arial, helvetica,verdana" size=2>7 
        g/liter</FONT></TD></TR></TBODY></TABLE></P>
      <P>Sterilize by autoclaving for 20 minutes at 15 psi (1.05 kg/cm 
      <SUP>2</SUP>) on liquid cycle. When the medium is removed from the 
      autoclave, swirl it gently to distribute the melted agar or agarose evenly 
      throughout the solution. <I>Be careful!</I> The fluid may be superheated 
      and may boil over when swirled. Allow the medium to cool to 50-60°C before 
      adding thermolabile substances (e.g., antibiotics). To avoid producing air 
      bubbles, mix the medium by swirling. Plates can then be poured directly 
      from the flask; allow approx. 30-35 ml of medium per 90-mm plate. To 
      remove bubbles from medium in the plate, flame the surface of the medium 
      with a Bunsen burner before the agar or agarose hardens. Set up a color 
      code (e.g., two red stripes for LB-ampicillin plates; one black stripe for 
      LB plates, etc.) and mark the edges of the plates with the appropriate 
      colored markers.</P>
      <P>When the medium has hardened completely, invert the plates and store 
      them at 4°C until needed. The plates should be removed from storage 1-2 
      hours before they are used. If the plates are fresh, they will "sweat" 
      when incubated at 37°C. When this condensation drops on the agar/agarose 
      surface, it allows bacterial colonies or bacteriophage plaques to spread 
      and increases the chances of cross-contamination. This problem can be 
      avoided by wiping off the condensation from the lids of the plates and 
      then incubating the plates for several hours at 37°C in an inverted 
      position before they are used. Alternatively, remove the liquid by shaking 
      the lid with a single, quick motion. To minimize the possibility of 
      contamination, hold the open plate in an inverted position while removing 
      the liquid from the lid.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>NaCl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 5 M 
      solution: Dissolve 292 g of NaCl in 800 ml of H<SUB>2</SUB>O. Adjust the 
      volume to 1 liter with H<SUB>2</SUB>O. Dispense into aliquots and 
      sterilize by autoclaving. Store the NaCl solution at room 
      temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Oligonucleotide hybridization solution 
      (13-7)</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>6x SSC</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>5x Denhardt's 
      reagent</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>10<SUP>6</SUP> to 
      10<SUP>7</SUP> dpm/ml radiolabeled 
oligonucleotide</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Oligonucleotide prehybridization solution 
      (13-7)</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>6x SSC</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>5x Denhardt’s 
      reagent</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.1% (w/v) 
  SDS</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>SDS</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Also called sodium lauryl 
      sulfate. To prepare a 20% (w/v) solution, dissolve 200 g of 
      electrophoresis-grade SDS in 900 ml of H<SUB>2</SUB>O. Heat to 68°C and 
      stir with a magnetic stirrer to assist dissolution. If necessary, adjust 
      the pH to 7.2 by adding a few drops of concentrated HCl. Adjust the volume 
      to 1 liter with H<SUB>2</SUB>O. Store at room temperature. Sterilization 
      is not necessary. Do not autoclave.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>SSC</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>For 20x solution: 
      Dissolve 175.3 g of NaCl and 88.2 g of sodium citrate in 800 ml of 
      H<SUB>2</SUB>O. Adjust the pH to 7.0 with a few drops of a 14 N solution 
      of HCl. Adjust the volume to 1 liter with H<SUB>2</SUB>O. Dispense into 
      aliquots. Sterilize by autoclaving. The final concentrations of the 
      ingredients are 3.0 M NaCl and 0.3 M sodium 
acetate.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>TE</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM Tris-Cl (desired 
      pH)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>10 mM EDTA (pH 
    8.0)</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>(10x Tris EDTA) Sterilize 
      solutions by autoclaving for 20 minutes at 15 psi (1.05 kg/cm 
      <SUP>2</SUP>) on liquid cycle. Store the buffer at room 
    temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Cl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve 121.1 g of Tris base in 800 ml of H<SUB>2</SUB>O. Adjust the 
      pH to the desired value by adding concentrated HCl.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2><I>pH</I></FONT> </TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2><I>HCl</I></FONT> 
        </TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.4 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>70 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.6 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>8.0 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>42 ml 
        </FONT></TD></TR></TBODY></TABLE>
      <P>(1 M) Allow the solution to cool to room temperature before making 
      final adjustments to the pH. Adjust the volume of the solution to 1 liter 
      with H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
      autoclaving.</P>
      <P>If the 1 M solution has a yellow color, discard it and obtain Tris of 
      better quality. The pH of Tris solutions is temperature-dependent and 
      decreases approx. 0.03 pH units for each 1°C increase in temperature. For 
      example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5°C, 
      25°C, and 37°C, respectively.</P></FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>YT</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>deionized H<SUB>2</SUB>O, 
      to 900 ml</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>tryptone, 16 
  g</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>yeast extract, 10 
    g</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>NaCl, 5 g</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>For solid medium, please 
      see Media Containing Agar or Agarose.</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare 2x YT medium, 
      shake until the solutes have dissolved. Adjust the pH to 7.0 with 5 N 
      NaOH. Adjust the volume of the solution to 1 liter with deionized 
      H<SUB>2</SUB>O. Sterilize by autoclaving for 20 minutes at 15 psi (1.05 
      kg/cm <SUP>2</SUP>) on liquid 
cycle.</FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;     helvetica" size=3><STRONG>CAUTIONS</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" width=11 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Ammonium formate</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Ammonium formate, see Formic Acid </FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>Radioactive substances</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">Radioactive substances: When planning an experiment that involves 
      the use of radioactivity, consider the physico-chemical properties of the 
      isotope (half-life, emission type, and energy), the chemical form of the 
      radioactivity, its radioactive concentration (specific activity), total 
      amount, and its chemical concentration. Order and use only as much as 
      needed. Always wear appropriate gloves, lab coat, and safety goggles when 
      handling radioactive material. X-rays and gamma rays are electromagnetic 
      waves of very short wavelengths either generated by technical devices or 
      emitted by radioactive materials. They might be emitted isotropically from 
      the source or may be focused into a beam. Their potential dangers depend 
      on the time period of exposure, the intensity experienced, and the 
      wavelengths used. Be aware that appropriate shielding is usually made of 
      lead or other similar material. The thickness of the shielding is 
      determined by the energy(s) of the X-rays or gamma rays. Consult the local 
      safety office for further guidance in the appropriate use and disposal of 
      radioactive materials. Always monitor thoroughly after using 
      radioisotopes. A convenient calculator to perform routine radioactivity 
      calculations can be found at: <A target=_blank 
      href="http://www.graphpad.com/calculators/radcalc.cfm">http://www.graphpad.com/calculators/radcalc.cfm</A>. 
      </FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>SDS</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">SDS (Sodium dodecyl sulfate) is toxic, an irritant, and poses a 
      risk of severe damage to the eyes. It may be harmful by inhalation, 
      ingestion, or skin absorption. Wear appropriate gloves and safety goggles. 
      Do not breathe the dust. </FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>REFERENCES</STRONG> 
</FONT><BR>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Zoller M.J. and 
  Smith M. 1982. Oligonucleotide-directed mutagenesis using M13-derived vectors: 
  An efficient and general procedure for the production of point mutations in 
  any fragment of DNA. <I>Nucleic Acids Res.</I> 10:6487-6500. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Zoller M.J. and 
  Smith M. 1983. Oligonucleotide-directed mutagenesis of DNA fragments cloned 
  into M13 vectors. <I>Methods Enzymol.</I> 100:468-500. 
</FONT></LI></OL><BR><BR><BR><BR><BR><FONT face=Arial,Verdana,Helvetica 
size=1><STRONG>Copyright © 2000 by Cold Spring Harbor Laboratory Press. All 
rights reserved.</STRONG><BR>No part of these pages, either text or image may be 
used for any purpose other than personal use. Therefore, reproduction 
modification, storage in a retrieval system or retransmission, in any form or by 
any means, electronic, mechanical, or otherwise, for reasons other than personal 
use, is strictly prohibited without prior written 
permission.</FONT><BR><BR>
</td></tr></table>
</BODY></HTML>
