<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Molecular Cloning | Protocol 17 Print Version</TITLE>
<META http-equiv=Content-Type content="text/html; charset=windows-1251"><!--================== Designed and Developed by WriteDesign =================--><!--======================= Powered by RomeBlack, Inc. =======================--><!--=== Copyright 2000, 2001 Cold Spring Harbor Labs. All Rights Reserved. ===-->
<META content="" name=keywords>
<META 
content="In this form of pulsed-field gel electrophoresis, electrodes are positioned on opposite sides of a vertically oriented gel. The DNA moves first toward one electrode and then toward the other, forming a zigzag pattern. The vector of this oscillation is a straight line from the loading well to the base of the gel. Variation in voltage and pulse time allows separation of DNAs ranging in size from 2 kb to >6000 kb. This protocol, supplied by Tommy Hyatt and Helen Hobbs (University of Texas Southwestern Medical Center, Dallas), describes the resolution of genomic DNA by TAFE, followed by blotting and hybridization." 
name=description>
<META content="MSHTML 5.50.4134.600" name=GENERATOR></HEAD>
<BODY bgColor=#ffffff background=../izo/fon.gif>
<table width=600 border="0" cellspacing="0" cellpadding="0"><tr><td width=600><IMG height=49 hspace=4 src="../izo/logo_csh.gif" 
width=169 align=left vspace=4><IMG height=49 hspace=4 
src="../izo/print_header.gif" width=293 align=left vspace=4><BR 
clear=all><BR><FONT face="arial,verdana, helvetica" size=4><STRONG>Chapter 5, 
Protocol 17</STRONG></FONT><BR><BR clear=all><FONT face=arial,verdana,helvetica 
size=3><STRONG>Pulsed-field Gel Electrophoresis via Transverse Alternating Field 
Electrophoresis Gels</STRONG></FONT><BR><BR><FONT 
face="arial,verdana, helvetica" size=2>In this form of pulsed-field gel 
electrophoresis, electrodes are positioned on opposite sides of a vertically 
oriented gel. The DNA moves first toward one electrode and then toward the 
other, forming a zigzag pattern. The vector of this oscillation is a straight 
line from the loading well to the base of the gel. Variation in voltage and 
pulse time allows separation of DNAs ranging in size from 2 kb to &gt;6000 kb. 
This protocol, supplied by Tommy Hyatt and Helen Hobbs (University of Texas 
Southwestern Medical Center, Dallas), describes the resolution of genomic DNA by 
TAFE, followed by blotting and hybridization.</FONT> <BR><BR>
<TABLE border=1>
  <TBODY>
  <TR>
    <TD width=700><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" 
      width=11 border=0><FONT face=arial,verdana,helvetica 
      size=2><STRONG>CAUTION</STRONG></FONT><BR><IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0><FONT 
      face="arial,verdana, helvetica" size=2> <STRONG>RECIPE</STRONG></FONT> 
  </TD></TR></TBODY></TABLE><BR><BR><FONT face=arial,verdana,helvetica 
size=3><STRONG>MATERIALS</STRONG></FONT><BR><BR>
<TABLE cellPadding=0 cellborder="0">
  <TBODY>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Buffers and Solutions</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Denaturation 
      solution</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>TAFE gel 
      electrophoresis buffer</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>TAFE gel 
      electrophoresis buffer containing 0.5 µg/ml ethidium bromide or an 
      appropriate dilution of SYBR Gold</FONT> <IMG height=12 alt="" hspace=4 
      src="../izo/circle.gif" width=12 border=0> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>A buffer of 
      0.045 M Tris-borate (pH 8.2) and 0.01 M EDTA has also been used 
      successfully in this protocol.<BR>For a discussion of staining agarose 
      gels, please see <A href="02.htm">Chapter 
      5, Protocol 2 </A>.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>TE (pH 
      8.0)</FONT> <IMG height=12 alt="" hspace=4 src="../izo/circle.gif" 
      width=12 border=0> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Nucleic Acids and Oligonucleotides</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>DNA size 
      standards</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Please see 
      Step 2.</I></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Genomic DNA of 
      interest</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2><I>Please see 
      Step 2.</I></FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 border=0></TD></TR>
  <TR>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" 
      size=2><STRONG>Additional Reagents</STRONG></FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 2 of this 
      protocol requires the reagents listed in <A href="13.htm">Chapter 
      5, Protocol 13 </A>or <A href="14.htm">Chapter 
      5, Protocol 14 </A>, <A href="15.htm">Chapter 
      5, Protocol 15 </A>, and <A href="16.htm">Chapter 
      5, Protocol 16 </A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Steps 9 and 10 
      of this protocol require the reagents listed in <A href="../06/08.htm">Chapter 
      6, Protocol 8 </A>.</FONT> </TD></TR>
  <TR>
    <TD vAlign=top><FONT face="arial,verdana, helvetica" size=2>Step 11 of 
      this protocol requires the reagents listed in <A href="../06/10.htm">Chapter 
      6, Protocol 10 </A>.</FONT> </TD></TR>
  <TR>
    <TD><IMG height=10 src="../izo/spacer.gif" width=1 
  border=0></TD></TR></TBODY></TABLE><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" 
size=3><STRONG>METHOD</STRONG></FONT> <BR><FONT face="arial,verdana, helvetica" 
color=#000000 size=2>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Cast a 1% 
  agarose gel in 1x TAFE gel buffer without ethidium bromide and allow the gel 
  to set. Pour the gel with the same buffer solution to be used to fill the gel 
  apparatus. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Prepare agarose 
  plugs containing the DNA of interest (please see <A href="13.htm">Chapter 
  5, Protocol 13 </A>for preparation of mammalian DNA or <A href="14.htm">Chapter 
  5, Protocol 14 </A>for preparation of yeast DNA embedded in plugs) and carry 
  out digestion with restriction enzymes as described in <A href="15.htm">Chapter 
  5, Protocol 15 </A>. Prepare and embed the appropriate DNA size standards as 
  described in <A href="14.htm">Chapter 
  5, Protocol 14 </A>and <A href="16.htm">Chapter 
  5, Protocol 16 </A>. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Rinse all of 
  the plugs in 10 volumes of TE (pH 8.0) for 30 minutes with two changes of 
  buffer. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Embed the 
  digested and rinsed DNA plugs in individual wells of the gel. Seal the plugs 
  in the wells with molten 1% agarose in 1x TAFE gel buffer. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Place the gel 
  in the TAFE apparatus filled with 1x TAFE gel buffer previously cooled to 
  14°C. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Connect the gel 
  apparatus to the appropriate power supply set to deliver 4-second pulses at a 
  constant current of 170-180 mA for 30 minutes. This setting forces the DNA to 
  enter the gel rapidly. After this time period, decrease the current input to 
  150 mA, change the pulse time to a setting optimum for the size range of DNAs 
  to be resolved (please see the table below), and continue electrophoresis for 
  12-18 hours.<BR><B>TAFE Gel Conditions for Separating DNAs of Various 
  Sizes</B> 
  <TABLE>
    <TBODY>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2><B>Size Range 
        (kb)</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2><B>Pulse 
        Time<BR>(Seconds)</B></FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2><B>Time 
        (Hours)</B></FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>5-50</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>1</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
size=2>20-100</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>3</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
size=2>50-250</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>10</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>14</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
      size=2>100-400</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>14</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
      size=2>200-1000</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>45</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>18</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica 
      size=2>200-1600</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>70</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>20</FONT></TD></TR>
    <TR>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>Larger 
        DNAs</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>see 
      text</FONT></TD>
      <TD width=5></FONT></TD>
      <TD vAlign=top><FONT face=arial,verdana,helvetica size=2>see 
      text</FONT></TD></TR></TBODY></TABLE><I>In a Tris-acetate/EDTA buffer, a pulse 
  time of 15 seconds will separate DNA fragments in the 50-400-kb size range. 
  This same range of fragments can be separated in the Tris-borate/EDTA buffer 
  using a program of 8-second pulses at 350 mA for 12 hours followed by 
  15-second pulses at 350 mA for an additional 12 hours.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Disconnect the 
  power supply, dismantle the gel apparatus, and stain the gel in 1x TAFE buffer 
  containing 0.5 µg/ml ethidium bromide or an appropriate dilution of SYBR Gold. 
  Photograph the gel under UV light.<BR><I>The standard technique used to detect 
  DNAs separated by PFGE is staining with ethidium bromide or SYBR Gold (please 
  see <A href="02.htm">Chapter 
  5, Protocol 2 </A>). To facilitate the detection of minor species of DNA 
  stained with ethidium bromide, the gels may be destained in H<SUB>2</SUB>O for 
  up to 1 hour before photography. For details of methods that can be used to 
  maximize the photographic detection of DNA stained with these dyes, please see 
  <A href="02.htm">Chapter 
  5, Protocol 2</A>.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Rinse the 
  stained gel twice with H<SUB>2</SUB>O. Pour off the second H<SUB>2</SUB>O 
  rinse and replace with denaturation solution. Incubate with gentle shaking for 
  30 minutes. Change the denaturation solution and incubate for an additional 30 
  minutes.<BR><I>In general, standard methods of Southern blotting as described 
  in Chapter 6 (please see <A href="08.htm">Chapter 
  5, Protocol 8 </A>and <A href="09.htm">Chapter 
  5, Protocol 9 </A>) can be used to detect hybridizing gene sequences in TAFE 
  gels. Some investigators find that the very large DNA fragments typically 
  analyzed by PFGE transfer to nylon membranes more efficiently after partial 
  hydrolysis of the DNA by acid treatment. This treatment causes partial 
  depurination and nicking of larger DNA fragments and in so doing enhances 
  capillary transfer. To include an acid hydrolysis step, after the 
  electrophoresis is complete, rinse the gel twice with H<SUB>2</SUB>O in Step 
  8, pour off the second H<SUB>2</SUB>O rinse, and replace with 25 mM HCl. Soak 
  the gel with gentle agitation for 3-5 minutes. Rinse the gel with 
  H<SUB>2</SUB>O and continue with the denaturation protocol of Step 8 and 
  onward.</I><BR><I>When acid treatments are used in the protocol, it is 
  important to stain the agarose gel after transfer of the DNA to the nylon 
  membrane. A slight residual smear of DNA should be visible. If no residual DNA 
  is detected, then the acid treatment may have been too harsh (i.e., too long 
  or too strong). Too much depurination and nicking increases the transfer of 
  the DNA but also tends to reduce the intensity of subsequent hybridization 
  signals.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Transfer the 
  DNA directly to a nylon membrane by capillary blotting in denaturation 
  solution (for details, please see <A href="../06/08.htm">Chapter 
  6, Protocol 8</A>).<BR><I>Some investigators find that transfer of larger DNA 
  fragments is enhanced when 6x SSC buffer is used in capillary blotting rather 
  than the more standard 10x SSC solution.</I> </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>After transfer, 
  affix the DNA to the nylon membrane by baking for 2 hours at 80°C, by UV 
  cross-linking, or by microwaving. </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Carry out 
  prehybridization and hybridization with labeled probes in a 
  formamide-containing buffer (for details, please see <A href="../06/10.htm">Chapter 
  6, Protocol 10 </A>).<BR><I>We typically use <SUP>32</SUP>P-radiolabeled 
  single-stranded bacteriophage M13 probes at a concentration of 5 x 
  10<SUP>6</SUP> to 6 x 10<SUP>6</SUP> cpm/ml of hybridization buffer to detect 
  a single-copy gene in a complex mammalian genome. A membrane hybridized for 
  12-16 hours in this fashion is washed in 500 ml of 2x SSC containing 1% (w/v) 
  SDS for 15 minutes at room temperature, scrubbed gently with a sponge, then 
  washed in 1 liter of 0.5x SSC containing 1% (w/v) SDS for 2 hours at 68°C, and 
  then subjected to autoradiography.</I> </FONT></LI></OL></FONT><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>RECIPES</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/circle.gif" width=12 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Denaturation Solution</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>1.5 M NaCl</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.5 M NaOH</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>For neutral transfer, 
      double-stranded DNA targets only.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>EDTA</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare EDTA at 0.5 M 
      (pH 8.0): Add 186.1 g of disodium EDTA•2H<SUB>2</SUB>O to 800 ml of 
      H<SUB>2</SUB>O. Stir vigorously on a magnetic stirrer. Adjust the pH to 
      8.0 with NaOH (approx. 20 g of NaOH pellets). Dispense into aliquots and 
      sterilize by autoclaving. The disodium salt of EDTA will not go into 
      solution until the pH of the solution is adjusted to approx. 8.0 by the 
      addition of NaOH.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>NaCl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>To prepare a 5 M 
      solution: Dissolve 292 g of NaCl in 800 ml of H<SUB>2</SUB>O. Adjust the 
      volume to 1 liter with H<SUB>2</SUB>O. Dispense into aliquots and 
      sterilize by autoclaving. Store the NaCl solution at room 
      temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>NaOH</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>The preparation of 10 N 
      NaOH involves a highly exothermic reaction, which can cause breakage of 
      glass containers. Prepare this solution with extreme care in plastic 
      beakers. To 800 ml of H<SUB>2</SUB>O, slowly add 400g of NaOH pellets, 
      stirring continuously. As an added precaution, place the beaker on ice. 
      When the pellets have dissolved completely, adjust the volume to 1 liter 
      with H<SUB>2</SUB>O. Store the solution in a plastic container at room 
      temperature. Sterilization is not 
necessary.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>TAFE Gel Electrophoresis 
      Buffer</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>20 mM Tris-acetate (pH 
      8.2)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>0.5 mM EDTA</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>Use acetic acid to adjust 
      the pH of the Tris solution to 8.2, and use the free acid of EDTA, not the 
      sodium salt. Concentrated solutions of TAFE buffer can also be purchased 
      (e.g., from Beckman). The TAFE gel electrophoresis buffer must be cooled 
      to 14°C before use.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>TE</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>100 mM Tris-Cl (desired 
      pH)</FONT></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>10 mM EDTA (pH 
    8.0)</FONT></TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>(10x Tris EDTA) Sterilize 
      solutions by autoclaving for 20 minutes at 15 psi (1.05 kg/cm 
      <SUP>2</SUP>) on liquid cycle. Store the buffer at room 
    temperature.</FONT></TD></TR></TBODY></TABLE><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left width=400><FONT face=arial,verdana,helvetica 
      color=#000000 size=2><STRONG>Tris-Cl</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=12 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD></TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD><FONT face="arial,verdana, helvetica" size=2>
      <P>Dissolve 121.1 g of Tris base in 800 ml of H<SUB>2</SUB>O. Adjust the 
      pH to the desired value by adding concentrated HCl.</P>
      <TABLE>
        <TBODY>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2><I>pH</I></FONT> </TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2><I>HCl</I></FONT> 
        </TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.4 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>70 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>7.6 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>60 ml </FONT></TD></TR>
        <TR>
          <TD><FONT face=arial,helvetica,verdana size=2>8.0 </FONT></TD>
          <TD width=4>&nbsp; </TD>
          <TD><FONT face=arial,helvetica,verdana size=2>42 ml 
        </FONT></TD></TR></TBODY></TABLE>
      <P>(1 M) Allow the solution to cool to room temperature before making 
      final adjustments to the pH. Adjust the volume of the solution to 1 liter 
      with H<SUB>2</SUB>O. Dispense into aliquots and sterilize by 
      autoclaving.</P>
      <P>If the 1 M solution has a yellow color, discard it and obtain Tris of 
      better quality. The pH of Tris solutions is temperature-dependent and 
      decreases approx. 0.03 pH units for each 1°C increase in temperature. For 
      example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5°C, 
      25°C, and 37°C, 
respectively.</P></FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;     helvetica" size=3><STRONG>CAUTIONS</STRONG> 
</FONT><IMG height=11 alt=CSH hspace=4 src="../izo/triangle.gif" width=11 
border=0> <BR><BR>
<TABLE border=0>
  <TBODY>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face=arial,verdana,helvetica color=#000000 
      size=2><STRONG>NaOH</STRONG></FONT> </TD></TR>
  <TR>
    <TD><IMG height=1 src="../izo/spacer.gif" width=40 border=0></TD>
    <TD vAlign=top align=left><FONT face="arial,verdana, helvetica" size=2 
      font="">NaOH, see Sodium hydroxide 
</FONT></TD></TR></TBODY></TABLE><BR><BR><BR><FONT 
face="arial,verdana,&#13;&#10;    helvetica" size=3><STRONG>REFERENCES</STRONG> 
</FONT><BR>
<OL>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Gardiner K. and 
  Patterson D. 1989. Transverse alternating field electrophoresis and 
  applications to mammalian genome mapping. <I>Electrophoresis</I> 10:296-302. 
  </FONT>
  <LI><FONT face="arial,verdana, helvetica" color=#000000 size=2>Gardiner K., 
  Laas W., and Patterson D. 1986. Fractionation of large mammalian DNA 
  restriction fragments using vertical pulsed-field gradient gel 
  electrophoresis. <I>Somat. Cell Mol. Genet.</I> 12:185-195. 
</FONT></LI></OL><BR><BR><BR><BR><BR><FONT face=Arial,Verdana,Helvetica 
size=1><STRONG>Copyright © 2000 by Cold Spring Harbor Laboratory Press. All 
rights reserved.</STRONG><BR>No part of these pages, either text or image may be 
used for any purpose other than personal use. Therefore, reproduction 
modification, storage in a retrieval system or retransmission, in any form or by 
any means, electronic, mechanical, or otherwise, for reasons other than personal 
use, is strictly prohibited without prior written 
permission.</FONT><BR><BR>
</td></tr></table>
</BODY></HTML>
